<!DOCTYPE html>
<html lang="en"><head>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-html/tabby.min.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/light-border.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-html.min.css" rel="stylesheet" data-mode="light">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles"><meta charset="utf-8">
  <meta name="generator" content="quarto-1.5.57">

  <meta name="dcterms.date" content="2024-11-14">
  <title>Biologia della Conservazione 24/25 – Biologia della Conservazione</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="../../site_libs/revealjs/dist/reset.css">
  <link rel="stylesheet" href="../../site_libs/revealjs/dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
      vertical-align: middle;
    }
    /* CSS for citations */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
      margin-bottom: 0em;
    }
    .hanging-indent div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }  </style>
  <link rel="stylesheet" href="../../site_libs/revealjs/dist/theme/quarto.css">
  <link href="../../site_libs/revealjs/plugin/quarto-line-highlight/line-highlight.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-menu/menu.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-menu/quarto-menu.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-chalkboard/font-awesome/css/all.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-chalkboard/style.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">
  <style type="text/css">

  .callout {
    margin-top: 1em;
    margin-bottom: 1em;  
    border-radius: .25rem;
  }

  .callout.callout-style-simple { 
    padding: 0em 0.5em;
    border-left: solid #acacac .3rem;
    border-right: solid 1px silver;
    border-top: solid 1px silver;
    border-bottom: solid 1px silver;
    display: flex;
  }

  .callout.callout-style-default {
    border-left: solid #acacac .3rem;
    border-right: solid 1px silver;
    border-top: solid 1px silver;
    border-bottom: solid 1px silver;
  }

  .callout .callout-body-container {
    flex-grow: 1;
  }

  .callout.callout-style-simple .callout-body {
    font-size: 1rem;
    font-weight: 400;
  }

  .callout.callout-style-default .callout-body {
    font-size: 0.9rem;
    font-weight: 400;
  }

  .callout.callout-titled.callout-style-simple .callout-body {
    margin-top: 0.2em;
  }

  .callout:not(.callout-titled) .callout-body {
      display: flex;
  }

  .callout:not(.no-icon).callout-titled.callout-style-simple .callout-content {
    padding-left: 1.6em;
  }

  .callout.callout-titled .callout-header {
    padding-top: 0.2em;
    margin-bottom: -0.2em;
  }

  .callout.callout-titled .callout-title  p {
    margin-top: 0.5em;
    margin-bottom: 0.5em;
  }
    
  .callout.callout-titled.callout-style-simple .callout-content  p {
    margin-top: 0;
  }

  .callout.callout-titled.callout-style-default .callout-content  p {
    margin-top: 0.7em;
  }

  .callout.callout-style-simple div.callout-title {
    border-bottom: none;
    font-size: .9rem;
    font-weight: 600;
    opacity: 75%;
  }

  .callout.callout-style-default  div.callout-title {
    border-bottom: none;
    font-weight: 600;
    opacity: 85%;
    font-size: 0.9rem;
    padding-left: 0.5em;
    padding-right: 0.5em;
  }

  .callout.callout-style-default div.callout-content {
    padding-left: 0.5em;
    padding-right: 0.5em;
  }

  .callout.callout-style-simple .callout-icon::before {
    height: 1rem;
    width: 1rem;
    display: inline-block;
    content: "";
    background-repeat: no-repeat;
    background-size: 1rem 1rem;
  }

  .callout.callout-style-default .callout-icon::before {
    height: 0.9rem;
    width: 0.9rem;
    display: inline-block;
    content: "";
    background-repeat: no-repeat;
    background-size: 0.9rem 0.9rem;
  }

  .callout-title {
    display: flex
  }
    
  .callout-icon::before {
    margin-top: 1rem;
    padding-right: .5rem;
  }

  .callout.no-icon::before {
    display: none !important;
  }

  .callout.callout-titled .callout-body > .callout-content > :last-child {
    padding-bottom: 0.5rem;
    margin-bottom: 0;
  }

  .callout.callout-titled .callout-icon::before {
    margin-top: .5rem;
    padding-right: .5rem;
  }

  .callout:not(.callout-titled) .callout-icon::before {
    margin-top: 1rem;
    padding-right: .5rem;
  }

  /* Callout Types */

  div.callout-note {
    border-left-color: #4582ec !important;
  }

  div.callout-note .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAEU0lEQVRYCcVXTWhcVRQ+586kSUMMxkyaElstCto2SIhitS5Ek8xUKV2poatCcVHtUlFQk8mbaaziwpWgglJwVaquitBOfhQXFlqlzSJpFSpIYyXNjBNiTCck7x2/8/LeNDOZxDuEkgOXe++553zfefee+/OYLOXFk3+1LLrRdiO81yNqZ6K9cG0P3MeFaMIQjXssE8Z1JzLO9ls20MBZX7oG8w9GxB0goaPrW5aNMp1yOZIa7Wv6o2ykpLtmAPs/vrG14Z+6d4jpbSKuhdcSyq9wGMPXjonwmESXrriLzFGOdDBLB8Y6MNYBu0dRokSygMA/mrun8MGFN3behm6VVAwg4WR3i6FvYK1T7MHo9BK7ydH+1uurECoouk5MPRyVSBrBHMYwVobG2aOXM07sWrn5qgB60rc6mcwIDJtQrnrEr44kmy+UO9r0u9O5/YbkS9juQckLed3DyW2XV/qWBBB3ptvI8EUY3I9p/67OW+g967TNr3Sotn3IuVlfMLVnsBwH4fsnebJvyGm5GeIUA3jljERmrv49SizPYuq+z7c2H/jlGC+Ghhupn/hcapqmcudB9jwJ/3jvnvu6vu5lVzF1fXyZuZZ7U8nRmVzytvT+H3kilYvH09mLWrQdwFSsFEsxFVs5fK7A0g8gMZjbif4ACpKbjv7gNGaD8bUrlk8x+KRflttr22JEMRUbTUwwDQScyzPgedQHZT0xnx7ujw2jfVfExwYHwOsDTjLdJ2ebmeQIlJ7neo41s/DrsL3kl+W2lWvAga0tR3zueGr6GL78M3ifH0rGXrBC2aAR8uYcIA5gwV8zIE8onoh8u0Fca/ciF7j1uOzEnqcIm59sEXoGc0+z6+H45V1CvAvHcD7THztu669cnp+L0okAeIc6zjbM/24LgGM1gZk7jnRu1aQWoU9sfUOuhrmtaPIO3YY1KLLWZaEO5TKUbMY5zx8W9UJ6elpLwKXbsaZ4EFl7B4bMtDv0iRipKoDQT2sNQI9b1utXFdYisi+wzZ/ri/1m7QfDgEuvgUUEIJPq3DhX/5DWNqIXDOweC2wvIR90Oq3lDpdMIgD2r0dXvGdsEW5H6x6HLRJYU7C69VefO1x8Gde1ZFSJLfWS1jbCnhtOPxmpfv2LXOA2Xk2tvnwKKPFuZ/oRmwBwqRQDcKNeVQkYcOjtWVBuM/JuYw5b6isojIkYxyYAFn5K7ZBF10fea52y8QltAg6jnMqNHFBmGkQ1j+U43HMi2xMar1Nv0zGsf1s8nUsmUtPOOrbFIR8bHFDMB5zL13Gmr/kGlCkUzedTzzmzsaJXhYawnA3UmARpiYj5ooJZiUoxFRtK3X6pgNPv+IZVPcnwbOl6f+aBaO1CNvPW9n9LmCp01nuSaTRF2YxHqZ8DYQT6WsXT+RD6eUztwYLZ8rM+rcPxamv1VQzFUkzFXvkiVrySGQgJNvXHJAxiU3/NwiC03rSf05VBaPtu/Z7/B8Yn/w7eguloAAAAAElFTkSuQmCC');
  }

  div.callout-note.callout-style-default .callout-title {
    background-color: #dae6fb
  }

  div.callout-important {
    border-left-color: #d9534f !important;
  }

  div.callout-important .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAEKklEQVRYCcVXTWhcVRS+575MJym48A+hSRFr00ySRQhURRfd2HYjk2SSTokuBCkU2o0LoSKKraKIBTcuFCoidGFD08nkBzdREbpQ1EDNIv8qSGMFUboImMSZd4/f9zJv8ibJMC8xJQfO3HPPPef7zrvvvnvviIkpC9nsw0UttFunbUhpFzFtarSd6WJkStVMw5xyVqYTvkwfzuf/5FgtkVoB0729j1rjXwThS7Vio+Mo6DNnvLfahoZ+i/o32lULuJ3NNiz7q6+pyAUkJaFF6JwaM2lUJlV0MlnQn5aTRbEu0SEqHUa0A4AdiGuB1kFXRfVyg5d87+Dg4DL6m2TLAub60ilj7A1Ec4odSAc8X95sHh7+ZRPCFo6Fnp7HfU/fBng/hi10CjCnWnJjsxvDNxWw0NfV6Rv5GgP3I3jGWXumdTD/3cbEOP2ZbOZp69yniG3FQ9z1jD7bnBu9Fc2tKGC2q+uAJOQHBDRiZX1x36o7fWBs7J9ownbtO+n0/qWkvW7UPIfc37WgT6ZGR++EOJyeQDSb9UB+DZ1G6DdLDzyS+b/kBCYGsYgJbSQHuThGKRcw5xdeQf8YdNHsc6ePXrlSYMBuSIAFTGAtQo+VuALo4BX83N190NWZWbynBjhOHsmNfFWLeL6v+ynsA58zDvvAC8j5PkbOcXCMg2PZFk3q8MjI7WAG/Dp9AwP7jdGBOOQkAvlFUB+irtm16I1Zw9YBcpGTGXYmk3kQIC/Cds55l+iMI3jqhjAuaoe+am2Jw5GT3Nbz3CkE12NavmzN5+erJW7046n/CH1RO/RVa8lBLozXk9uqykkGAyRXLWlLv5jyp4RFsG5vGVzpDLnIjTWgnRy2Rr+tDKvRc7Y8AyZq10jj8DqXdnIRNtFZb+t/ZRtXcDiVnzpqx8mPcDWxgARUqx0W1QB9MeUZiNrV4qP+Ehc+BpNgATsTX8ozYKL2NtFYAHc84fG7ndxUPr+AR/iQSns7uSUufAymwDOb2+NjK27lEFocm/EE2WpyIy/Hi66MWuMKJn8RvxIcj87IM5Vh9663ziW36kR0HNenXuxmfaD8JC7tfKbrhFr7LiZCrMjrzTeGx+PmkosrkNzW94ObzwocJ7A1HokLolY+AvkTiD/q1H0cN48c5EL8Crkttsa/AXQVDmutfyku0E7jShx49XqV3MFK8IryDhYVbj7Sj2P2eBxwcXoe8T8idsKKPRcnZw1b+slFTubwUwhktrfnAt7J++jwQtLZcm3sr9LQrjRzz6cfMv9aLvgmnAGvpoaGLxM4mAEaLV7iAzQ3oU0IvD5x9ix3yF2RAAuYAOO2f7PEFWCXZ4C9Pb2UsgDeVnFSpbFK7/IWu7TPTvBqzbGdCHOJQSxiEjt6IyZmxQyEJHv6xyQsYk//moVFsN2zP6fRImjfq7/n/wFDguUQFNEwugAAAABJRU5ErkJggg==');
  }

  div.callout-important.callout-style-default .callout-title {
    background-color: #f7dddc
  }

  div.callout-warning {
    border-left-color: #f0ad4e !important;
  }

  div.callout-warning .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAETklEQVRYCeVWW2gcVRg+58yaTUnizqbipZeX4uWhBEniBaoUX1Ioze52t7sRq6APio9V9MEaoWlVsFasRq0gltaAPuxms8lu0gcviE/FFOstVbSIxgcv6SU7EZqmdc7v9+9mJtNks51NTUH84ed889/PP+cmxP+d5FIbMJmNbpREu4WUkiTtCicKny0l1pIKmBzovF2S+hIJHX8iEu3hZJ5lNZGqyRrGSIQpq15AzF28jgpeY6yk6GVdrfFqdrD6Iw+QlB8g0YS2g7dyQmXM/IDhBhT0UCiRf59lfqmmDvzRt6kByV/m4JjtzuaujMUM2c5Z2d6JdKrRb3K2q6mA+oYVz8JnDdKPmmNthzkAk/lN63sYPgevrguc72aZX/L9C6x09GYyxBgCX4NlvyGUHOKELlm5rXeR1kchuChJt4SSwyddZRXgvwMGvYo4QSlk3/zkHD8UHxwVJA6zjZZqP8v8kK8OWLnIZtLyCAJagYC4rTGW/9Pqj92N/c+LUaAj27movwbi19tk/whRCIE7Q9vyI6yvRpftAKVTdUjOW40X3h5OXsKCdmFcx0xlLJoSuQngnrJe7Kcjm4OMq9FlC7CMmScQANuNvjfP3PjGXDBaUQmbp296S5L4DrpbrHN1T87ZVEZVCzg1FF0Ft+dKrlLukI+/c9ENo+TvlTDbYFvuKPtQ9+l052rXrgKoWkDAFnvh0wTOmYn8R5f4k/jN/fZiCM1tQx9jQQ4ANhqG4hiL0qIFTGViG9DKB7GYzgubnpofgYRwO+DFjh0Zin2m4b/97EDkXkc+f6xYAPX0KK2I/7fUQuwzuwo/L3AkcjugPNixC8cHf0FyPjWlItmLxWw4Ou9YsQCr5fijMGoD/zpdRy95HRysyXA74MWOnscpO4j2y3HAVisw85hX5+AFBRSHt4ShfLFkIMXTqyKFc46xdzQM6XbAi702a7sy04J0+feReMFKp5q9esYLCqAZYw/k14E/xcLLsFElaornTuJB0svMuJINy8xkIYuL+xPAlWRceH6+HX7THJ0djLUom46zREu7tTkxwmf/FdOZ/sh6Q8qvEAiHpm4PJ4a/doJe0gH1t+aHRgCzOvBvJedEK5OFE5jpm4AGP2a8Dxe3gGJ/pAutug9Gp6he92CsSsWBaEcxGx0FHytmIpuqGkOpldqNYQK8cSoXvd+xLxXADw0kf6UkJNFtdo5MOgaLjiQOQHcn+A6h5NuL2s0qsC2LOM75PcF3yr5STuBSAcGG+meA14K/CI21HcS4LBT6tv0QAh8Dr5l93AhZzG5ZJ4VxAqdZUEl9z7WJ4aN+svMvwHHL21UKTd1mqvChH7/Za5xzXBBKrUcB0TQ+Ulgkfbi/H/YT5EptrGzsEK7tR1B7ln9BBwckYfMiuSqklSznIuoIIOM42MQO+QnduCoFCI0bpkzjCjddHPN/F+2Yu+sd9bKNpVwHhbS3LluK/0zgfwD0xYI5dXuzlQAAAABJRU5ErkJggg==');
  }

  div.callout-warning.callout-style-default .callout-title {
    background-color: #fcefdc
  }

  div.callout-tip {
    border-left-color: #02b875 !important;
  }

  div.callout-tip .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAADr0lEQVRYCe1XTWgTQRj9ZjZV8a9SPIkKgj8I1bMHsUWrqYLVg4Ue6v9BwZOxSYsIerFao7UiUryIqJcqgtpimhbBXoSCVxUFe9CTiogUrUp2Pt+3aUI2u5vdNh4dmMzOzHvvezuz8xNFM0mjnbXaNu1MvFWRXkXEyE6aYOYJpdW4IXuA4r0fo8qqSMDBU0v1HJUgVieAXxzCsdE/YJTdFcVIZQNMyhruOMJKXYFoLfIfIvVIMWdsrd+Rpd86ZmyzzjJmLStqRn0v8lzkb4rVIXvnpScOJuAn2ACC65FkPzEdEy4TPWRLJ2h7z4cArXzzaOdKlbOvKKX25Wl00jSnrwVxAg3o4dRxhO13RBSdNvH0xSARv3adTXbBdTf64IWO2vH0LT+cv4GR1DJt+DUItaQogeBX/chhbTBxEiZ6gftlDNXTrvT7co4ub5A6gp9HIcHvzTa46OS5fBeP87Qm0fQkr4FsYgVQ7Qg+ZayaDg9jhg1GkWj8RG6lkeSacrrHgDaxdoBiZPg+NXV/KifMuB6//JmYH4CntVEHy/keA6x4h4CU5oFy8GzrBS18cLJMXcljAKB6INjWsRcuZBWVaS3GDrqB7rdapVIeA+isQ57Eev9eCqzqOa81CY05VLd6SamW2wA2H3SiTbnbSxmzfp7WtKZkqy4mdyAlGx7ennghYf8voqp9cLSgKdqNfa6RdRsAAkPwRuJZNbpByn+RrJi1RXTwdi8RQF6ymDwGMAtZ6TVE+4uoKh+MYkcLsT0Hk8eAienbiGdjJHZTpmNjlbFJNKDVAp2fJlYju6IreQxQ08UJDNYdoLSl6AadO+fFuCQqVMB1NJwPm69T04Wv5WhfcWyfXQB+wXRs1pt+nCknRa0LVzSA/2B+a9+zQJadb7IyyV24YAxKp2Jqs3emZTuNnKxsah+uabKbMk7CbTgJx/zIgQYErIeTKRQ9yD9wxVof5YolPHqaWo7TD6tJlh7jQnK5z2n3+fGdggIOx2kaa2YI9QWarc5Ce1ipNWMKeSG4DysFF52KBmTNMmn5HqCFkwy34rDg05gDwgH3bBi+sgFhN/e8QvRn8kbamCOhgrZ9GJhFDgfcMHzFb6BAtjKpFhzTjwv1KCVuxHvCbsSiEz4CANnj84cwHdFXAbAOJ4LTSAawGWFn5tDhLMYz6nWeU2wJfIhmIJBefcd/A5FWQWGgrWzyORZ3Q6HuV+Jf0Bj+BTX69fm1zWgK7By1YTXchFDORywnfQ7GpzOo6S+qECrsx2ifVQAAAABJRU5ErkJggg==');
  }

  div.callout-tip.callout-style-default .callout-title {
    background-color: #ccf1e3
  }

  div.callout-caution {
    border-left-color: #fd7e14 !important;
  }

  div.callout-caution .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAACV0lEQVRYCdVWzWoUQRCuqp2ICBLJXgITZL1EfQDBW/bkzUMUD7klD+ATSHBEfAIfQO+iXsWDxJsHL96EHAwhgzlkg8nBg25XWb0zIb0zs9muYYWkoKeru+vn664fBqElyZNuyh167NXJ8Ut8McjbmEraKHkd7uAnAFku+VWdb3reSmRV8PKSLfZ0Gjn3a6Xlcq9YGb6tADjn+lUfTXtVmaZ1KwBIvFI11rRXlWlatwIAAv2asaa9mlB9wwygiDX26qaw1yYPzFXg2N1GgG0FMF8Oj+VIx7E/03lHx8UhvYyNZLN7BwSPgekXXLribw7w5/c8EF+DBK5idvDVYtEEwMeYefjjLAdEyQ3M9nfOkgnPTEkYU+sxMq0BxNR6jExrAI31H1rzvLEfRIdgcv1XEdj6QTQAS2wtstEALLG1yEZ3QhH6oDX7ExBSFEkFINXH98NTrme5IOaaA7kIfiu2L8A3qhH9zRbukdCqdsA98TdElyeMe5BI8Rs2xHRIsoTSSVFfCFCWGPn9XHb4cdobRIWABNf0add9jakDjQJpJ1bTXOJXnnRXHRf+dNL1ZV1MBRCXhMbaHqGI1JkKIL7+i8uffuP6wVQAzO7+qVEbF6NbS0LJureYcWXUUhH66nLR5rYmva+2tjRFtojkM2aD76HEGAD3tPtKM309FJg5j/K682ywcWJ3PASCcycH/22u+Bh7Aa0ehM2Fu4z0SAE81HF9RkB21c5bEn4Dzw+/qNOyXr3DCTQDMBOdhi4nAgiFDGCinIa2owCEChUwD8qzd03PG+qdW/4fDzjUMcE1ZpIAAAAASUVORK5CYII=');
  }

  div.callout-caution.callout-style-default .callout-title {
    background-color: #ffe5d0
  }

  </style>
  <style type="text/css">
    .reveal div.sourceCode {
      margin: 0;
      overflow: auto;
    }
    .reveal div.hanging-indent {
      margin-left: 1em;
      text-indent: -1em;
    }
    .reveal .slide:not(.center) {
      height: 100%;
    }
    .reveal .slide.scrollable {
      overflow-y: auto;
    }
    .reveal .footnotes {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide .absolute {
      position: absolute;
      display: block;
    }
    .reveal .footnotes ol {
      counter-reset: ol;
      list-style-type: none; 
      margin-left: 0;
    }
    .reveal .footnotes ol li:before {
      counter-increment: ol;
      content: counter(ol) ". "; 
    }
    .reveal .footnotes ol li > p:first-child {
      display: inline-block;
    }
    .reveal .slide ul,
    .reveal .slide ol {
      margin-bottom: 0.5em;
    }
    .reveal .slide ul li,
    .reveal .slide ol li {
      margin-top: 0.4em;
      margin-bottom: 0.2em;
    }
    .reveal .slide ul[role="tablist"] li {
      margin-bottom: 0;
    }
    .reveal .slide ul li > *:first-child,
    .reveal .slide ol li > *:first-child {
      margin-block-start: 0;
    }
    .reveal .slide ul li > *:last-child,
    .reveal .slide ol li > *:last-child {
      margin-block-end: 0;
    }
    .reveal .slide .columns:nth-child(3) {
      margin-block-start: 0.8em;
    }
    .reveal blockquote {
      box-shadow: none;
    }
    .reveal .tippy-content>* {
      margin-top: 0.2em;
      margin-bottom: 0.7em;
    }
    .reveal .tippy-content>*:last-child {
      margin-bottom: 0.2em;
    }
    .reveal .slide > img.stretch.quarto-figure-center,
    .reveal .slide > img.r-stretch.quarto-figure-center {
      display: block;
      margin-left: auto;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-left,
    .reveal .slide > img.r-stretch.quarto-figure-left  {
      display: block;
      margin-left: 0;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-right,
    .reveal .slide > img.r-stretch.quarto-figure-right  {
      display: block;
      margin-left: auto;
      margin-right: 0; 
    }
  </style>
<meta property="og:title" content="Biologia della Conservazione – Biologia della Conservazione 24/25">
<meta property="og:description" content="Class 4 - Populations and conservation genetics">
<meta property="og:site_name" content="Biologia della Conservazione 24/25">
<meta property="og:locale" content="en_US">
</head>
<body class="quarto-light">
  <div class="reveal">
    <div class="slides">

<section id="title-slide" class="quarto-title-block center">
  <h1 class="title">Biologia della Conservazione</h1>
  <p class="subtitle">Class 4 - Populations and conservation genetics</p>

<div class="quarto-title-authors">
</div>

  <p class="date">2024-11-14</p>
</section>
<section id="whats-is-a-population" class="slide level2">
<h2>Whats is a population?</h2>
<blockquote>
<p>All individuals of the same species coexisting at the same time and in the same geographic area <span class="citation" data-cites="VanDyke2020">(<a href="#/references" role="doc-biblioref" onclick="">Dyke &amp; Lamb, 2020</a>)</span>.</p>
</blockquote>
<ul>
<li>This definition is strictly biological, but it could be more complex when we refer to individuals that are harvested, threatened or are perceived controversially by the public opinion</li>
<li>In the above cases, a population may be defined using political and social arguments rather than biological ones</li>
<li>For example, Michigan and Wisconsin manage their Wolf (<em>Canis lupus</em>) population independently even though they are neighboring states and even though wolf individuals, obviously, move across the boundaries as they please</li>
</ul>
</section>
<section class="slide level2">

<ul>
<li>Even if they “feel” artificial, political and social perspectives should always be considered when defining populations, as politicians are the ones that ultimately support or deny conservation policies</li>
<li>Populations mutate over time due to individuals reproducing or dying, migrating or dispersing</li>
<li><span style="color: blue;">Recruitment</span>; <span style="color: blue;">fecundity</span>; <span style="color: blue;">immigration</span>; <span style="color: blue;">emigration</span>; <span style="color: blue;">dispersal</span>!</li>
<li>We can study populations by understanding how probable each one of the above mentioned events is, and what factors influence these probabilities (<span style="color: red;">population demography</span>)</li>
<li>We distinguish primarily between intrinsic factors (characteristics and behavior of the species and the population) and extrinsic factors (environmental and ecological processes)</li>
<li>The ultimate goal of studying demographic processes is to avoid extinction! The process of extinction hardly happens over night, and is usually preceded by population decline <span class="citation" data-cites="VanDyke2020">(<a href="#/references" role="doc-biblioref" onclick="">Dyke &amp; Lamb, 2020</a>)</span></li>
</ul>
</section>
<section id="population-models" class="slide level2">
<h2>Population models</h2>
<ul>
<li>The simplest model of population dynamic can be written as the sum of births (B) and migrants entering the populations (I) minus mortality (D) and individuals leaving or exiting the population (E)</li>
</ul>
<p><span class="math display">\[
\label{popmod}
\tag{4.1}
N_{(t+1)} = N_{(t)} + (B_t + I_t) - (D_t + E_t)
\]</span></p>
<div class="fragment">
<ul>
<li>If a population is increasing exponentially (i.e., increasing with and increasing rate over time) its growth will be defined by <span class="math inline">\(N\)</span> and a rate of increase <span class="math inline">\(r\)</span> which is rate of birth minus rate of death (<span class="math inline">\(r = b-d\)</span>)</li>
<li>Without considering migration (neither in nor out of the population) <span class="math inline">\(B\)</span> becomes a function of birth rate (<span class="math inline">\(b\)</span>) and the population size <span class="math inline">\(N\)</span></li>
<li>Similarly, the number of deaths (<span class="math inline">\(D\)</span>) is a function of <span class="math inline">\(d\)</span> the per capita death rate, and population size</li>
</ul>
</div>
</section>
<section class="slide level2">


<img data-src="class_4_files/figure-revealjs/unnamed-chunk-1-1.png" class="quarto-figure quarto-figure-center r-stretch" width="960"><ul>
<li>In exponential growth change in numbers over change in time is expressed as:</li>
</ul>
<p><span class="math display">\[
\label{expgrth}
\tag{4.2}
\frac{dN}{dt} = rN
\]</span></p>
</section>
<section class="slide level2">

<ul>
<li>The exponential growth curve is a good starting point, although unrealistic for most species</li>
<li>A more realistic model should account for species’ ecology and the fact that species exist in their environment</li>
<li>The limits imposed by the environment are usually identified using <span class="math inline">\(K\)</span>, i.e.&nbsp;the carrying capacity</li>
</ul>
<p><span class="math display">\[
\label{expgrthcc}
\tag{4.3}
\frac{dN}{dt} = rN (\frac{K-N}{K})
\]</span></p>
<ul>
<li><p>With small values of <span class="math inline">\(N\)</span> relative to <span class="math inline">\(K\)</span>, <span class="math inline">\((\frac{K-N}{K}) \approx 1\)</span> and the population grows almost exponentially</p></li>
<li><p>Changes in <span class="math inline">\(N\)</span> directly influence the growth rate of the population: <span style="color: red;">density dependence</span>!</p></li>
</ul>
</section>
<section class="slide level2">


<img data-src="class_4_files/figure-revealjs/unnamed-chunk-3-1.png" class="quarto-figure quarto-figure-center r-stretch" width="960"><ul>
<li>This model assumes all individuals are demographic equivalent, an unlikely assumption in many systems, especially with long lived individuals!</li>
</ul>
</section>
<section id="population-perturbations" class="slide level2">
<h2>Population perturbations</h2>
<h3 id="density-independent-factors">Density independent factors</h3>
<ul>
<li><p>Many factors influencing population size are described as <span style="color: red;">stochastic factors</span></p></li>
<li><p>A population whose growth is only a function of stochastic variation can be described as density independent. <span class="citation" data-cites="Shaffer1981">Shaffer (<a href="#/references" role="doc-biblioref" onclick="">1981</a>)</span> recognizes 4 main sources of such stochasticity:</p>
<ul>
<li>genetic stochasticity;</li>
<li>demographic stochasticity;</li>
<li>environmental stochasticity;</li>
<li>natural catastrophes!</li>
</ul></li>
<li><p>The effects of the above sources of population variation are not predetermined, but subject to uncertainties that are more or less probable according to specific characteristics of the population analysed</p></li>
</ul>
</section>
<section class="slide level2">

<h3 id="density-dependent-factors">Density dependent factors</h3>
<ul>
<li><p>Density dependent factors can regulate the size of a population in a much more precise way than density independent ones</p></li>
<li><p>To better understand these factors it is useful to debunk the assumptions of the logistic growth curves</p>
<ul>
<li><p>Time lag in density dependence: the addition of new individuals in the population may not have an immediate effect in shaping the growth curve</p></li>
<li><p>Traditional logistic curves assume constant values of K: it is more accurate to define K not as a constant value that a population cannot exceed, but as a population size reflecting equilibrium between a population and its resources created by their interaction</p></li>
<li><p>Individuals in a population are not contributing equally to the population growth: age and sex structure play a pivotal role</p></li>
</ul></li>
</ul>
</section>
<section id="how-do-we-estimate-the-size-of-a-population" class="slide level2">
<h2>How do we estimate the size of a population?</h2>
<ul>
<li>It is often impractical and/or impossible to count every single individual in a population</li>
</ul>
<div class="fragment">
<ul>
<li>Capture-Mark-Recapture experiments allow to estimate a population’s size</li>
</ul>
</div>
<div class="fragment">
<ul>
<li>Some individuals are captured and marked</li>
<li>The marked individuals are released in their environment</li>
<li>A second sampling trip is conducted and another portion of the individuals are collected</li>
<li>The number of marked individuals collected in the second expedition is expected to be proportional to the number of marked individuals in the entire population</li>
<li>It is important to have sufficient time between sampling occasions, to allow for the marked individuals to redistribute themselves among the unmarked population</li>
</ul>
</div>
</section>
<section id="tagging-systems" class="slide level2">
<h2>Tagging systems</h2>
<div class="columns">
<div class="column" style="width:50%;">
<ul>
<li>A tagging system is a system allowing us to tag and label things to provide those things a unique identifier</li>
<li>People have been tagging animals (wild and captive) for hundreds of years and have developed different ways to tag pretty much every single living things</li>
<li>An animal tag can be as simple as a piece of metal or plastic with a number carved on it or with a standardized meaning</li>
</ul>
</div><div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_4_1.png"></p>
<figcaption>A male <em>Cyclura carinata</em> with colored bead tags. Photo credits: Giuliano Colosimo.</figcaption>
</figure>
</div>
</div></div>
</section>
<section id="passive-integrated-transponders" class="slide level2">
<h2>Passive integrated transponders</h2>
<div class="columns">
<div class="column" style="width:50%;">
<ul>
<li>More sophisticated tags based on radio-frequency identification (RFID)</li>
<li>When the tag is within reach of a specific radio frequency, it emits a signal containing an alpha-numeric code unique to the animal it has been implanted</li>
<li>Widely used for wildlife fauna, especially in long-term capture-mark-recapture studies</li>
<li>They generally do not carry any other information but the individual’s ID</li>
</ul>
</div><div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_4_2.png"></p>
<figcaption>A Passive Integrated Transponder by Trovan®. Photo credits: Giuliano Colosimo.</figcaption>
</figure>
</div>
</div></div>
</section>
<section id="patterns-tagging" class="slide level2">
<h2>Patterns tagging</h2>
<div class="columns">
<div class="column" style="width:50%;">
<h4 id="toes-and-nail-clipping">Toes and nail clipping</h4>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_4_3.png"></p>
<figcaption>This image is from <span class="citation" data-cites="Waichman1992">Waichman (<a href="#/references" role="doc-biblioref" onclick="">1992</a>)</span>. The author describes all possible combination of toe clipping to uniquely identify individuals</figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<h4 id="features-tag">Features tag</h4>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_4_4.png"></p>
<figcaption>Toads of the genus <em>Bombina</em> have unique spotted patterns. Photo credits: <a href="https://www.naturamediterraneo.com/forum/topic.asp?TOPIC_ID=273624">marioeffe</a></figcaption>
</figure>
</div>
</div></div>
</section>
<section id="photo-tagging" class="slide level2">
<h2>Photo tagging</h2>

<img data-src="./class_4_5.png" class="r-stretch quarto-figure-center"><p class="caption">Development of a photo recognition software to identify iguanas in the wild. Image courtesy of Nathalie Duporge.</p></section>
<section id="satellite-tags" class="slide level2">
<h2>Satellite tags</h2>
<ul>
<li>More complex electronic devices that not only identify an individual, but also provide useful information about that animal and its environment <span class="citation" data-cites="Loreti2019 Loreti2020 Colosimo2022b">(<a href="#/references" role="doc-biblioref" onclick="">Colosimo et al., 2022</a>; <a href="#/references" role="doc-biblioref" onclick="">Loreti et al., 2019</a>, <a href="#/references" role="doc-biblioref" onclick="">2020</a>)</span></li>
</ul>

<img data-src="./class_4_6.png" class="r-stretch quarto-figure-center"><p class="caption">A male <em>Conolophus marthae</em> with a Wireless Sensor Node (WSN) attached behind its hindlegs. This is a more sofisticated electronic tag that not only allows to identify the individual but also allows to collect important information such as GPS data, temperature and UV light. Photo credits: Giuliano Colosimo.</p></section>
<section id="estimates-of-census-population-size" class="slide level2">
<h2>Estimates of census population size</h2>
<p>A conservationist is working on a critical endangered species and needs to estimate how many individuals are left. He/she goes to the field and captures 15 individuals. These individuals are tagged and released on site. One week later the researcher comes back and catches 20 individuals, 10 of which are tagged</p>
<p>Lets define some variables as:</p>
<ul>
<li><span class="math inline">\(N\)</span> = Number of animals in the population</li>
<li><span class="math inline">\(n\)</span> = Number of animals marked on the first visit</li>
<li><span class="math inline">\(K\)</span> = Number of animals captured on the second visit</li>
<li><span class="math inline">\(k\)</span> = Number of animals captured in the second visit that were marked (i.e., recaptured individuals)</li>
</ul>
<p>We want to use the data we collected in the field to have an estimate of <span class="math inline">\(N\)</span></p>
</section>
<section id="lincoln-peterson-estimator-of-n-seber1982" class="slide level2">
<h2>Lincoln-Peterson estimator of N <span class="citation" data-cites="Seber1982">(<a href="#/references" role="doc-biblioref" onclick="">Seber, 1982</a>)</span></h2>
<div class="columns">
<div class="column" style="width:50%;">
<ul>
<li>Useful with only two sampling events</li>
<li>Assumes that <span class="math inline">\(N\)</span> is constant over the time of the experiment (i.e., there are no deaths or births)</li>
<li>Assumes a closed population (i.e., there are no migrants)</li>
<li>Catchability is the same among individuals, even the tagged ones</li>
<li>Tags are not lost between sampling events</li>
<li>Individuals have the time to randomly redistribute over their habitat</li>
</ul>
</div><div class="column" style="width:50%;">
<ul>
<li><span class="math inline">\(N\)</span> = Number of animals in the population</li>
<li><span class="math inline">\(n\)</span> = Number of animals marked on the first visit</li>
<li><span class="math inline">\(K\)</span> = Number of animals captured on the second visit</li>
<li><span class="math inline">\(k\)</span> = Number of animals captured in the second visit that were marked (i.e., recaptured individuals)</li>
</ul>
<p><span class="math display">\[
\label{lp}
\tag{4.4}
N=\frac{K*n}{k}
\]</span></p>
</div></div>
</section>
<section id="schnable-estimator-of-n" class="slide level2">
<h2>Schnable estimator of <span class="math inline">\(N\)</span></h2>
<div class="columns">
<div class="column" style="width:50%;">
<ul>
<li>Extension of the LP estimator</li>
<li>Same assumptions of LP</li>
<li>Allows to incorporate <span class="math inline">\(s\)</span> sampling events in the calculation</li>
<li>It depends simply on observing how the proportion of marked animals in catches increases as more animals have been marked <span class="citation" data-cites="Sutherland1996">(<a href="#/references" role="doc-biblioref" onclick="">Sutherland, 2006</a>)</span></li>
</ul>
</div><div class="column" style="width:50%;">
<ul>
<li><span class="math inline">\(N\)</span> = Number of animals in the population</li>
<li><span class="math inline">\(n_i\)</span> = Number of animals captured on <span class="math inline">\(i^{th}\)</span> visit</li>
<li><span class="math inline">\(m_i\)</span> = Number of animals marked on <span class="math inline">\(i^{th}\)</span> visit</li>
<li><span class="math inline">\(k_i\)</span> = Number of animals captured on <span class="math inline">\(i^{th}\)</span> visit that were marked (i.e., recaptured individuals)</li>
<li><span class="math inline">\(s\)</span> = Number of visits</li>
</ul>
<p><span class="math display">\[
\label{schnable}
\tag{4.5}
N=\frac{\sum_{i=1}^s  n_i*m_i}{\sum_{i=1}^s k_i}
\]</span></p>
</div></div>
</section>
<section id="jolly-seber" class="slide level2">
<h2>Jolly-Seber</h2>
<ul>
<li><p>Jolly-Seber <span class="citation" data-cites="Jolly1965 Seber1982">(<a href="#/references" role="doc-biblioref" onclick="">Jolly, 1965</a>; <a href="#/references" role="doc-biblioref" onclick="">Seber, 1982</a>)</span> is the method of election to estimate N starting from multiple sampling occasions</p></li>
<li><p>When working with multiple sampling occasions we first need to work out some numbers that will be useful to estimate the quantity of interest</p></li>
<li><p>The following set of four numbers are needed in order to begin the calculation</p>
<ul>
<li><span class="math inline">\(n_i\)</span> = total number of animals caught in the <span class="math inline">\(i^{th}\)</span> sample</li>
<li><span class="math inline">\(R_i\)</span> = number of animals that are released after the <span class="math inline">\(i^{th}\)</span> sample</li>
<li><span class="math inline">\(m_i\)</span> = number of animals in the <span class="math inline">\(i^{th}\)</span> sample that carry marks from previous captures</li>
<li><span class="math inline">\(m_{ij}\)</span> = number of animals in the <span class="math inline">\(i^{th}\)</span> sample that were most recently caught in the <span class="math inline">\(j^{th}\)</span> sample</li>
</ul></li>
</ul>
</section>
<section class="slide level2">


<img data-src="./js_1.png" class="r-stretch quarto-figure-center"><p class="caption">Capture history of sampled individuals. After <span class="citation" data-cites="Sutherland1996">Sutherland (<a href="#/references" role="doc-biblioref" onclick="">2006</a>)</span>, p.&nbsp;121</p></section>
<section class="slide level2">


<img data-src="./js_2.png" class="r-stretch quarto-figure-center"><p class="caption">Summary of capture data from previous table. After <span class="citation" data-cites="Sutherland1996">Sutherland (<a href="#/references" role="doc-biblioref" onclick="">2006</a>)</span>, p.&nbsp;121</p></section>
<section class="slide level2">

<ul>
<li>Having worked out the numbers with various capture histories and used these to produce the four sets of key numbers</li>
<li><span class="math inline">\(z_i\)</span> = the number of animals caught both before and after the <span class="math inline">\(i^{th}\)</span> sample but not in the <span class="math inline">\(i^{th}\)</span> sample itself</li>
<li><span class="math inline">\(r_i\)</span> = the number of animals that were released from the <span class="math inline">\(i^{th}\)</span> sample and were subsequently recaptured <span class="citation" data-cites="Sutherland1996">(<a href="#/references" role="doc-biblioref" onclick="">Sutherland, 2006</a>)</span></li>
</ul>

<img data-src="./js_3.png" class="r-stretch quarto-figure-center"><p class="caption">Capture history data sourced from <span class="citation" data-cites="Jolly1965">Jolly (<a href="#/references" role="doc-biblioref" onclick="">1965</a>)</span></p></section>
<section class="slide level2">

<ul>
<li><span class="math inline">\(M_i\)</span> = number of marked animals in the population when the <span class="math inline">\(i^{th}\)</span> sample is taken (but not including animals newly marked in the <span class="math inline">\(i^{th}\)</span> sample)</li>
</ul>
<p><span class="math display">\[
    \label{mi}
    \tag{4.6}
    m_1 + (R_i + 1)z_i /(r_i + 1)
\]</span></p>
<ul>
<li><span class="math inline">\(N_i\)</span> = population size at the time of the <span class="math inline">\(i^{th}\)</span> sample</li>
</ul>
<p><span class="math display">\[
    \label{ni}
    \tag{4.7}
    M_i (n_i + 1)/(m_i + 1)
    \]</span></p>
<ul>
<li><span class="math inline">\(\phi_i\)</span> = proportion of the population surviving (and remaining in the study area) from the <span class="math inline">\(i^{th}\)</span> sampling occasion to the <span class="math inline">\((i + 1)^{th}\)</span></li>
</ul>
<p><span class="math display">\[
\label{fi}
\tag{4.8}
M_{i+1}/(M_i − m_i + R_1)
\]</span></p>
</section>
<section class="slide level2">

<ul>
<li><span class="math inline">\(B_i\)</span> = number of animals that enter the population between the <span class="math inline">\(i^{th}\)</span> and <span class="math inline">\((i + 1)^{th}\)</span> samples and survive until the <span class="math inline">\((i + 1)^{th}\)</span> sampling occasion</li>
</ul>
<p><span class="math display">\[
    \label{bi}
    \tag{4.9}
    N_{i+1} − \phi_i (N_i − n_i + R_i)
\]</span></p>
<ul>
<li><p><span class="math inline">\(M_2 = 10 + (143 + 1)14/(80 + 1) = 34.89\)</span></p></li>
<li><p><span class="math inline">\(N_2 = 34.89(146 + 1)/(10 + 1) = 466.1\)</span></p></li>
<li><p><span class="math inline">\(\phi_2 = 169.46/(34.89 − 10 + 143) = 1.009\)</span></p></li>
<li><p><span class="math inline">\(B_2 = 758.1 − 1.009(466.2 − 146 + 143) = 290.7\)</span></p></li>
</ul>
<p><br></p>
<ul>
<li>Note that one cannot calculate <span class="math inline">\(M\)</span> for the last sample, <span class="math inline">\(N\)</span> for the first or last, <span class="math inline">\(\phi\)</span> for the last two and <span class="math inline">\(B\)</span> for the first or last two. <span class="math inline">\(M_1\)</span> is always zero</li>
</ul>
</section>
<section id="population-genetics" class="slide level2">
<h2>Population genetics</h2>
<ul>
<li><p>Genetics (and genomic) have an important role in conservation biology and towards the protection of biodiversity</p></li>
<li><p>Darwin was the first to consider the importance of genetics and evolution in the persistence of natural populations. He postulated that because of the reduced population size of deer in British natural reserves they may experience loss of vigor <span class="citation" data-cites="Darwin1896">(<a href="#/references" role="doc-biblioref" onclick="">Darwin, 1896</a>)</span></p></li>
<li><p>The modern concern for genetics in conservation began in the 1970s when Frankel began to raise the alarm about the loss of primitive crop varieties and their replacement by genetically uniform cultivars <span class="citation" data-cites="Frankel1970 Frankel1974">(<a href="#/references" role="doc-biblioref" onclick="">Frankel, 1974</a>; <a href="#/references" role="doc-biblioref" onclick="">Frankel, Bennett, Food and Agriculture Organization of the United Nations, &amp; International Biological Programme, 1970</a>)</span></p></li>
</ul>
</section>
<section class="slide level2">

<ul>
<li><p>Most articles dealing with conservation and genetics fit into one of the five general categories <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>:</p>
<ol type="1">
<li><p>Management and reintroduction of captive populations, and the restoration of biological communities</p></li>
<li><p>Description and identification of individuals, genetic population structure, kin relationships, and taxonomic relationships</p></li>
<li><p>Detection and prediction of the effects of habitat loss, fragmentation, isolation, and genetic rescue</p></li>
<li><p>Detection and prediction of the effects of hybridization and introgression</p></li>
<li><p>Understanding the relationships between fitness of individuals or local adaptation of populations and environmental factors</p></li>
</ol></li>
</ul>
</section>
<section class="slide level2">

<ul>
<li>As in other sciences, conservation genetics has benefited from the use of model organisms for experiments as well as modelistic and numerical approaches</li>
<li>How much gene flow is required to prevent the inbreeding effects of small population size?
<ul>
<li>We can use species with short generation time and easy to control (<em>Drosophila</em>, peas, <em>Arabidopsis thaliana</em>) to make empirical experiments and answer the above question</li>
</ul></li>
<li>Describing individuals from a molecular genetic standpoint can also help to better understand the basic biology of species and populations <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>
<ul>
<li>Total population size can be estimated from the genotypes in populations that are difficult to sample with the classic methodologies <span class="citation" data-cites="Luikart2010">(<a href="#/references" role="doc-biblioref" onclick="">Luikart, Ryman, Tallmon, Schwartz, &amp; Allendorf, 2010</a>)</span></li>
</ul></li>
<li>Genetic analysis can help detecting cryptic effects of climate change on the distribution of species <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>
<ul>
<li><span class="citation" data-cites="Gurgel2020">Gurgel, Camacho, Minne, Wernberg, &amp; Coleman (<a href="#/references" role="doc-biblioref" onclick="">2020</a>)</span> documented the reduction in genetic diversity after a heatwave despite the sea weed population had recovered</li>
</ul></li>
</ul>
</section>
<section class="slide level2">


<img data-src="class_4_files/figure-revealjs/genVSgenom-1.png" width="960" class="r-stretch quarto-figure-center"><p class="caption">Interacting factors in conservation of natural populations. Difference between factors that could be adressed with traditional conservation genetics tools (neutral markers; in blue) and genomic approach (red). Adapted from <span class="citation" data-cites="Allendorf2022">Allendorf et al. (<a href="#/references" role="doc-biblioref" onclick="">2022</a>)</span></p></section>
<section id="what-should-we-conserve" class="slide level2">
<h2>What should we conserve?</h2>
<ul>
<li>In 1990 IUCN identified genetic diversity as one of the three most important element of conservation biology (Genetic diversity, Species diversity, Ecosystem diversity)</li>
</ul>
<div class="panel-tabset">
<ul id="tabset-1" class="panel-tabset-tabby"><li><a data-tabby-default="" href="#tabset-1-1">Phylogenetic diversity</a></li><li><a href="#tabset-1-2">Species or ecosystems</a></li><li><a href="#tabset-1-3">Populations or species</a></li></ul>
<div class="tab-content">
<div id="tabset-1-1">
<ul>
<li>A monotypic genus receives higher priority <span class="citation" data-cites="USWF1983">(<a href="#/references" role="doc-biblioref" onclick="">USA Fish and Wildlife Services, 1983</a>)</span>. The Tuatara case from New Zealand is paradigmatic <span class="citation" data-cites="Daugherty1990 Gemmell2020">(<a href="#/references" role="doc-biblioref" onclick="">Daugherty, Cree, Hay, &amp; Thompson, 1990</a>; <a href="#/references" role="doc-biblioref" onclick="">Gemmell et al., 2020</a>)</span></li>
<li><span class="citation" data-cites="VaneWright1991">Vane-Wright, Humphries, &amp; Williams (<a href="#/references" role="doc-biblioref" onclick="">1991</a>)</span> developed a method to assign a conservation value to distinct phylogenetic lineages. This value is inversely proportional to the number of branching points to other extant lineages</li>
</ul>
</div>
<div id="tabset-1-2">
<blockquote>
<p>Conservation requires a balanced approach based on habitat protection that also takes into account the natural history and viability of individual species. This is particularly challenging for wide-ranging species <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</p>
</blockquote>
<ul>
<li>The case of Chinook salmon and killer whales <span class="citation" data-cites="Groskreutz2019">(<a href="#/references" role="doc-biblioref" onclick="">Groskreutz et al., 2019</a>)</span></li>
</ul>
</div>
<div id="tabset-1-3">
<ul>
<li>Genetically distinct populations should probably receive more attention than species from a conservation perspective</li>
<li>The functional units of an ecosystem are populations</li>
<li>Experiments conducted in the field of community genetics hint towards a correlation between individual alleles within species and community composition and diversity <span class="citation" data-cites="Crutsinger2016">(<a href="#/references" role="doc-biblioref" onclick="">Crutsinger, 2016</a>)</span></li>
<li>Genetic variation of a Tasmanian blue gum tree affects the distribution of insects, birds, and marsupials <span class="citation" data-cites="Whitham2008">(<a href="#/references" role="doc-biblioref" onclick="">Whitham et al., 2008</a>)</span></li>
</ul>
</div>
</div>
</div>
</section>
<section class="slide level2">

<ul>
<li>Conservation genetics ultimately looks for and tries to maintain the <span style="color: red;">allelic diversity</span> within a population</li>
</ul>
<div class="callout callout-important callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>Question</strong></p>
</div>
<div class="callout-content">
<p>What are alleles?</p>
</div>
</div>
</div>
<div class="fragment">
<div class="callout callout-tip callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>Answer</strong></p>
</div>
<div class="callout-content">
<p>Alleles are different forms of the same gene! The number of alleles for a certain gene in a population is a measure of the genetic diversity of that specific population</p>
</div>
</div>
</div>
</div>
<div class="fragment">
<ul>
<li>Another goal of conservation genetics is that to prevent the fixation of alleles! When an allele is fixed (i.e., its frequency is 1) it means that:
<ul>
<li>there is no more variation at that specific gene</li>
<li>the population could experience fitness loss if the fixed allele is deleterious and it is found together with other deleterious mutations</li>
</ul></li>
</ul>
</div>
</section>
<section id="fundamental-vocabulary" class="slide level2">
<h2>Fundamental vocabulary</h2>
<table class="caption-top">
<colgroup>
<col style="width: 25%">
<col style="width: 75%">
</colgroup>
<thead>
<tr class="header">
<th>Term</th>
<th>Definition</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Locus</td>
<td>The term refers both to the position on the chromosome where a specific gene is found, and, in general, to any genetic segment, even a single nucleotide base, of interest. An example of a locus could be the cytochrome oxidase sub-unit I gene (generally called COI), or the nucleotide base at position 350 on a specific sequence.</td>
</tr>
<tr class="even">
<td>Allele</td>
<td>Variant of a specific locus on homologous chromosomes (containing the same type of genetic information).</td>
</tr>
<tr class="odd">
<td>Genotype</td>
<td>Combination of alleles at one or more loci (genes) in an individual. For example, the genotype of a diploid individual characterized for the bi-allelic gene A might be A1A1, A1A2, or A2A2.</td>
</tr>
</tbody>
</table>
</section>
<section class="slide level2">

<table class="caption-top">
<colgroup>
<col style="width: 26%">
<col style="width: 73%">
</colgroup>
<thead>
<tr class="header">
<th></th>
<th>(continues)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Phenotype</td>
<td>The obvious and measurable manifestation of a specific genotype.</td>
</tr>
<tr class="even">
<td>Genome</td>
<td>The set of nucleotide sequences, coding and not, generally organized into chromosomes, which characterize a species or an individual.</td>
</tr>
<tr class="odd">
<td>Homozygote</td>
<td>Individual with two identical copies of a specific allele registered at the locus (gene) of interest.</td>
</tr>
<tr class="even">
<td>Heterozygote</td>
<td>Individual with two different alleles characterized at the locus (gene) of interest.</td>
</tr>
<tr class="odd">
<td>Allele frequency</td>
<td>The frequency of a specific allele in a population. The term gene frequency is often used to indicate the same concept.</td>
</tr>
</tbody>
</table>
</section>
<section class="slide level2">

<table class="caption-top">
<colgroup>
<col style="width: 26%">
<col style="width: 73%">
</colgroup>
<thead>
<tr class="header">
<th></th>
<th>(continues)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Polymorphic</td>
<td>The presence in a species or population of 2 or more alleles at a certain gene. For example, gene A in a population might have 4 alleles: A1, A2, A3, and A4. A locus is considered polymorphic if the frequency of its most frequent allele is less than 0.99 or 0.95.</td>
</tr>
<tr class="even">
<td>Monomorphic</td>
<td>A population or species is monomorphic for a gene when all individuals have the same allele. In other words, the frequency of that allele in the population is 1.</td>
</tr>
<tr class="odd">
<td>Average heterozygosity</td>
<td>Average number of heterozygous loci in an individual or population.</td>
</tr>
<tr class="even">
<td>Co-dominance</td>
<td>Situation where all genotypes can be identified by their phenotypes. In the case of gene A with two codominant alleles, the A1A1, A1A2, and A2A2 genotypes all produce different and distinguishable phenotypes.</td>
</tr>
</tbody>
</table>
</section>
<section class="slide level2">

<table class="caption-top">
<colgroup>
<col style="width: 26%">
<col style="width: 73%">
</colgroup>
<thead>
<tr class="header">
<th></th>
<th>(continues)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Allelic diversity</td>
<td>Mean number of alleles per locus.</td>
</tr>
<tr class="even">
<td>Genetic distance</td>
<td>A measure of distance obtained by comparing 2 or more nucleotide sequences or by comparing the allele and genotype frequencies of two or more individuals or populations.</td>
</tr>
</tbody>
</table>
</section>
<section id="molecular-techniques" class="slide level2">
<h2>Molecular techniques</h2>

<img data-src="./molmarkers.png" class="r-stretch quarto-figure-center"><p class="caption">Figure reproduced from <span class="citation" data-cites="Allendorf2022">Allendorf et al. (<a href="#/references" role="doc-biblioref" onclick="">2022</a>)</span></p></section>
<section id="fundamental-measures-of-genetic-diversity" class="slide level2">
<h2>Fundamental measures of genetic diversity</h2>
<ul>
<li>Polymorphism</li>
</ul>
<p><span class="math display">\[
P = number\ of\ polymorphic\ loci/total\ number\ of\ loci
\]</span></p>
<ul>
<li>Allele frequency (for diploid individuals)</li>
</ul>
<p><span class="math display">\[
p = \frac{2n_{Hom} + n_{Het}}{2n}
\]</span> where <span class="math inline">\(p\)</span> is the frequency of the allele (usually the dominant one) and <span class="math inline">\(n\)</span> is the number of individuals that are homozygotes (<span class="math inline">\(n_{Hom}\)</span>) and heterozygotes (<span class="math inline">\(n_{Het}\)</span>) for that allele, respectively</p>
<ul>
<li>Average heterozygosity <span class="math inline">\(H = \sum H_i/N\)</span> where <span class="math inline">\(H_i\)</span> is the average heterozygosity at locus <span class="math inline">\(i\)</span>, and <span class="math inline">\(N\)</span> is the total number of loci</li>
</ul>
</section>
<section class="slide level2">

<ul>
<li>Describing the genetic variation using mathematical tool is a good and necessary exercise, but it is not enough!</li>
<li>We need to develop models that could tell us about how various evolutionary factors (natural selection, mutations, population size, geographic fragmentation, drift) affect the persistence of populations and species</li>
</ul>
<blockquote>
<p>Models are essential for understanding the genetics of natural populations in a variety of ways: <em>i)</em> models make us define the parameters that need to be considered; <em>ii)</em> models allow us to test hypotheses; <em>iii)</em> models allow us to generalize results; iv) models allow us to predict how a system will operate in the future <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022, p. 95</a>)</span>.</p>
</blockquote>
<ul>
<li>As a general rule of thumb, models that we develop to understand natural populations should be as simple as possible
<ul>
<li>Simpler models are easier to reject, and, therefore, are more useful</li>
<li>Simple models are likely to be more general and therefore more applicable to a wider number of situations <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022, p. 95</a>)</span></li>
</ul></li>
</ul>
</section>
<section id="the-simpler-model-hardy-weinberg-principle" class="slide level2">
<h2>The simpler model: Hardy-Weinberg principle</h2>
<ul>
<li>In 1908, Godfrey Harold Hardy (a British mathematician) and Wilhelm Weinberg (a German physician), independently developed a math model describing the dynamics of a diploid, sexually reproducing population</li>
</ul>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./hardy.jpg"></p>
<figcaption>Gregory Harol Hardy, figure from the Wiki page</figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./weinberg.jpg"></p>
<figcaption>Wilhelm Weinberg, figure from <a href="http://www.esp.org/people/" class="uri">http://www.esp.org/people/</a></figcaption>
</figure>
</div>
</div></div>
</section>
<section class="slide level2">

<ul>
<li>Hardy-Weinberg model suggests that a population reaches an equilibrium where allele frequencies are constant and there is a specific relation between genotypic and allelic frequencies</li>
</ul>
<div class="fragment">
<p>Assumptions of the model:</p>
<table class="caption-top">
<colgroup>
<col style="width: 26%">
<col style="width: 73%">
</colgroup>
<thead>
<tr class="header">
<th>Assumption</th>
<th>Significance</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Random mating</td>
<td>Individuals show no significant preference or dislike for other individuals with respect to specific traits or loci.</td>
</tr>
<tr class="even">
<td>No mutation</td>
<td>Mutation is one of the few forces creating variability. We assume that the information is transmitted from generation to generation without the influence of mutation.</td>
</tr>
<tr class="odd">
<td>Infinite population size</td>
<td>No population in nature is actually infinite (although populations of some species may as well be)! This assumption is fundamental to construct a null model and compare the effect of small population size.</td>
</tr>
</tbody>
</table>
</div>
</section>
<section class="slide level2">

<table class="caption-top">
<colgroup>
<col style="width: 26%">
<col style="width: 73%">
</colgroup>
<thead>
<tr class="header">
<th></th>
<th>(continues)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>No natural selection</td>
<td>We will assume that all alleles and genotypes have the same fitness and that there is no differential reproduction or survival. This assumption is very unlikely to be true in nature, especially because selection may be reduced for the loci under analysis, but still present at all other loci in an individual.</td>
</tr>
<tr class="even">
<td>No immigration</td>
<td>We will first consider a situation where only one population of a certain species is present. This means that we will not consider gene flow and exchange of genetic material from and to other populations.</td>
</tr>
</tbody>
</table>
<div class="fragment">
<div class="callout callout-important callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>Question</strong></p>
</div>
<div class="callout-content">
<p>What is the main consequence of the assumptions we have just described?</p>
</div>
</div>
</div>
</div>
</section>
<section class="slide level2">


<img data-src="class_4_files/figure-revealjs/hwProps-1.png" width="960" class="r-stretch quarto-figure-center"><p class="caption">Organization of genetic variability in a Hardy-Weinberg equilibrium population typed at a biallelic locus</p></section>
<section id="references" class="slide level2 smaller scrollable">
<h2>References</h2>

<div class="quarto-auto-generated-content">
<p><img src="../../figures/Tor-Vergata_logo_cmyk.png" class="slide-logo"></p>
<div class="footer footer-default">
<p><a href="https://giulianocolosimo.github.io/biologia_della_conservazione_24-25/">Biologia della conservazione 24/25 - Giuliano Colosimo, PhD</a></p>
</div>
</div>
<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" data-line-spacing="2" role="list">
<div id="ref-Allendorf2022" class="csl-entry" role="listitem">
Allendorf, F. W., Funk, W. C., Aitken, S. N., Byrne, M., Luikart, G., &amp; Antunes, A. (2022). <em><span class="nocase">Conservation and the Genomics of Populations</span></em>. Oxford University Press. Retrieved from <a href="https://doi.org/10.1093/oso/9780198856566.001.0001">https://doi.org/10.1093/oso/9780198856566.001.0001</a>
</div>
<div id="ref-Colosimo2022b" class="csl-entry" role="listitem">
Colosimo, G., Gargano, M., Loreti, P., Bracciale, L., De Luca, M., Catini, A., Di Natale, C., et al. (2022). Remote tracking of gal<span>á</span>pagos pink land iguana reveals large elevational shifts in habitat use. <em>Journal for Nature Conservation</em>, <em>68</em>, 126210. Retrieved from <a href="https://www.sciencedirect.com/science/article/pii/S1617138122000838">https://www.sciencedirect.com/science/article/pii/S1617138122000838</a>
</div>
<div id="ref-Crutsinger2016" class="csl-entry" role="listitem">
Crutsinger, G. M. (2016). A community genetics perspective: Opportunities for the coming decade. <em>New Phytologist</em>, <em>210</em>(1), 65–70. Retrieved from <a href="https://nph.onlinelibrary.wiley.com/doi/abs/10.1111/nph.13537">https://nph.onlinelibrary.wiley.com/doi/abs/10.1111/nph.13537</a>
</div>
<div id="ref-Darwin1896" class="csl-entry" role="listitem">
Darwin, C. (1896). <em>The variation of animals and plants under domestication</em> (Vol. II). New York, USA: Appleton; Co.
</div>
<div id="ref-Daugherty1990" class="csl-entry" role="listitem">
Daugherty, C. H., Cree, A., Hay, J. M., &amp; Thompson, M. B. (1990). Neglected taxonomy and continuing extinctions of tuatara (sphenodon). <em>Nature</em>, <em>347</em>(6289), 177–179. Springer Science; Business Media <span>LLC</span>. Retrieved from <a href="https://doi.org/10.1038%2F347177a0">https://doi.org/10.1038%2F347177a0</a>
</div>
<div id="ref-VanDyke2020" class="csl-entry" role="listitem">
Dyke, F. V., &amp; Lamb, R. L. (2020). <em>Conservation biology</em>. Springer International Publishing. Retrieved from <a href="https://doi.org/10.1007%2F978-3-030-39534-6">https://doi.org/10.1007%2F978-3-030-39534-6</a>
</div>
<div id="ref-Frankel1974" class="csl-entry" role="listitem">
Frankel, O. H. (1974). <a href="https://doi.org/Genetic Considerations in Ecological Restoration"><span class="nocase">Genetic conservation: our evolutionary responsibility</span></a>. <em>Genetics</em>, <em>78</em>(1), 53–65.
</div>
<div id="ref-Frankel1970" class="csl-entry" role="listitem">
Frankel, O. H., Bennett, E., Food and Agriculture Organization of the United Nations, &amp; International Biological Programme. (1970). <em>Genetic resources in plants: Their exploration and conservation</em>. IBP handbook no. 11. Blackwell Scientific. Retrieved from <a href="https://books.google.it/books?id=7cocAQAAIAAJ">https://books.google.it/books?id=7cocAQAAIAAJ</a>
</div>
<div id="ref-Gemmell2020" class="csl-entry" role="listitem">
Gemmell, N. J., Rutherford, K., Prost, S., Tollis, M., Winter, D., Macey, J. R., Adelson, D. L., et al. (2020). The tuatara genome reveals ancient features of amniote evolution. <em>Nature</em>, <em>584</em>(7821), 403–409. Springer Science; Business Media <span>LLC</span>. Retrieved from <a href="https://doi.org/10.1038%2Fs41586-020-2561-9">https://doi.org/10.1038%2Fs41586-020-2561-9</a>
</div>
<div id="ref-Groskreutz2019" class="csl-entry" role="listitem">
Groskreutz, M., Durban, J., Fearnbach, H., Barrett-Lennard, L., Towers, J., &amp; Ford, J. (2019). <a href="https://doi.org/10.3354/esr00993">Decadal changes in adult size of salmon-eating killer whales in the eastern north pacific</a>. <em>Endangered Species Research</em>, <em>40</em>.
</div>
<div id="ref-Gurgel2020" class="csl-entry" role="listitem">
Gurgel, C. F. D., Camacho, O., Minne, A. J. P., Wernberg, T., &amp; Coleman, M. A. (2020). Marine heatwave drives cryptic loss of genetic diversity in underwater forests. <em>Current Biology</em>, <em>30</em>(7), 1199–1206.e2. Retrieved from <a href="https://www.sciencedirect.com/science/article/pii/S0960982220300956">https://www.sciencedirect.com/science/article/pii/S0960982220300956</a>
</div>
<div id="ref-Jolly1965" class="csl-entry" role="listitem">
Jolly, G. M. (1965). Explicit estimates from capture-recapture data with both death and immigration-stochastic model. <em>Biometrika</em>, <em>52</em>(1/2), 225–247. [Oxford University Press, Biometrika Trust]. Retrieved April 28, 2023, from <a href="http://www.jstor.org/stable/2333826">http://www.jstor.org/stable/2333826</a>
</div>
<div id="ref-Loreti2020" class="csl-entry" role="listitem">
Loreti, P., Bracciale, L., Colosimo, G., Vera, C., Gerber, G. P., De Luca, M., &amp; Gentile, G. (2020). <span class="nocase">Assessment and validation of miniaturized technology for the remote tracking of critically endangered Gal<span class="nocase">á</span>pagos pink land iguana (<em>Conolophus marthae</em>)</span>. <em>Animal Biotelemetry</em>, <em>8</em>(1), 3. BioMed Central. Retrieved from <a href="https://doi.org/10.1186/s40317-020-0192-4">https://doi.org/10.1186/s40317-020-0192-4</a>
</div>
<div id="ref-Loreti2019" class="csl-entry" role="listitem">
Loreti, P., Catini, A., De Luca, M., Bracciale, L., Gentile, G., &amp; Di Natale, C. (2019). <span class="nocase">The Design of an Energy Harvesting Wireless Sensor Node for Tracking Pink Iguanas</span>. <em>Sensors</em>, <em>19</em>(5), 985. Retrieved from <a href="http://www.mdpi.com/1424-8220/19/5/985">http://www.mdpi.com/1424-8220/19/5/985</a>
</div>
<div id="ref-Luikart2010" class="csl-entry" role="listitem">
Luikart, G., Ryman, N., Tallmon, D. A., Schwartz, M. K., &amp; Allendorf, F. W. (2010). Estimation of census and effective population sizes: The increasing usefulness of DNA-based approaches. <em>Conservation Genetics</em>, <em>11</em>(2), 355–373. Retrieved from <a href="https://link.springer.com/content/pdf/10.1007/s10592-010-0050-7.pdf">https://link.springer.com/content/pdf/10.1007/s10592-010-0050-7.pdf</a>
</div>
<div id="ref-Seber1982" class="csl-entry" role="listitem">
Seber, G. A. F. (1982). <em>The estimation of animal abundance and related parameters.</em> <em>Biometrische Zeitschrift</em> (Vol. 16, pp. 80–80). Caldwel, New Jersey: Blackburn Press. Retrieved from <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/bimj.19740160115">https://onlinelibrary.wiley.com/doi/abs/10.1002/bimj.19740160115</a>
</div>
<div id="ref-Shaffer1981" class="csl-entry" role="listitem">
Shaffer, M. L. (1981). <a href="https://doi.org/10.2307/1308256"><span class="nocase">Minimum Population Sizes for Species Conservation</span></a>. <em>BioScience</em>, <em>31</em>(2), 131–134.
</div>
<div id="ref-Sutherland1996" class="csl-entry" role="listitem">
Sutherland, W. J. (Ed.). (2006). <em>Ecological census techniques: A handbook</em> (2nd ed.). Cambridge University Press.
</div>
<div id="ref-USWF1983" class="csl-entry" role="listitem">
USA Fish and Wildlife Services. (1983). Endangered and threatened species listing and recovery priorities guidelines. <em>Federal Register</em>, <em>48</em>(184), 43098–43105.
</div>
<div id="ref-VaneWright1991" class="csl-entry" role="listitem">
Vane-Wright, R. I., Humphries, C. J., &amp; Williams, P. H. (1991). What to protect?—systematics and the agony of choice. <em>Biological Conservation</em>, <em>55</em>(3), 235–254. Retrieved from <a href="https://www.sciencedirect.com/science/article/pii/000632079190030D">https://www.sciencedirect.com/science/article/pii/000632079190030D</a>
</div>
<div id="ref-Waichman1992" class="csl-entry" role="listitem">
Waichman, A. (1992). An alphanumeric code for toe clipping amphibians and reptiles. <em>Herpetological Review</em>, <em>23</em>, 19–21.
</div>
<div id="ref-Whitham2008" class="csl-entry" role="listitem">
Whitham, T. G., DiFazio, S. P., Schweitzer, J. A., Shuster, S. M., Allan, G. J., Bailey, J. K., &amp; Woolbright, S. A. (2008). Extending genomics to natural communities and ecosystems. <em>Science</em>, <em>320</em>(5875), 492–495. Retrieved from <a href="https://www.science.org/doi/abs/10.1126/science.1153918">https://www.science.org/doi/abs/10.1126/science.1153918</a>
</div>
</div>
</section>
    </div>
  </div>

  <script>window.backupDefine = window.define; window.define = undefined;</script>
  <script src="../../site_libs/revealjs/dist/reveal.js"></script>
  <!-- reveal.js plugins -->
  <script src="../../site_libs/revealjs/plugin/quarto-line-highlight/line-highlight.js"></script>
  <script src="../../site_libs/revealjs/plugin/pdf-export/pdfexport.js"></script>
  <script src="../../site_libs/revealjs/plugin/reveal-menu/menu.js"></script>
  <script src="../../site_libs/revealjs/plugin/reveal-menu/quarto-menu.js"></script>
  <script src="../../site_libs/revealjs/plugin/reveal-chalkboard/plugin.js"></script>
  <script src="../../site_libs/revealjs/plugin/multiplex/socket.io.js"></script>
  <script src="../../site_libs/revealjs/plugin/multiplex/multiplex.js"></script>
  <script src="../../site_libs/revealjs/plugin/quarto-support/support.js"></script>
  

  <script src="../../site_libs/revealjs/plugin/notes/notes.js"></script>
  <script src="../../site_libs/revealjs/plugin/search/search.js"></script>
  <script src="../../site_libs/revealjs/plugin/zoom/zoom.js"></script>
  <script src="../../site_libs/revealjs/plugin/math/math.js"></script>
  <script>window.define = window.backupDefine; window.backupDefine = undefined;</script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
'controlsAuto': true,
'previewLinksAuto': false,
'pdfSeparateFragments': false,
'autoAnimateEasing': "ease",
'autoAnimateDuration': 1,
'autoAnimateUnmatched': true,
'menu': {"side":"left","useTextContentForMissingTitles":true,"markers":false,"loadIcons":false,"custom":[{"title":"Tools","icon":"<i class=\"fas fa-gear\"></i>","content":"<ul class=\"slide-menu-items\">\n<li class=\"slide-tool-item active\" data-item=\"0\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.fullscreen(event)\"><kbd>f</kbd> Fullscreen</a></li>\n<li class=\"slide-tool-item\" data-item=\"1\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.speakerMode(event)\"><kbd>s</kbd> Speaker View</a></li>\n<li class=\"slide-tool-item\" data-item=\"2\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>o</kbd> Slide Overview</a></li>\n<li class=\"slide-tool-item\" data-item=\"3\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.togglePdfExport(event)\"><kbd>e</kbd> PDF Export Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"4\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleChalkboard(event)\"><kbd>b</kbd> Toggle Chalkboard</a></li>\n<li class=\"slide-tool-item\" data-item=\"5\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleNotesCanvas(event)\"><kbd>c</kbd> Toggle Notes Canvas</a></li>\n<li class=\"slide-tool-item\" data-item=\"6\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.downloadDrawings(event)\"><kbd>d</kbd> Download Drawings</a></li>\n<li class=\"slide-tool-item\" data-item=\"7\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.keyboardHelp(event)\"><kbd>?</kbd> Keyboard Help</a></li>\n</ul>"}],"openButton":true},
'chalkboard': {"buttons":true,"boardmarkerWidth":5},
'multiplex': {"secret":null,"id":"e8cb4fdfc4716a78","url":"https://reveal-multiplex.glitch.me/"},
'smaller': false,
 
        // Display controls in the bottom right corner
        controls: false,

        // Help the user learn the controls by providing hints, for example by
        // bouncing the down arrow when they first encounter a vertical slide
        controlsTutorial: false,

        // Determines where controls appear, "edges" or "bottom-right"
        controlsLayout: 'edges',

        // Visibility rule for backwards navigation arrows; "faded", "hidden"
        // or "visible"
        controlsBackArrows: 'faded',

        // Display a presentation progress bar
        progress: true,

        // Display the page number of the current slide
        slideNumber: 'c/t',

        // 'all', 'print', or 'speaker'
        showSlideNumber: 'all',

        // Add the current slide number to the URL hash so that reloading the
        // page/copying the URL will return you to the same slide
        hash: true,

        // Start with 1 for the hash rather than 0
        hashOneBasedIndex: false,

        // Flags if we should monitor the hash and change slides accordingly
        respondToHashChanges: true,

        // Push each slide change to the browser history
        history: true,

        // Enable keyboard shortcuts for navigation
        keyboard: true,

        // Enable the slide overview mode
        overview: true,

        // Disables the default reveal.js slide layout (scaling and centering)
        // so that you can use custom CSS layout
        disableLayout: false,

        // Vertical centering of slides
        center: false,

        // Enables touch navigation on devices with touch input
        touch: true,

        // Loop the presentation
        loop: false,

        // Change the presentation direction to be RTL
        rtl: false,

        // see https://revealjs.com/vertical-slides/#navigation-mode
        navigationMode: 'linear',

        // Randomizes the order of slides each time the presentation loads
        shuffle: false,

        // Turns fragments on and off globally
        fragments: true,

        // Flags whether to include the current fragment in the URL,
        // so that reloading brings you to the same fragment position
        fragmentInURL: false,

        // Flags if the presentation is running in an embedded mode,
        // i.e. contained within a limited portion of the screen
        embedded: false,

        // Flags if we should show a help overlay when the questionmark
        // key is pressed
        help: true,

        // Flags if it should be possible to pause the presentation (blackout)
        pause: true,

        // Flags if speaker notes should be visible to all viewers
        showNotes: false,

        // Global override for autoplaying embedded media (null/true/false)
        autoPlayMedia: null,

        // Global override for preloading lazy-loaded iframes (null/true/false)
        preloadIframes: null,

        // Number of milliseconds between automatically proceeding to the
        // next slide, disabled when set to 0, this value can be overwritten
        // by using a data-autoslide attribute on your slides
        autoSlide: 0,

        // Stop auto-sliding after user input
        autoSlideStoppable: true,

        // Use this method for navigation when auto-sliding
        autoSlideMethod: null,

        // Specify the average time in seconds that you think you will spend
        // presenting each slide. This is used to show a pacing timer in the
        // speaker view
        defaultTiming: null,

        // Enable slide navigation via mouse wheel
        mouseWheel: false,

        // The display mode that will be used to show slides
        display: 'block',

        // Hide cursor if inactive
        hideInactiveCursor: true,

        // Time before the cursor is hidden (in ms)
        hideCursorTime: 5000,

        // Opens links in an iframe preview overlay
        previewLinks: true,

        // Transition style (none/fade/slide/convex/concave/zoom)
        transition: 'fade',

        // Transition speed (default/fast/slow)
        transitionSpeed: 'default',

        // Transition style for full page slide backgrounds
        // (none/fade/slide/convex/concave/zoom)
        backgroundTransition: 'none',

        // Number of slides away from the current that are visible
        viewDistance: 3,

        // Number of slides away from the current that are visible on mobile
        // devices. It is advisable to set this to a lower number than
        // viewDistance in order to save resources.
        mobileViewDistance: 2,

        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1600,

        height: 900,

        // Factor of the display size that should remain empty around the content
        margin: 0.1,

        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [QuartoLineHighlight, PdfExport, RevealMenu, RevealChalkboard, QuartoSupport,

          RevealMath,
          
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    
    <script>
      // htmlwidgets need to know to resize themselves when slides are shown/hidden.
      // Fire the "slideenter" event (handled by htmlwidgets.js) when the current
      // slide changes (different for each slide format).
      (function () {
        // dispatch for htmlwidgets
        function fireSlideEnter() {
          const event = window.document.createEvent("Event");
          event.initEvent("slideenter", true, true);
          window.document.dispatchEvent(event);
        }

        function fireSlideChanged(previousSlide, currentSlide) {
          fireSlideEnter();

          // dispatch for shiny
          if (window.jQuery) {
            if (previousSlide) {
              window.jQuery(previousSlide).trigger("hidden");
            }
            if (currentSlide) {
              window.jQuery(currentSlide).trigger("shown");
            }
          }
        }

        // hookup for slidy
        if (window.w3c_slidy) {
          window.w3c_slidy.add_observer(function (slide_num) {
            // slide_num starts at position 1
            fireSlideChanged(null, w3c_slidy.slides[slide_num - 1]);
          });
        }

      })();
    </script>

    <script id="quarto-html-after-body" type="application/javascript">
    window.document.addEventListener("DOMContentLoaded", function (event) {
      const toggleBodyColorMode = (bsSheetEl) => {
        const mode = bsSheetEl.getAttribute("data-mode");
        const bodyEl = window.document.querySelector("body");
        if (mode === "dark") {
          bodyEl.classList.add("quarto-dark");
          bodyEl.classList.remove("quarto-light");
        } else {
          bodyEl.classList.add("quarto-light");
          bodyEl.classList.remove("quarto-dark");
        }
      }
      const toggleBodyColorPrimary = () => {
        const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
        if (bsSheetEl) {
          toggleBodyColorMode(bsSheetEl);
        }
      }
      toggleBodyColorPrimary();  
      const tabsets =  window.document.querySelectorAll(".panel-tabset-tabby")
      tabsets.forEach(function(tabset) {
        const tabby = new Tabby('#' + tabset.id);
      });
      const isCodeAnnotation = (el) => {
        for (const clz of el.classList) {
          if (clz.startsWith('code-annotation-')) {                     
            return true;
          }
        }
        return false;
      }
      const onCopySuccess = function(e) {
        // button target
        const button = e.trigger;
        // don't keep focus
        button.blur();
        // flash "checked"
        button.classList.add('code-copy-button-checked');
        var currentTitle = button.getAttribute("title");
        button.setAttribute("title", "Copied!");
        let tooltip;
        if (window.bootstrap) {
          button.setAttribute("data-bs-toggle", "tooltip");
          button.setAttribute("data-bs-placement", "left");
          button.setAttribute("data-bs-title", "Copied!");
          tooltip = new bootstrap.Tooltip(button, 
            { trigger: "manual", 
              customClass: "code-copy-button-tooltip",
              offset: [0, -8]});
          tooltip.show();    
        }
        setTimeout(function() {
          if (tooltip) {
            tooltip.hide();
            button.removeAttribute("data-bs-title");
            button.removeAttribute("data-bs-toggle");
            button.removeAttribute("data-bs-placement");
          }
          button.setAttribute("title", currentTitle);
          button.classList.remove('code-copy-button-checked');
        }, 1000);
        // clear code selection
        e.clearSelection();
      }
      const getTextToCopy = function(trigger) {
          const codeEl = trigger.previousElementSibling.cloneNode(true);
          for (const childEl of codeEl.children) {
            if (isCodeAnnotation(childEl)) {
              childEl.remove();
            }
          }
          return codeEl.innerText;
      }
      const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
        text: getTextToCopy
      });
      clipboard.on('success', onCopySuccess);
      if (window.document.getElementById('quarto-embedded-source-code-modal')) {
        // For code content inside modals, clipBoardJS needs to be initialized with a container option
        // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
        const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
          text: getTextToCopy,
          container: window.document.getElementById('quarto-embedded-source-code-modal')
        });
        clipboardModal.on('success', onCopySuccess);
      }
        var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
        var mailtoRegex = new RegExp(/^mailto:/);
          var filterRegex = new RegExp("https:\/\/giulianocolosimo\.github\.com");
        var isInternal = (href) => {
            return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
        }
        // Inspect non-navigation links and adorn them if external
     	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
        for (var i=0; i<links.length; i++) {
          const link = links[i];
          if (!isInternal(link.href)) {
            // undo the damage that might have been done by quarto-nav.js in the case of
            // links that we want to consider external
            if (link.dataset.originalHref !== undefined) {
              link.href = link.dataset.originalHref;
            }
          }
        }
      function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
        const config = {
          allowHTML: true,
          maxWidth: 500,
          delay: 100,
          arrow: false,
          appendTo: function(el) {
              return el.closest('section.slide') || el.parentElement;
          },
          interactive: true,
          interactiveBorder: 10,
          theme: 'light-border',
          placement: 'bottom-start',
        };
        if (contentFn) {
          config.content = contentFn;
        }
        if (onTriggerFn) {
          config.onTrigger = onTriggerFn;
        }
        if (onUntriggerFn) {
          config.onUntrigger = onUntriggerFn;
        }
          config['offset'] = [0,0];
          config['maxWidth'] = 700;
        window.tippy(el, config); 
      }
      const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
      for (var i=0; i<noterefs.length; i++) {
        const ref = noterefs[i];
        tippyHover(ref, function() {
          // use id or data attribute instead here
          let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
          try { href = new URL(href).hash; } catch {}
          const id = href.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note) {
            return note.innerHTML;
          } else {
            return "";
          }
        });
      }
      const findCites = (el) => {
        const parentEl = el.parentElement;
        if (parentEl) {
          const cites = parentEl.dataset.cites;
          if (cites) {
            return {
              el,
              cites: cites.split(' ')
            };
          } else {
            return findCites(el.parentElement)
          }
        } else {
          return undefined;
        }
      };
      var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
      for (var i=0; i<bibliorefs.length; i++) {
        const ref = bibliorefs[i];
        const citeInfo = findCites(ref);
        if (citeInfo) {
          tippyHover(citeInfo.el, function() {
            var popup = window.document.createElement('div');
            citeInfo.cites.forEach(function(cite) {
              var citeDiv = window.document.createElement('div');
              citeDiv.classList.add('hanging-indent');
              citeDiv.classList.add('csl-entry');
              var biblioDiv = window.document.getElementById('ref-' + cite);
              if (biblioDiv) {
                citeDiv.innerHTML = biblioDiv.innerHTML;
              }
              popup.appendChild(citeDiv);
            });
            return popup.innerHTML;
          });
        }
      }
    });
    </script>
    

</body></html>