<!DOCTYPE html>
<html lang="en"><head>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-html/tabby.min.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/light-border.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting-e26003cea8cd680ca0c55a263523d882.css" rel="stylesheet" id="quarto-text-highlighting-styles"><meta charset="utf-8">
  <meta name="generator" content="quarto-1.6.39">

  <meta name="dcterms.date" content="2024-11-18">
  <title>Biologia della Conservazione 24/25 – Biologia della Conservazione</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="../../site_libs/revealjs/dist/reset.css">
  <link rel="stylesheet" href="../../site_libs/revealjs/dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
      vertical-align: middle;
    }
    /* CSS for syntax highlighting */
    pre > code.sourceCode { white-space: pre; position: relative; }
    pre > code.sourceCode > span { line-height: 1.25; }
    pre > code.sourceCode > span:empty { height: 1.2em; }
    .sourceCode { overflow: visible; }
    code.sourceCode > span { color: inherit; text-decoration: inherit; }
    div.sourceCode { margin: 1em 0; }
    pre.sourceCode { margin: 0; }
    @media screen {
    div.sourceCode { overflow: auto; }
    }
    @media print {
    pre > code.sourceCode { white-space: pre-wrap; }
    pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
    }
    pre.numberSource code
      { counter-reset: source-line 0; }
    pre.numberSource code > span
      { position: relative; left: -4em; counter-increment: source-line; }
    pre.numberSource code > span > a:first-child::before
      { content: counter(source-line);
        position: relative; left: -1em; text-align: right; vertical-align: baseline;
        border: none; display: inline-block;
        -webkit-touch-callout: none; -webkit-user-select: none;
        -khtml-user-select: none; -moz-user-select: none;
        -ms-user-select: none; user-select: none;
        padding: 0 4px; width: 4em;
        color: #aaaaaa;
      }
    pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
    div.sourceCode
      { color: #003b4f; background-color: #f1f3f5; }
    @media screen {
    pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
    }
    code span { color: #003b4f; } /* Normal */
    code span.al { color: #ad0000; } /* Alert */
    code span.an { color: #5e5e5e; } /* Annotation */
    code span.at { color: #657422; } /* Attribute */
    code span.bn { color: #ad0000; } /* BaseN */
    code span.bu { } /* BuiltIn */
    code span.cf { color: #003b4f; font-weight: bold; } /* ControlFlow */
    code span.ch { color: #20794d; } /* Char */
    code span.cn { color: #8f5902; } /* Constant */
    code span.co { color: #5e5e5e; } /* Comment */
    code span.cv { color: #5e5e5e; font-style: italic; } /* CommentVar */
    code span.do { color: #5e5e5e; font-style: italic; } /* Documentation */
    code span.dt { color: #ad0000; } /* DataType */
    code span.dv { color: #ad0000; } /* DecVal */
    code span.er { color: #ad0000; } /* Error */
    code span.ex { } /* Extension */
    code span.fl { color: #ad0000; } /* Float */
    code span.fu { color: #4758ab; } /* Function */
    code span.im { color: #00769e; } /* Import */
    code span.in { color: #5e5e5e; } /* Information */
    code span.kw { color: #003b4f; font-weight: bold; } /* Keyword */
    code span.op { color: #5e5e5e; } /* Operator */
    code span.ot { color: #003b4f; } /* Other */
    code span.pp { color: #ad0000; } /* Preprocessor */
    code span.sc { color: #5e5e5e; } /* SpecialChar */
    code span.ss { color: #20794d; } /* SpecialString */
    code span.st { color: #20794d; } /* String */
    code span.va { color: #111111; } /* Variable */
    code span.vs { color: #20794d; } /* VerbatimString */
    code span.wa { color: #5e5e5e; font-style: italic; } /* Warning */
    /* CSS for citations */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
      margin-bottom: 0em;
    }
    .hanging-indent div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }  </style>
  <link rel="stylesheet" href="../../site_libs/revealjs/dist/theme/quarto-fbff88913fe25870c86f77c0ed158baa.css">
  <link href="../../site_libs/revealjs/plugin/quarto-line-highlight/line-highlight.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-menu/menu.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-menu/quarto-menu.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-chalkboard/font-awesome/css/all.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-chalkboard/style.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">
  <style type="text/css">
    .reveal div.sourceCode {
      margin: 0;
      overflow: auto;
    }
    .reveal div.hanging-indent {
      margin-left: 1em;
      text-indent: -1em;
    }
    .reveal .slide:not(.center) {
      height: 100%;
    }
    .reveal .slide.scrollable {
      overflow-y: auto;
    }
    .reveal .footnotes {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide .absolute {
      position: absolute;
      display: block;
    }
    .reveal .footnotes ol {
      counter-reset: ol;
      list-style-type: none; 
      margin-left: 0;
    }
    .reveal .footnotes ol li:before {
      counter-increment: ol;
      content: counter(ol) ". "; 
    }
    .reveal .footnotes ol li > p:first-child {
      display: inline-block;
    }
    .reveal .slide ul,
    .reveal .slide ol {
      margin-bottom: 0.5em;
    }
    .reveal .slide ul li,
    .reveal .slide ol li {
      margin-top: 0.4em;
      margin-bottom: 0.2em;
    }
    .reveal .slide ul[role="tablist"] li {
      margin-bottom: 0;
    }
    .reveal .slide ul li > *:first-child,
    .reveal .slide ol li > *:first-child {
      margin-block-start: 0;
    }
    .reveal .slide ul li > *:last-child,
    .reveal .slide ol li > *:last-child {
      margin-block-end: 0;
    }
    .reveal .slide .columns:nth-child(3) {
      margin-block-start: 0.8em;
    }
    .reveal blockquote {
      box-shadow: none;
    }
    .reveal .tippy-content>* {
      margin-top: 0.2em;
      margin-bottom: 0.7em;
    }
    .reveal .tippy-content>*:last-child {
      margin-bottom: 0.2em;
    }
    .reveal .slide > img.stretch.quarto-figure-center,
    .reveal .slide > img.r-stretch.quarto-figure-center {
      display: block;
      margin-left: auto;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-left,
    .reveal .slide > img.r-stretch.quarto-figure-left  {
      display: block;
      margin-left: 0;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-right,
    .reveal .slide > img.r-stretch.quarto-figure-right  {
      display: block;
      margin-left: auto;
      margin-right: 0; 
    }
  </style>
  <link href="../../site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">
  <script src="../../site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
  <link href="../../site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
  <script src="../../site_libs/datatables-binding-0.33/datatables.js"></script>
  <script src="../../site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
  <link href="../../site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet">
  <link href="../../site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet">
  <script src="../../site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
  <link href="../../site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet">
  <script src="../../site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<meta property="og:title" content="Biologia della Conservazione – Biologia della Conservazione 24/25">
<meta property="og:description" content="Class 5 - HW and Genetic drift">
<meta property="og:site_name" content="Biologia della Conservazione 24/25">
<meta property="og:locale" content="en_US">
</head>
<body class="quarto-light">
  <div class="reveal">
    <div class="slides">

<section id="title-slide" class="quarto-title-block center">
  <h1 class="title">Biologia della Conservazione</h1>
  <p class="subtitle">Class 5 - HW and Genetic drift</p>

<div class="quarto-title-authors">
</div>

  <p class="date">2024-11-18</p>
</section>
<section id="analyzing-the-hardy-weinberg-model" class="slide level2">
<h2>Analyzing the Hardy-Weinberg model</h2>
<ul>
<li>We begin considering a diploid population with n = 15 individuals</li>
</ul>

<img data-src="./population10a.png" class="quarto-figure quarto-figure-center r-stretch"></section>
<section class="slide level2">

<ul>
<li>Being diploids each individual has two alleles per locus (A and a)</li>
</ul>

<img data-src="./population10c.png" class="quarto-figure quarto-figure-center r-stretch"></section>
<section class="slide level2">

<ul>
<li>We can summarize the information in a table:</li>
</ul>
<table class="caption-top">
<thead>
<tr class="header">
<th><span class="math inline">\(n_{AA}\)</span></th>
<th><span class="math inline">\(n_{Aa}\)</span></th>
<th><span class="math inline">\(n_{aa}\)</span></th>
<th><span class="math inline">\(n\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>6</td>
<td>7</td>
<td>2</td>
<td>15</td>
</tr>
</tbody>
</table>
<ul>
<li>We can estimate the allele frequencies:</li>
</ul>
<p><span class="math display">\[
p = f(A) = \frac{2n_{AA} + n_{Aa}}{2n} = \frac{12 + 7}{30} \approx 0.633
\]</span> and</p>
<p><span class="math display">\[
q = f(a) = \frac{2n_{aa} + n_{Aa}}{2n}= \frac{4 + 7}{30} \approx 0.367
\]</span></p>
<ul>
<li>Since there are only two alleles we can write <span class="math inline">\(p + q = 1\)</span></li>
</ul>
</section>
<section id="expansion-of-the-hardy-weinberg-model-with-2-alleles" class="slide level2">
<h2>Expansion of the Hardy-Weinberg model with &gt; 2 alleles</h2>
<ul>
<li><p>In this previous basic example we only had 2 alleles and therefore <span class="math inline">\(p+q=1\)</span>.</p></li>
<li><p>Because of our assumption of random mating the assortment of alleles in the zygotes can be estimated from the allele frequencies using a binomial expansion:</p></li>
</ul>
<p><span class="math display">\[
(p+q)^2=p^2+2pq+q^2
\]</span></p>
<ul>
<li>If we want to add another allele (<span class="math inline">\(A_1, A_2, A_3\)</span>)…</li>
</ul>
<p><span class="math display">\[
(p+q+r)^2=p^2+2pq+q^2+2pr+2qr+r^2
\]</span></p>
</section>
<section class="slide level2">

<ul>
<li><p>“All alleles and genotypes have the same fitness with no differential reproduction or survival” and individuals are “mating at random”</p></li>
<li><p>If this is true, the frequencies of male and female gametes correspond to the allele frequencies and zygotes are formed randomly choosing gametes from the gene pool</p></li>
</ul>

<img data-src="./genePool.png" class="quarto-figure quarto-figure-center r-stretch"></section>
<section id="basic-probability-theory" class="slide level2">
<h2>Basic probability theory</h2>
<ul>
<li>The probability of two independent events, such as the union of two gametes at random in a zygote, is calculated as the product of the probability of the single events</li>
</ul>
<div class="cell columns column-output-location">
<div class="column">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a></a>dice <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">2</span>,<span class="dv">3</span>,<span class="dv">4</span>,<span class="dv">5</span>,<span class="dv">6</span>)</span>
<span id="cb1-2"><a></a>dice</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div><div class="column fragment">
<div class="cell-output cell-output-stdout">
<pre><code>[1] 1 2 3 4 5 6</code></pre>
</div>
</div></div>
<p><br></p>
<div class="cell columns column-output-location">
<div class="column">
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a></a><span class="fu">sample</span>(dice, <span class="at">size =</span> <span class="dv">1</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div><div class="column fragment">
<div class="cell-output cell-output-stdout">
<pre><code>[1] 4</code></pre>
</div>
</div></div>
<p><br></p>
<div class="cell columns column-output-location">
<div class="column">
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb5-1"><a></a><span class="fu">sample</span>(dice, <span class="at">size =</span> <span class="dv">1</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div><div class="column fragment">
<div class="cell-output cell-output-stdout">
<pre><code>[1] 3</code></pre>
</div>
</div></div>
<p><br></p>
<div class="cell columns column-output-location">
<div class="column">
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a></a>sim_dice <span class="ot">&lt;-</span> <span class="fu">sample</span>(dice, <span class="at">size =</span> <span class="dv">100000</span>,</span>
<span id="cb7-2"><a></a>                   <span class="at">replace =</span> T)</span>
<span id="cb7-3"><a></a><span class="fu">table</span>(sim_dice)<span class="sc">/</span><span class="dv">100000</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div><div class="column fragment">
<div class="cell-output cell-output-stdout">
<pre><code>sim_dice
      1       2       3       4       5       6 
0.16774 0.16649 0.16765 0.16483 0.16685 0.16644 </code></pre>
</div>
</div></div>
<div class="fragment">
<ul>
<li>The probability of casting a dice and obtaining one of the six numbers is <span class="math inline">\(\frac{1}{6} \approx 0.17\)</span></li>
</ul>
</div>
</section>
<section class="slide level2">

<ul>
<li>The probability of casting the dice twice and obtaining the same number is <span class="math inline">\(\frac{1}{6} * \frac{1}{6} = \frac{1}{36} \approx 0.028\)</span></li>
</ul>
<div class="fragment">
<ul>
<li>We want to recreate a sample of 15 individuals starting from the gamete pool of our original population with <span class="math inline">\(n = 15\)</span> and <span class="math inline">\(p \approx 0.633\)</span> and <span class="math inline">\(q \approx 0.367\)</span></li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./genePoolHWA0.png" class="quarto-figure quarto-figure-center" style="width:50.0%"></p>
</figure>
</div>
</div>
</section>
<section class="slide level2">

<div class="cell">
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb9-1"><a></a>p <span class="ot">&lt;-</span> <span class="fl">0.633</span> </span>
<span id="cb9-2"><a></a>q <span class="ot">&lt;-</span> <span class="fl">0.367</span>  </span>
<span id="cb9-3"><a></a><span class="fu">sample</span>(<span class="fu">c</span>(<span class="st">"A"</span>, <span class="st">"a"</span>), <span class="at">size =</span> <span class="dv">1</span>, <span class="at">prob =</span> <span class="fu">c</span>(p, q))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="fragment">
<ul>
<li>Every time we sample an allele, we add a color. Remember A = <span style="color: red;">red</span> and a = <span style="color: yellow;">yellow</span></li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./genePoolGif.gif" class="quarto-figure quarto-figure-center" style="width:65.0%"></p>
</figure>
</div>
</div>
</section>
<section class="slide level2">


<img data-src="./genePoolHWA30.png" class="quarto-figure quarto-figure-center r-stretch"><ul>
<li>We summarize the results in a table</li>
</ul>
<table class="caption-top">
<thead>
<tr class="header">
<th><span class="math inline">\(n_{AA}\)</span></th>
<th><span class="math inline">\(n_{Aa}\)</span></th>
<th><span class="math inline">\(n_{aa}\)</span></th>
<th><span class="math inline">\(n\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>7</td>
<td>7</td>
<td>1</td>
<td>15</td>
</tr>
</tbody>
</table>
</section>
<section class="slide level2">

<ul>
<li>The genotype and allele frequencies of the new generation are different from the previous one. We repeat the experiment one more time</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb10"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb10-1"><a></a>p <span class="ot">&lt;-</span> <span class="fl">0.633</span> </span>
<span id="cb10-2"><a></a>q <span class="ot">&lt;-</span> <span class="fl">0.367</span>  </span>
<span id="cb10-3"><a></a>newSample <span class="ot">&lt;-</span> <span class="fu">sample</span>(<span class="fu">c</span>(<span class="st">"A"</span>, <span class="st">"a"</span>), <span class="at">size =</span> <span class="dv">30</span>,</span>
<span id="cb10-4"><a></a>                    <span class="at">prob =</span> <span class="fu">c</span>(p, q),</span>
<span id="cb10-5"><a></a>                    <span class="at">replace =</span> T)</span>
<span id="cb10-6"><a></a><span class="fu">table</span>(newSample)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>newSample
 a  A 
10 20 </code></pre>
</div>
</div>
<ul>
<li>Once again, the genotype and allele frequencies of the new generation are different from the previous one, and it will likely be different most of the time we do this again</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb12"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb12-1"><a></a>p <span class="ot">&lt;-</span> <span class="fl">0.633</span> </span>
<span id="cb12-2"><a></a>q <span class="ot">&lt;-</span> <span class="fl">0.367</span>  </span>
<span id="cb12-3"><a></a>newSample <span class="ot">&lt;-</span> <span class="fu">sample</span>(<span class="fu">c</span>(<span class="st">"A"</span>, <span class="st">"a"</span>), <span class="at">size =</span> <span class="dv">30</span>,</span>
<span id="cb12-4"><a></a>                    <span class="at">prob =</span> <span class="fu">c</span>(p, q),</span>
<span id="cb12-5"><a></a>                    <span class="at">replace =</span> T)</span>
<span id="cb12-6"><a></a><span class="fu">table</span>(newSample)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>newSample
 a  A 
12 18 </code></pre>
</div>
</div>
</section>
<section class="slide level2">

<div class="cell" data-layout-align="center">
<div class="sourceCode cell-code" id="cb14"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb14-1"><a></a><span class="co"># This code has been modified from prof. Gratton</span></span>
<span id="cb14-2"><a></a>newSample <span class="ot">&lt;-</span> <span class="fu">t</span>(<span class="fu">sapply</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">10000</span>, <span class="cf">function</span>(i){<span class="fu">table</span>(<span class="fu">factor</span>(<span class="fu">sample</span>(<span class="fu">c</span>(<span class="st">"A"</span>,<span class="st">"a"</span>), <span class="at">size =</span> <span class="dv">30</span>, <span class="at">prob =</span> <span class="fu">c</span>(p,q), <span class="at">replace =</span> T),</span>
<span id="cb14-3"><a></a>                                               <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"A"</span>,<span class="st">"a"</span>)))}))</span>
<span id="cb14-4"><a></a><span class="fu">hist</span>(newSample[,<span class="st">"A"</span>]<span class="sc">/</span><span class="dv">30</span>, </span>
<span id="cb14-5"><a></a>     <span class="at">xlim =</span> <span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">1</span>), <span class="at">xlab =</span> <span class="st">"f(A)"</span>, </span>
<span id="cb14-6"><a></a>     <span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="dv">1</span>, <span class="at">by=</span> <span class="fl">0.1</span>),</span>
<span id="cb14-7"><a></a>     <span class="at">col =</span> <span class="st">"darkgoldenrod3"</span>, </span>
<span id="cb14-8"><a></a>     <span class="at">border =</span> <span class="cn">NA</span>, </span>
<span id="cb14-9"><a></a>     <span class="at">main =</span> <span class="st">"10,000 samples of size 30"</span>,</span>
<span id="cb14-10"><a></a>     <span class="at">cex.axis =</span> <span class="dv">2</span>, <span class="at">cex.main =</span> <span class="dv">2</span>, <span class="at">cex.lab =</span> <span class="dv">2</span>)</span>
<span id="cb14-11"><a></a><span class="fu">abline</span>(<span class="at">v =</span> <span class="fl">0.637</span>, <span class="at">lty =</span> <span class="dv">2</span>, <span class="at">lwd =</span> <span class="dv">2</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="class_5_files/figure-revealjs/unnamed-chunk-4-1.png" class="quarto-figure quarto-figure-center r-stretch" width="960"></section>
<section class="slide level2">

<ul>
<li><p>We notice that there is a pretty large variability around the original value of p = 0.633</p></li>
<li><p>In other words, the sampling error is quite large</p></li>
</ul>

<img data-src="class_5_files/figure-revealjs/unnamed-chunk-5-1.png" class="quarto-figure quarto-figure-center r-stretch" width="960"></section>
<section class="slide level2">

<div class="cell" data-layout-align="center">
<div class="sourceCode cell-code" id="cb15"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb15-1"><a></a>newSample <span class="ot">&lt;-</span> <span class="fu">t</span>(<span class="fu">sapply</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">10000</span>, <span class="cf">function</span>(i){<span class="fu">table</span>(<span class="fu">factor</span>(<span class="fu">sample</span>(<span class="fu">c</span>(<span class="st">"A"</span>,<span class="st">"a"</span>), <span class="at">size =</span> <span class="dv">100</span>, <span class="at">prob =</span> <span class="fu">c</span>(p,q), <span class="at">replace =</span> T),</span>
<span id="cb15-2"><a></a>                                               <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"A"</span>,<span class="st">"a"</span>)))}))</span>
<span id="cb15-3"><a></a><span class="fu">hist</span>(newSample[,<span class="st">"A"</span>]<span class="sc">/</span><span class="dv">100</span>, </span>
<span id="cb15-4"><a></a>     <span class="at">xlim =</span> <span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">1</span>), <span class="at">xlab =</span> <span class="st">"f(A)"</span>, </span>
<span id="cb15-5"><a></a>     <span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="dv">1</span>, <span class="at">by=</span> <span class="fl">0.01</span>),</span>
<span id="cb15-6"><a></a>     <span class="at">col =</span> <span class="st">"darkgoldenrod3"</span>, </span>
<span id="cb15-7"><a></a>     <span class="at">border =</span> <span class="cn">NA</span>, </span>
<span id="cb15-8"><a></a>     <span class="at">main =</span> <span class="st">"10,000 samples of size 100"</span>,</span>
<span id="cb15-9"><a></a>     <span class="at">cex.axis =</span> <span class="dv">2</span>, <span class="at">cex.main =</span> <span class="dv">2</span>, <span class="at">cex.lab =</span> <span class="dv">2</span>)</span>
<span id="cb15-10"><a></a><span class="fu">abline</span>(<span class="at">v =</span> <span class="fl">0.637</span>, <span class="at">lty =</span> <span class="dv">2</span>, <span class="at">lwd =</span> <span class="dv">2</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="class_5_files/figure-revealjs/unnamed-chunk-6-1.png" class="quarto-figure quarto-figure-center r-stretch" width="960"></section>
<section class="slide level2">


<img data-src="./samplingError.png" class="quarto-figure quarto-figure-center r-stretch"></section>
<section id="summary-of-the-hardy-weinberg-model" class="slide level2">
<h2>Summary of the Hardy-Weinberg model</h2>
<ul>
<li><p>The larger the sample dimension, the lower is the variation in allele frequencies with respect to the original value</p></li>
<li><p>If all the assumptions of the HW model are met, we should expect no variation at all in allele frequency, generation after generation</p></li>
<li><p>In natural population it is very hard to meet all the assumptions of the HW, in particular the one of infinite population size</p></li>
<li><p>The HW model is a very effective and simple way to “see” the effects of various evolutionary forces operating at the population level. It provides a “null model” against which compare what we see in our data</p></li>
<li><p>Once we found that something is different between what we expect from the HW model and what we see in nature at the population genetic level, we can start investigate the relative role of the various evolutionary forces</p></li>
</ul>
</section>
<section id="testing-a-null-hypothesis" class="slide level2">
<h2>Testing a null hypothesis</h2>
<ul>
<li>A fundamental aspect of every scientific discipline is the capability to discern between alternative hypotheses. The general approach is to present a formal or null-hypothesis (<span class="math inline">\(H_0\)</span>) and a statistical test to estimate the probability that it holds true</li>
<li>Such test usually implies the calculation of a P-value (<em>i.e.</em>, a probability distribution)</li>
<li>Lower P-values correspond to higher degree of certainty that the null is rejected and an alternative hypothesis (<span class="math inline">\(H_1\)</span>) is preferred to explain the data</li>
<li>When <span class="math inline">\(P=0.002\)</span> it means that, if the null holds true, you expect to see a result as extreme or even more extreme than the one you have measured only 2 times out of 1000 experiment</li>
<li>The general threshold value (<span class="math inline">\(\alpha\)</span>) is <span class="math inline">\(P=0.05\)</span>, meaning that you have 5 chances in 100 of rejecting the null hypothesis when it is true</li>
</ul>
</section>
<section class="slide level2">

<h3 id="excercise-to-test-the-hw-proportions">Excercise to test the HW proportions</h3>
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-93c78a364d7e06b3febe" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-93c78a364d7e06b3febe">{"x":{"filter":"none","vertical":false,"data":[["A1A1","A1A2","A2A2"],[907,616,124]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>Genotypes<\/th>\n      <th>Number<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":1},{"name":"Genotypes","targets":0},{"name":"Number","targets":1}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div class="callout callout-important callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>Problem</strong></p>
</div>
<div class="callout-content">
<p>Calculate allele frequency, genotype frequency and expected genotype frequency.</p>
</div>
</div>
</div>
</section>
<section class="slide level2">

<ul>
<li>Allele frequency:</li>
</ul>
<p><span class="math display">\[
p = f(A) = \frac{2n_{11} + n_{12}}{2n} = \frac{(2*907)+616}{2*1647} =0.738
\]</span></p>
<p><span class="math display">\[
q = f(a) = \frac{2n_{22} + n_{12}}{2n} = \frac{(2*124)+616}{2*1647} =0.262
\]</span></p>
<div class="fragment">
<ul>
<li>Observed genotype frequency:</li>
</ul>
<p><span class="math display">\[
f(AA) =\frac{n_{11}}{n}  = \frac{907}{1647}=0.551; \ \ \ f(Aa) =\frac{n_{12}}{n}  = \frac{616}{1647}=0.374
\]</span></p>
<p><span class="math display">\[
f(aa) =\frac{n_{22}}{n}  = \frac{124}{1647}=0.075
\]</span></p>
</div>
</section>
<section class="slide level2">

<ul>
<li>Expected genotype frequency:</li>
</ul>
<p><span class="math display">\[
p^2 = f(AA)^2 = 0.738^2 = 0.545; \ \ \ 2pq = f(Aa) = 2*0.738*0.262=0.387
\]</span></p>
<p><span class="math display">\[
q^2 = f(aa)=0.262^2=0.068
\]</span></p>
</section>
<section id="is-this-difference-significant" class="slide level2">
<h2>Is this difference significant?</h2>
<p><span class="math display">\[
\chi^2=\sum\frac{(Observed-Expected)^2}{Expected}
\]</span></p>
<ul>
<li><p>This methodology allows us to understand if the difference between observed and expected outcomes is greater than we would expect by chance</p></li>
<li><p>The empirical rule is to run the <span class="math inline">\(\chi^2\)</span> on the actual genotype count rather than the genotype frequencies</p></li>
</ul>
<div class="fragment">
<div class="cell">
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb16-1"><a></a>((<span class="dv">907</span><span class="fl">-897.615</span>)<span class="sc">^</span><span class="dv">2</span><span class="sc">/</span><span class="fl">897.615</span>)<span class="sc">+</span>((<span class="dv">616</span><span class="fl">-637.389</span>)<span class="sc">^</span><span class="dv">2</span><span class="sc">/</span><span class="fl">637.389</span>)<span class="sc">+</span>((<span class="dv">124</span><span class="fl">-111.996</span>)<span class="sc">^</span><span class="dv">2</span><span class="sc">/</span><span class="fl">111.996</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 2.102498</code></pre>
</div>
</div>
<ul>
<li>The computed <span class="math inline">\(\chi^2\)</span> value is compared to a set of values calculated under the assumption that the null hypothesis we are testing is correct; our null hypothesis is that the population from which the samples was drawn is in HW proportions.</li>
</ul>
<div class="cell columns column-output-location">
<div class="column">
<div class="sourceCode cell-code" id="cb18"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb18-1"><a></a><span class="fu">pchisq</span>(<span class="at">q =</span> <span class="fl">2.102498</span>, <span class="at">df =</span> <span class="dv">1</span>, <span class="at">lower.tail =</span> F)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div><div class="column fragment">
<div class="cell-output cell-output-stdout">
<pre><code>[1] 0.1470587</code></pre>
</div>
</div></div>
</div>
</section>
<section class="slide level2">


<img data-src="class_5_files/figure-revealjs/chiSq2-1.png" width="960" class="r-stretch"></section>
<section class="slide level2">

<h3 id="degrees-of-freedom">Degrees of freedom</h3>
<ul>
<li>The number of degrees of freedom is the number of values in the final calculation of a statistic that are free to vary <span class="citation" data-cites="Wiki_DoF_2023">(<a href="#/references" role="doc-biblioref" onclick="">Wikipedia contributors, 2023</a>)</span></li>
<li>If a car dealer is giving away 15 random cars to the first 15 customers, the first 14 that show up at the car dealer will have the freedom to pick and choose the car they want. The <span class="math inline">\(15^{th}\)</span> will not be able to do so as only one car will be remaining. Similarly, if you calculated the mean of a sample of 15 numbers, the first 14 numbers can vary freely, but the value of the <span class="math inline">\(15^{th}\)</span> number is determined as the value needed to achieve the given sample mean!</li>
<li>Degrees of freedom for tests for Hardy–Weinberg proportions are estimated according to: <span class="math inline">\(df=\# genotypes − \# alleles\)</span>. This is because the number of genotypes in a population can not vary freely, but is bound to the number of alleles present in that population</li>
</ul>
</section>
<section class="slide level2">


<img data-src="class_5_files/figure-revealjs/pvalxsquplot-1.png" width="960" class="r-stretch"></section>
<section id="when-not-to-use-the-chi2-test" class="slide level2">
<h2>When not to use the <span class="math inline">\(\chi^2\)</span> test</h2>
<ul>
<li>The expected count of individuals is &lt; 5 in any of the possible genotype considered</li>
<li>Very low allele frequencies</li>
<li>The sample size is too small</li>
<li>There are too many alleles at a locus</li>
</ul>
<h3 id="a-valid-alternative-fishers-exact-test">A valid alternative: Fisher’s exact test</h3>
<ul>
<li>Based on the calculation of probabilities</li>
<li><span class="math inline">\(P(genotype\ counts|allele \ counts)\)</span> assuming HWE</li>
<li>Let’s work out an example with a small population: 4 individuals <span class="math inline">\(AA\)</span>, 2 individuals <span class="math inline">\(Aa\)</span> and 1 individuals <span class="math inline">\(aa\)</span></li>
<li>What genotype configurations could give us the same allele count?</li>
</ul>
</section>
<section class="slide level2">

<table class="caption-top">
<thead>
<tr class="header">
<th style="text-align: center;"><span class="math inline">\(n_{AA}\)</span></th>
<th style="text-align: center;"><span class="math inline">\(n_{Aa}\)</span></th>
<th style="text-align: center;"><span class="math inline">\(n_{aa}\)</span></th>
<th style="text-align: center;"><span class="math inline">\(n_{A}\)</span></th>
<th style="text-align: center;"><span class="math inline">\(n_{a}\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">5</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">2</td>
<td style="text-align: center;">10</td>
<td style="text-align: center;">4</td>
</tr>
<tr class="even">
<td style="text-align: center;">4</td>
<td style="text-align: center;">2</td>
<td style="text-align: center;">1</td>
<td style="text-align: center;">10</td>
<td style="text-align: center;">4</td>
</tr>
<tr class="odd">
<td style="text-align: center;">3</td>
<td style="text-align: center;">4</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">10</td>
<td style="text-align: center;">4</td>
</tr>
</tbody>
</table>
<div class="fragment">
<ul>
<li>Can we estimate the theoretical probability of each combination of genotypes?</li>
</ul>
</div>
<div class="fragment">
<p><span class="math display">\[
P(n_{Aa}|n_A, n_a, HWE)=\frac{n!}{n_{AA}!*n_{Aa}!*n_{aa}!}*\frac{2^{n_{Aa}}*n_A!*n_a!}{(2n)!}
\]</span></p>
</div>
<div class="fragment">
<table class="caption-top">
<thead>
<tr class="header">
<th style="text-align: center;"><span class="math inline">\(n_{AA}\)</span></th>
<th style="text-align: center;"><span class="math inline">\(n_{Aa}\)</span></th>
<th style="text-align: center;"><span class="math inline">\(n_{aa}\)</span></th>
<th style="text-align: center;">Theoretical probability</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">5</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">2</td>
<td style="text-align: center;">0.021</td>
</tr>
<tr class="even">
<td style="text-align: center;">4</td>
<td style="text-align: center;">2</td>
<td style="text-align: center;">1</td>
<td style="text-align: center;">0.419</td>
</tr>
<tr class="odd">
<td style="text-align: center;">3</td>
<td style="text-align: center;">4</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0.559</td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="other-options-for-testing-null-hypothesis" class="slide level2">
<h2>Other options for testing null hypothesis</h2>
<ul>
<li>Likelihood, the probability of observing the data given some parameter value under a certain statistical model, particularly useful when there are many alleles at a single locus</li>
<li>Permutation tests, randomization of the dataset by sampling, particularly useful when many genotypes are expected to have very low frequency</li>
</ul>
<h3 id="multiple-tests">Multiple tests</h3>
<ul>
<li><p>If we examine 10 loci in 10 population samples, 100 tests of HW proportions will be performed. If all of these loci are in HW proportions (<em>i.e.</em>, our null hypothesis is true at all loci in all populations), we expect to find five significant tests if we use the 5% significance level <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022, pp. 102–103</a>)</span>.</p></li>
<li><p>The Bonferroni correction allows to adjust the significance level (say 5%) by dividing it by the number of tests performed. In the previous example <span class="math inline">\(0.05/100 = 0.0005\)</span>. The critical <span class="math inline">\(\chi^2\)</span> value for <span class="math inline">\(P = 0.0005\)</span> with one degree of freedom is 12.1.</p></li>
</ul>
</section>
<section id="genetic-drift" class="slide level2">
<h2>Genetic Drift</h2>
<ul>
<li>Genetic drift is the random change in allele frequencies from generation to generation because of sampling error. That is, the finite number of genes transmitted to progeny will be an imperfect sample of the allele frequencies in the previous generation <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022, p. 114</a>)</span></li>
<li>Genetic drift is the primary cause bringing about allele frequency changes throughout the genome over time <span class="citation" data-cites="Chen2018 Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>; <a href="#/references" role="doc-biblioref" onclick="">Chen et al., 2018</a>)</span></li>
</ul>

<img data-src="./samplingError.png" class="quarto-figure quarto-figure-center r-stretch" style="width:60.0%"></section>
<section class="slide level2">

<ul>
<li><p>With many samples (<span class="math inline">\(n &gt;&gt; 1\)</span>) the probability that their alleles all are <span class="math inline">\(A_1\)</span> or <span class="math inline">\(A_2\)</span> becomes <span class="math inline">\((\frac{1}{2})^{2n}\)</span>, and it will be less likely to loose one of the two alleles</p></li>
<li><p>The probability that a population has <span class="math inline">\(k\)</span> copies of allele <span class="math inline">\(A_1\)</span> and <span class="math inline">\(2n-k\)</span> copies of <span class="math inline">\(A_2\)</span> allele is <span class="math inline">\(\binom{2n}{k}p^kq^{2n-k}\)</span> where <span class="math inline">\(\binom{2n}{k}\)</span> is the binomial function <span class="math inline">\(\frac{2n!}{k!(2n-k)!}\)</span></p></li>
<li><p>The variance around the mean allele frequency is the binomial sampling variance: <span class="math inline">\(\sigma^2_p =\frac{p_0q_0}{2n}\)</span></p></li>
<li><p>Variances are higher in smaller than larger populations</p></li>
<li><p>In small population the process of sampling happens every single generation, and its effect on allele frequency is cumulative:</p></li>
<li><p>Genetic drift will eventually cause all except one allele to go to fixation.</p></li>
<li><p>The probability that a gamete does not contain allele <span class="math inline">\(A_1\)</span> (with frequency <span class="math inline">\(p\)</span> in the parental population) is <span class="math inline">\(1-p\)</span></p></li>
</ul>
<p><span class="math display">\[
P(fixation \ A_2)=(1-p)^{2n}
\]</span></p>
</section>
<section class="slide level2">

<div class="callout callout-important callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>Problem</strong></p>
</div>
<div class="callout-content">
<p>Diploid population with <span class="math inline">\(n = 4\)</span> with allele <span class="math inline">\(A_1\)</span> with a frequency <span class="math inline">\(p = 0.25\)</span> and allele <span class="math inline">\(A_2\)</span> with a frequency <span class="math inline">\(q = 0.75\)</span>. What is the probability of losing allele <span class="math inline">\(A_1\)</span> in one generation of sampling to recreate a population of the same size?</p>
</div>
</div>
</div>
<div class="fragment">
<div class="callout callout-tip callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>Solution</strong></p>
</div>
<div class="callout-content">
<p><span class="math inline">\((1-0.25)^8=\)</span> 0.1001129</p>
</div>
</div>
</div>
</div>
</section>
<section class="slide level2">

<h3 id="genetic-drift-1">Genetic Drift</h3>

<img data-src="class_5_files/figure-revealjs/drift1-1.png" width="960" class="r-stretch quarto-figure-center"><p class="caption">Allele frequency distribution at t = 0, with N = 100 <span class="citation" data-cites="Russo2021">(<a href="#/references" role="doc-biblioref" onclick="">Russo, Colosimo, Fanelli, Franceschini, &amp; Parisi, 2021</a>)</span>.</p></section>
<section class="slide level2">

<h3 id="genetic-drift-2">Genetic Drift</h3>

<img data-src="class_5_files/figure-revealjs/drift2-1.png" width="960" class="r-stretch quarto-figure-center"><p class="caption">Allele frequency distribution at t = 1, with N = 100 <span class="citation" data-cites="Russo2021">(<a href="#/references" role="doc-biblioref" onclick="">Russo et al., 2021</a>)</span>.</p></section>
<section class="slide level2">

<h3 id="genetic-drift-3">Genetic Drift</h3>

<img data-src="class_5_files/figure-revealjs/drift3-1.png" width="960" class="r-stretch quarto-figure-center"><p class="caption">Allele frequencies variation after 200 generations while maintaining N = 100 <span class="citation" data-cites="Russo2021">(<a href="#/references" role="doc-biblioref" onclick="">Russo et al., 2021</a>)</span>.</p></section>
<section id="references" class="slide level2 smaller scrollable">
<h2>References</h2>


<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" data-line-spacing="2" role="list">
<div id="ref-Allendorf2022" class="csl-entry" role="listitem">
Allendorf, F. W., Funk, W. C., Aitken, S. N., Byrne, M., Luikart, G., &amp; Antunes, A. (2022). <em><span class="nocase">Conservation and the Genomics of Populations</span></em>. Oxford University Press. Retrieved from <a href="https://doi.org/10.1093/oso/9780198856566.001.0001">https://doi.org/10.1093/oso/9780198856566.001.0001</a>
</div>
<div id="ref-Chen2018" class="csl-entry" role="listitem">
Chen, N., Juric, I., Cosgrove, E. J., Bowman, R., Fitzpatrick, J. W., Schoech, S. J., Clark, A. G., et al. (2018). Allele frequency dynamics in a pedigreed natural population. <em>Proceedings of the National Academy of Sciences</em>, <em>116</em>(6), 2158–2164. Proceedings of the National Academy of Sciences. Retrieved from <a href="https://doi.org/10.1073%2Fpnas.1813852116">https://doi.org/10.1073%2Fpnas.1813852116</a>
</div>
<div id="ref-Russo2021" class="csl-entry" role="listitem">
Russo, T., Colosimo, G., Fanelli, C., Franceschini, S., &amp; Parisi, A. (2021). Un viaggio nell’ambiente r. Retrieved from <a href="https://tommaso-russo.github.io/libroR/index.html">https://tommaso-russo.github.io/libroR/index.html</a>
</div>
<div id="ref-Wiki_DoF_2023" class="csl-entry" role="listitem">
Wikipedia contributors. (2023). Degrees of freedom (statistics) — <span>Wikipedia</span><span>,</span> the free encyclopedia. Retrieved from <a href="https://en.wikipedia.org/w/index.php?title=Degrees_of_freedom_(statistics)&amp;oldid=1144691234">https://en.wikipedia.org/w/index.php?title=Degrees_of_freedom_(statistics)&amp;oldid=1144691234</a>
</div>
</div>
</section>
    </div>
  <div class="quarto-auto-generated-content" style="display: none;">
<p><img src="../../figures/Tor-Vergata_logo_cmyk.png" class="slide-logo"></p>
<div class="footer footer-default">
<p><a href="https://giulianocolosimo.github.io/biologia_della_conservazione_24-25/">Biologia della conservazione 24/25 - Giuliano Colosimo, PhD</a></p>
</div>
</div></div>

  <script>window.backupDefine = window.define; window.define = undefined;</script>
  <script src="../../site_libs/revealjs/dist/reveal.js"></script>
  <!-- reveal.js plugins -->
  <script src="../../site_libs/revealjs/plugin/quarto-line-highlight/line-highlight.js"></script>
  <script src="../../site_libs/revealjs/plugin/pdf-export/pdfexport.js"></script>
  <script src="../../site_libs/revealjs/plugin/reveal-menu/menu.js"></script>
  <script src="../../site_libs/revealjs/plugin/reveal-menu/quarto-menu.js"></script>
  <script src="../../site_libs/revealjs/plugin/reveal-chalkboard/plugin.js"></script>
  <script src="../../site_libs/revealjs/plugin/multiplex/socket.io.js"></script>
  <script src="../../site_libs/revealjs/plugin/multiplex/multiplex.js"></script>
  <script src="../../site_libs/revealjs/plugin/quarto-support/support.js"></script>
  

  <script src="../../site_libs/revealjs/plugin/notes/notes.js"></script>
  <script src="../../site_libs/revealjs/plugin/search/search.js"></script>
  <script src="../../site_libs/revealjs/plugin/zoom/zoom.js"></script>
  <script src="../../site_libs/revealjs/plugin/math/math.js"></script>
  <script>window.define = window.backupDefine; window.backupDefine = undefined;</script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
'controlsAuto': true,
'previewLinksAuto': false,
'pdfSeparateFragments': false,
'autoAnimateEasing': "ease",
'autoAnimateDuration': 1,
'autoAnimateUnmatched': true,
'jumpToSlide': true,
'menu': {"side":"left","useTextContentForMissingTitles":true,"markers":false,"loadIcons":false,"custom":[{"title":"Tools","icon":"<i class=\"fas fa-gear\"></i>","content":"<ul class=\"slide-menu-items\">\n<li class=\"slide-tool-item active\" data-item=\"0\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.fullscreen(event)\"><kbd>f</kbd> Fullscreen</a></li>\n<li class=\"slide-tool-item\" data-item=\"1\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.speakerMode(event)\"><kbd>s</kbd> Speaker View</a></li>\n<li class=\"slide-tool-item\" data-item=\"2\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>o</kbd> Slide Overview</a></li>\n<li class=\"slide-tool-item\" data-item=\"3\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.togglePdfExport(event)\"><kbd>e</kbd> PDF Export Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"4\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleScrollView(event)\"><kbd>r</kbd> Scroll View Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"5\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleChalkboard(event)\"><kbd>b</kbd> Toggle Chalkboard</a></li>\n<li class=\"slide-tool-item\" data-item=\"6\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleNotesCanvas(event)\"><kbd>c</kbd> Toggle Notes Canvas</a></li>\n<li class=\"slide-tool-item\" data-item=\"7\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.downloadDrawings(event)\"><kbd>d</kbd> Download Drawings</a></li>\n<li class=\"slide-tool-item\" data-item=\"8\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.keyboardHelp(event)\"><kbd>?</kbd> Keyboard Help</a></li>\n</ul>"}],"openButton":true},
'chalkboard': {"buttons":true,"boardmarkerWidth":5},
'multiplex': {"secret":"17318841652335252412","id":"ccf9880199d44fb5","url":"https://reveal-multiplex.glitch.me/"},
'smaller': false,
 
        // Display controls in the bottom right corner
        controls: false,

        // Help the user learn the controls by providing hints, for example by
        // bouncing the down arrow when they first encounter a vertical slide
        controlsTutorial: false,

        // Determines where controls appear, "edges" or "bottom-right"
        controlsLayout: 'edges',

        // Visibility rule for backwards navigation arrows; "faded", "hidden"
        // or "visible"
        controlsBackArrows: 'faded',

        // Display a presentation progress bar
        progress: true,

        // Display the page number of the current slide
        slideNumber: 'c/t',

        // 'all', 'print', or 'speaker'
        showSlideNumber: 'all',

        // Add the current slide number to the URL hash so that reloading the
        // page/copying the URL will return you to the same slide
        hash: true,

        // Start with 1 for the hash rather than 0
        hashOneBasedIndex: false,

        // Flags if we should monitor the hash and change slides accordingly
        respondToHashChanges: true,

        // Push each slide change to the browser history
        history: true,

        // Enable keyboard shortcuts for navigation
        keyboard: true,

        // Enable the slide overview mode
        overview: true,

        // Disables the default reveal.js slide layout (scaling and centering)
        // so that you can use custom CSS layout
        disableLayout: false,

        // Vertical centering of slides
        center: false,

        // Enables touch navigation on devices with touch input
        touch: true,

        // Loop the presentation
        loop: false,

        // Change the presentation direction to be RTL
        rtl: false,

        // see https://revealjs.com/vertical-slides/#navigation-mode
        navigationMode: 'linear',

        // Randomizes the order of slides each time the presentation loads
        shuffle: false,

        // Turns fragments on and off globally
        fragments: true,

        // Flags whether to include the current fragment in the URL,
        // so that reloading brings you to the same fragment position
        fragmentInURL: false,

        // Flags if the presentation is running in an embedded mode,
        // i.e. contained within a limited portion of the screen
        embedded: false,

        // Flags if we should show a help overlay when the questionmark
        // key is pressed
        help: true,

        // Flags if it should be possible to pause the presentation (blackout)
        pause: true,

        // Flags if speaker notes should be visible to all viewers
        showNotes: false,

        // Global override for autoplaying embedded media (null/true/false)
        autoPlayMedia: null,

        // Global override for preloading lazy-loaded iframes (null/true/false)
        preloadIframes: null,

        // Number of milliseconds between automatically proceeding to the
        // next slide, disabled when set to 0, this value can be overwritten
        // by using a data-autoslide attribute on your slides
        autoSlide: 0,

        // Stop auto-sliding after user input
        autoSlideStoppable: true,

        // Use this method for navigation when auto-sliding
        autoSlideMethod: null,

        // Specify the average time in seconds that you think you will spend
        // presenting each slide. This is used to show a pacing timer in the
        // speaker view
        defaultTiming: null,

        // Enable slide navigation via mouse wheel
        mouseWheel: false,

        // The display mode that will be used to show slides
        display: 'block',

        // Hide cursor if inactive
        hideInactiveCursor: true,

        // Time before the cursor is hidden (in ms)
        hideCursorTime: 5000,

        // Opens links in an iframe preview overlay
        previewLinks: true,

        // Transition style (none/fade/slide/convex/concave/zoom)
        transition: 'fade',

        // Transition speed (default/fast/slow)
        transitionSpeed: 'default',

        // Transition style for full page slide backgrounds
        // (none/fade/slide/convex/concave/zoom)
        backgroundTransition: 'none',

        // Number of slides away from the current that are visible
        viewDistance: 3,

        // Number of slides away from the current that are visible on mobile
        // devices. It is advisable to set this to a lower number than
        // viewDistance in order to save resources.
        mobileViewDistance: 2,

        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1600,

        height: 900,

        // Factor of the display size that should remain empty around the content
        margin: 0.1,

        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [QuartoLineHighlight, PdfExport, RevealMenu, RevealChalkboard, QuartoSupport,

          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    
    <script>
      // htmlwidgets need to know to resize themselves when slides are shown/hidden.
      // Fire the "slideenter" event (handled by htmlwidgets.js) when the current
      // slide changes (different for each slide format).
      (function () {
        // dispatch for htmlwidgets
        function fireSlideEnter() {
          const event = window.document.createEvent("Event");
          event.initEvent("slideenter", true, true);
          window.document.dispatchEvent(event);
        }

        function fireSlideChanged(previousSlide, currentSlide) {
          fireSlideEnter();

          // dispatch for shiny
          if (window.jQuery) {
            if (previousSlide) {
              window.jQuery(previousSlide).trigger("hidden");
            }
            if (currentSlide) {
              window.jQuery(currentSlide).trigger("shown");
            }
          }
        }

        // hookup for slidy
        if (window.w3c_slidy) {
          window.w3c_slidy.add_observer(function (slide_num) {
            // slide_num starts at position 1
            fireSlideChanged(null, w3c_slidy.slides[slide_num - 1]);
          });
        }

      })();
    </script>

    <script id="quarto-html-after-body" type="application/javascript">
    window.document.addEventListener("DOMContentLoaded", function (event) {
      const toggleBodyColorMode = (bsSheetEl) => {
        const mode = bsSheetEl.getAttribute("data-mode");
        const bodyEl = window.document.querySelector("body");
        if (mode === "dark") {
          bodyEl.classList.add("quarto-dark");
          bodyEl.classList.remove("quarto-light");
        } else {
          bodyEl.classList.add("quarto-light");
          bodyEl.classList.remove("quarto-dark");
        }
      }
      const toggleBodyColorPrimary = () => {
        const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
        if (bsSheetEl) {
          toggleBodyColorMode(bsSheetEl);
        }
      }
      toggleBodyColorPrimary();  
      const tabsets =  window.document.querySelectorAll(".panel-tabset-tabby")
      tabsets.forEach(function(tabset) {
        const tabby = new Tabby('#' + tabset.id);
      });
      const isCodeAnnotation = (el) => {
        for (const clz of el.classList) {
          if (clz.startsWith('code-annotation-')) {                     
            return true;
          }
        }
        return false;
      }
      const onCopySuccess = function(e) {
        // button target
        const button = e.trigger;
        // don't keep focus
        button.blur();
        // flash "checked"
        button.classList.add('code-copy-button-checked');
        var currentTitle = button.getAttribute("title");
        button.setAttribute("title", "Copied!");
        let tooltip;
        if (window.bootstrap) {
          button.setAttribute("data-bs-toggle", "tooltip");
          button.setAttribute("data-bs-placement", "left");
          button.setAttribute("data-bs-title", "Copied!");
          tooltip = new bootstrap.Tooltip(button, 
            { trigger: "manual", 
              customClass: "code-copy-button-tooltip",
              offset: [0, -8]});
          tooltip.show();    
        }
        setTimeout(function() {
          if (tooltip) {
            tooltip.hide();
            button.removeAttribute("data-bs-title");
            button.removeAttribute("data-bs-toggle");
            button.removeAttribute("data-bs-placement");
          }
          button.setAttribute("title", currentTitle);
          button.classList.remove('code-copy-button-checked');
        }, 1000);
        // clear code selection
        e.clearSelection();
      }
      const getTextToCopy = function(trigger) {
          const codeEl = trigger.previousElementSibling.cloneNode(true);
          for (const childEl of codeEl.children) {
            if (isCodeAnnotation(childEl)) {
              childEl.remove();
            }
          }
          return codeEl.innerText;
      }
      const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
        text: getTextToCopy
      });
      clipboard.on('success', onCopySuccess);
      if (window.document.getElementById('quarto-embedded-source-code-modal')) {
        const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
          text: getTextToCopy,
          container: window.document.getElementById('quarto-embedded-source-code-modal')
        });
        clipboardModal.on('success', onCopySuccess);
      }
        var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
        var mailtoRegex = new RegExp(/^mailto:/);
          var filterRegex = new RegExp("https:\/\/giulianocolosimo\.github\.com");
        var isInternal = (href) => {
            return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
        }
        // Inspect non-navigation links and adorn them if external
     	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
        for (var i=0; i<links.length; i++) {
          const link = links[i];
          if (!isInternal(link.href)) {
            // undo the damage that might have been done by quarto-nav.js in the case of
            // links that we want to consider external
            if (link.dataset.originalHref !== undefined) {
              link.href = link.dataset.originalHref;
            }
          }
        }
      function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
        const config = {
          allowHTML: true,
          maxWidth: 500,
          delay: 100,
          arrow: false,
          appendTo: function(el) {
              return el.closest('section.slide') || el.parentElement;
          },
          interactive: true,
          interactiveBorder: 10,
          theme: 'light-border',
          placement: 'bottom-start',
        };
        if (contentFn) {
          config.content = contentFn;
        }
        if (onTriggerFn) {
          config.onTrigger = onTriggerFn;
        }
        if (onUntriggerFn) {
          config.onUntrigger = onUntriggerFn;
        }
          config['offset'] = [0,0];
          config['maxWidth'] = 700;
        window.tippy(el, config); 
      }
      const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
      for (var i=0; i<noterefs.length; i++) {
        const ref = noterefs[i];
        tippyHover(ref, function() {
          // use id or data attribute instead here
          let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
          try { href = new URL(href).hash; } catch {}
          const id = href.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note) {
            return note.innerHTML;
          } else {
            return "";
          }
        });
      }
      const findCites = (el) => {
        const parentEl = el.parentElement;
        if (parentEl) {
          const cites = parentEl.dataset.cites;
          if (cites) {
            return {
              el,
              cites: cites.split(' ')
            };
          } else {
            return findCites(el.parentElement)
          }
        } else {
          return undefined;
        }
      };
      var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
      for (var i=0; i<bibliorefs.length; i++) {
        const ref = bibliorefs[i];
        const citeInfo = findCites(ref);
        if (citeInfo) {
          tippyHover(citeInfo.el, function() {
            var popup = window.document.createElement('div');
            citeInfo.cites.forEach(function(cite) {
              var citeDiv = window.document.createElement('div');
              citeDiv.classList.add('hanging-indent');
              citeDiv.classList.add('csl-entry');
              var biblioDiv = window.document.getElementById('ref-' + cite);
              if (biblioDiv) {
                citeDiv.innerHTML = biblioDiv.innerHTML;
              }
              popup.appendChild(citeDiv);
            });
            return popup.innerHTML;
          });
        }
      }
    });
    </script>
    

</body></html>