<!DOCTYPE html>
<html lang="en"><head>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-html/tabby.min.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/light-border.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-html.min.css" rel="stylesheet" data-mode="light">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles"><meta charset="utf-8">
  <meta name="generator" content="quarto-1.5.57">

  <meta name="dcterms.date" content="2024-11-21">
  <title>Biologia della Conservazione 24/25 – Biologia della Conservazione</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="../../site_libs/revealjs/dist/reset.css">
  <link rel="stylesheet" href="../../site_libs/revealjs/dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
      vertical-align: middle;
    }
    /* CSS for syntax highlighting */
    pre > code.sourceCode { white-space: pre; position: relative; }
    pre > code.sourceCode > span { line-height: 1.25; }
    pre > code.sourceCode > span:empty { height: 1.2em; }
    .sourceCode { overflow: visible; }
    code.sourceCode > span { color: inherit; text-decoration: inherit; }
    div.sourceCode { margin: 1em 0; }
    pre.sourceCode { margin: 0; }
    @media screen {
    div.sourceCode { overflow: auto; }
    }
    @media print {
    pre > code.sourceCode { white-space: pre-wrap; }
    pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
    }
    pre.numberSource code
      { counter-reset: source-line 0; }
    pre.numberSource code > span
      { position: relative; left: -4em; counter-increment: source-line; }
    pre.numberSource code > span > a:first-child::before
      { content: counter(source-line);
        position: relative; left: -1em; text-align: right; vertical-align: baseline;
        border: none; display: inline-block;
        -webkit-touch-callout: none; -webkit-user-select: none;
        -khtml-user-select: none; -moz-user-select: none;
        -ms-user-select: none; user-select: none;
        padding: 0 4px; width: 4em;
        color: #aaaaaa;
      }
    pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
    div.sourceCode
      { color: #003b4f; background-color: #f1f3f5; }
    @media screen {
    pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
    }
    code span { color: #003b4f; } /* Normal */
    code span.al { color: #ad0000; } /* Alert */
    code span.an { color: #5e5e5e; } /* Annotation */
    code span.at { color: #657422; } /* Attribute */
    code span.bn { color: #ad0000; } /* BaseN */
    code span.bu { } /* BuiltIn */
    code span.cf { color: #003b4f; font-weight: bold; } /* ControlFlow */
    code span.ch { color: #20794d; } /* Char */
    code span.cn { color: #8f5902; } /* Constant */
    code span.co { color: #5e5e5e; } /* Comment */
    code span.cv { color: #5e5e5e; font-style: italic; } /* CommentVar */
    code span.do { color: #5e5e5e; font-style: italic; } /* Documentation */
    code span.dt { color: #ad0000; } /* DataType */
    code span.dv { color: #ad0000; } /* DecVal */
    code span.er { color: #ad0000; } /* Error */
    code span.ex { } /* Extension */
    code span.fl { color: #ad0000; } /* Float */
    code span.fu { color: #4758ab; } /* Function */
    code span.im { color: #00769e; } /* Import */
    code span.in { color: #5e5e5e; } /* Information */
    code span.kw { color: #003b4f; font-weight: bold; } /* Keyword */
    code span.op { color: #5e5e5e; } /* Operator */
    code span.ot { color: #003b4f; } /* Other */
    code span.pp { color: #ad0000; } /* Preprocessor */
    code span.sc { color: #5e5e5e; } /* SpecialChar */
    code span.ss { color: #20794d; } /* SpecialString */
    code span.st { color: #20794d; } /* String */
    code span.va { color: #111111; } /* Variable */
    code span.vs { color: #20794d; } /* VerbatimString */
    code span.wa { color: #5e5e5e; font-style: italic; } /* Warning */
    /* CSS for citations */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
      margin-bottom: 0em;
    }
    .hanging-indent div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }  </style>
  <link rel="stylesheet" href="../../site_libs/revealjs/dist/theme/quarto.css">
  <link href="../../site_libs/revealjs/plugin/quarto-line-highlight/line-highlight.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-menu/menu.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-menu/quarto-menu.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-chalkboard/font-awesome/css/all.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-chalkboard/style.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">
  <style type="text/css">

  .callout {
    margin-top: 1em;
    margin-bottom: 1em;  
    border-radius: .25rem;
  }

  .callout.callout-style-simple { 
    padding: 0em 0.5em;
    border-left: solid #acacac .3rem;
    border-right: solid 1px silver;
    border-top: solid 1px silver;
    border-bottom: solid 1px silver;
    display: flex;
  }

  .callout.callout-style-default {
    border-left: solid #acacac .3rem;
    border-right: solid 1px silver;
    border-top: solid 1px silver;
    border-bottom: solid 1px silver;
  }

  .callout .callout-body-container {
    flex-grow: 1;
  }

  .callout.callout-style-simple .callout-body {
    font-size: 1rem;
    font-weight: 400;
  }

  .callout.callout-style-default .callout-body {
    font-size: 0.9rem;
    font-weight: 400;
  }

  .callout.callout-titled.callout-style-simple .callout-body {
    margin-top: 0.2em;
  }

  .callout:not(.callout-titled) .callout-body {
      display: flex;
  }

  .callout:not(.no-icon).callout-titled.callout-style-simple .callout-content {
    padding-left: 1.6em;
  }

  .callout.callout-titled .callout-header {
    padding-top: 0.2em;
    margin-bottom: -0.2em;
  }

  .callout.callout-titled .callout-title  p {
    margin-top: 0.5em;
    margin-bottom: 0.5em;
  }
    
  .callout.callout-titled.callout-style-simple .callout-content  p {
    margin-top: 0;
  }

  .callout.callout-titled.callout-style-default .callout-content  p {
    margin-top: 0.7em;
  }

  .callout.callout-style-simple div.callout-title {
    border-bottom: none;
    font-size: .9rem;
    font-weight: 600;
    opacity: 75%;
  }

  .callout.callout-style-default  div.callout-title {
    border-bottom: none;
    font-weight: 600;
    opacity: 85%;
    font-size: 0.9rem;
    padding-left: 0.5em;
    padding-right: 0.5em;
  }

  .callout.callout-style-default div.callout-content {
    padding-left: 0.5em;
    padding-right: 0.5em;
  }

  .callout.callout-style-simple .callout-icon::before {
    height: 1rem;
    width: 1rem;
    display: inline-block;
    content: "";
    background-repeat: no-repeat;
    background-size: 1rem 1rem;
  }

  .callout.callout-style-default .callout-icon::before {
    height: 0.9rem;
    width: 0.9rem;
    display: inline-block;
    content: "";
    background-repeat: no-repeat;
    background-size: 0.9rem 0.9rem;
  }

  .callout-title {
    display: flex
  }
    
  .callout-icon::before {
    margin-top: 1rem;
    padding-right: .5rem;
  }

  .callout.no-icon::before {
    display: none !important;
  }

  .callout.callout-titled .callout-body > .callout-content > :last-child {
    padding-bottom: 0.5rem;
    margin-bottom: 0;
  }

  .callout.callout-titled .callout-icon::before {
    margin-top: .5rem;
    padding-right: .5rem;
  }

  .callout:not(.callout-titled) .callout-icon::before {
    margin-top: 1rem;
    padding-right: .5rem;
  }

  /* Callout Types */

  div.callout-note {
    border-left-color: #4582ec !important;
  }

  div.callout-note .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAEU0lEQVRYCcVXTWhcVRQ+586kSUMMxkyaElstCto2SIhitS5Ek8xUKV2poatCcVHtUlFQk8mbaaziwpWgglJwVaquitBOfhQXFlqlzSJpFSpIYyXNjBNiTCck7x2/8/LeNDOZxDuEkgOXe++553zfefee+/OYLOXFk3+1LLrRdiO81yNqZ6K9cG0P3MeFaMIQjXssE8Z1JzLO9ls20MBZX7oG8w9GxB0goaPrW5aNMp1yOZIa7Wv6o2ykpLtmAPs/vrG14Z+6d4jpbSKuhdcSyq9wGMPXjonwmESXrriLzFGOdDBLB8Y6MNYBu0dRokSygMA/mrun8MGFN3behm6VVAwg4WR3i6FvYK1T7MHo9BK7ydH+1uurECoouk5MPRyVSBrBHMYwVobG2aOXM07sWrn5qgB60rc6mcwIDJtQrnrEr44kmy+UO9r0u9O5/YbkS9juQckLed3DyW2XV/qWBBB3ptvI8EUY3I9p/67OW+g967TNr3Sotn3IuVlfMLVnsBwH4fsnebJvyGm5GeIUA3jljERmrv49SizPYuq+z7c2H/jlGC+Ghhupn/hcapqmcudB9jwJ/3jvnvu6vu5lVzF1fXyZuZZ7U8nRmVzytvT+H3kilYvH09mLWrQdwFSsFEsxFVs5fK7A0g8gMZjbif4ACpKbjv7gNGaD8bUrlk8x+KRflttr22JEMRUbTUwwDQScyzPgedQHZT0xnx7ujw2jfVfExwYHwOsDTjLdJ2ebmeQIlJ7neo41s/DrsL3kl+W2lWvAga0tR3zueGr6GL78M3ifH0rGXrBC2aAR8uYcIA5gwV8zIE8onoh8u0Fca/ciF7j1uOzEnqcIm59sEXoGc0+z6+H45V1CvAvHcD7THztu669cnp+L0okAeIc6zjbM/24LgGM1gZk7jnRu1aQWoU9sfUOuhrmtaPIO3YY1KLLWZaEO5TKUbMY5zx8W9UJ6elpLwKXbsaZ4EFl7B4bMtDv0iRipKoDQT2sNQI9b1utXFdYisi+wzZ/ri/1m7QfDgEuvgUUEIJPq3DhX/5DWNqIXDOweC2wvIR90Oq3lDpdMIgD2r0dXvGdsEW5H6x6HLRJYU7C69VefO1x8Gde1ZFSJLfWS1jbCnhtOPxmpfv2LXOA2Xk2tvnwKKPFuZ/oRmwBwqRQDcKNeVQkYcOjtWVBuM/JuYw5b6isojIkYxyYAFn5K7ZBF10fea52y8QltAg6jnMqNHFBmGkQ1j+U43HMi2xMar1Nv0zGsf1s8nUsmUtPOOrbFIR8bHFDMB5zL13Gmr/kGlCkUzedTzzmzsaJXhYawnA3UmARpiYj5ooJZiUoxFRtK3X6pgNPv+IZVPcnwbOl6f+aBaO1CNvPW9n9LmCp01nuSaTRF2YxHqZ8DYQT6WsXT+RD6eUztwYLZ8rM+rcPxamv1VQzFUkzFXvkiVrySGQgJNvXHJAxiU3/NwiC03rSf05VBaPtu/Z7/B8Yn/w7eguloAAAAAElFTkSuQmCC');
  }

  div.callout-note.callout-style-default .callout-title {
    background-color: #dae6fb
  }

  div.callout-important {
    border-left-color: #d9534f !important;
  }

  div.callout-important .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAEKklEQVRYCcVXTWhcVRS+575MJym48A+hSRFr00ySRQhURRfd2HYjk2SSTokuBCkU2o0LoSKKraKIBTcuFCoidGFD08nkBzdREbpQ1EDNIv8qSGMFUboImMSZd4/f9zJv8ibJMC8xJQfO3HPPPef7zrvvvnvviIkpC9nsw0UttFunbUhpFzFtarSd6WJkStVMw5xyVqYTvkwfzuf/5FgtkVoB0729j1rjXwThS7Vio+Mo6DNnvLfahoZ+i/o32lULuJ3NNiz7q6+pyAUkJaFF6JwaM2lUJlV0MlnQn5aTRbEu0SEqHUa0A4AdiGuB1kFXRfVyg5d87+Dg4DL6m2TLAub60ilj7A1Ec4odSAc8X95sHh7+ZRPCFo6Fnp7HfU/fBng/hi10CjCnWnJjsxvDNxWw0NfV6Rv5GgP3I3jGWXumdTD/3cbEOP2ZbOZp69yniG3FQ9z1jD7bnBu9Fc2tKGC2q+uAJOQHBDRiZX1x36o7fWBs7J9ownbtO+n0/qWkvW7UPIfc37WgT6ZGR++EOJyeQDSb9UB+DZ1G6DdLDzyS+b/kBCYGsYgJbSQHuThGKRcw5xdeQf8YdNHsc6ePXrlSYMBuSIAFTGAtQo+VuALo4BX83N190NWZWbynBjhOHsmNfFWLeL6v+ynsA58zDvvAC8j5PkbOcXCMg2PZFk3q8MjI7WAG/Dp9AwP7jdGBOOQkAvlFUB+irtm16I1Zw9YBcpGTGXYmk3kQIC/Cds55l+iMI3jqhjAuaoe+am2Jw5GT3Nbz3CkE12NavmzN5+erJW7046n/CH1RO/RVa8lBLozXk9uqykkGAyRXLWlLv5jyp4RFsG5vGVzpDLnIjTWgnRy2Rr+tDKvRc7Y8AyZq10jj8DqXdnIRNtFZb+t/ZRtXcDiVnzpqx8mPcDWxgARUqx0W1QB9MeUZiNrV4qP+Ehc+BpNgATsTX8ozYKL2NtFYAHc84fG7ndxUPr+AR/iQSns7uSUufAymwDOb2+NjK27lEFocm/EE2WpyIy/Hi66MWuMKJn8RvxIcj87IM5Vh9663ziW36kR0HNenXuxmfaD8JC7tfKbrhFr7LiZCrMjrzTeGx+PmkosrkNzW94ObzwocJ7A1HokLolY+AvkTiD/q1H0cN48c5EL8Crkttsa/AXQVDmutfyku0E7jShx49XqV3MFK8IryDhYVbj7Sj2P2eBxwcXoe8T8idsKKPRcnZw1b+slFTubwUwhktrfnAt7J++jwQtLZcm3sr9LQrjRzz6cfMv9aLvgmnAGvpoaGLxM4mAEaLV7iAzQ3oU0IvD5x9ix3yF2RAAuYAOO2f7PEFWCXZ4C9Pb2UsgDeVnFSpbFK7/IWu7TPTvBqzbGdCHOJQSxiEjt6IyZmxQyEJHv6xyQsYk//moVFsN2zP6fRImjfq7/n/wFDguUQFNEwugAAAABJRU5ErkJggg==');
  }

  div.callout-important.callout-style-default .callout-title {
    background-color: #f7dddc
  }

  div.callout-warning {
    border-left-color: #f0ad4e !important;
  }

  div.callout-warning .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAETklEQVRYCeVWW2gcVRg+58yaTUnizqbipZeX4uWhBEniBaoUX1Ioze52t7sRq6APio9V9MEaoWlVsFasRq0gltaAPuxms8lu0gcviE/FFOstVbSIxgcv6SU7EZqmdc7v9+9mJtNks51NTUH84ed889/PP+cmxP+d5FIbMJmNbpREu4WUkiTtCicKny0l1pIKmBzovF2S+hIJHX8iEu3hZJ5lNZGqyRrGSIQpq15AzF28jgpeY6yk6GVdrfFqdrD6Iw+QlB8g0YS2g7dyQmXM/IDhBhT0UCiRf59lfqmmDvzRt6kByV/m4JjtzuaujMUM2c5Z2d6JdKrRb3K2q6mA+oYVz8JnDdKPmmNthzkAk/lN63sYPgevrguc72aZX/L9C6x09GYyxBgCX4NlvyGUHOKELlm5rXeR1kchuChJt4SSwyddZRXgvwMGvYo4QSlk3/zkHD8UHxwVJA6zjZZqP8v8kK8OWLnIZtLyCAJagYC4rTGW/9Pqj92N/c+LUaAj27movwbi19tk/whRCIE7Q9vyI6yvRpftAKVTdUjOW40X3h5OXsKCdmFcx0xlLJoSuQngnrJe7Kcjm4OMq9FlC7CMmScQANuNvjfP3PjGXDBaUQmbp296S5L4DrpbrHN1T87ZVEZVCzg1FF0Ft+dKrlLukI+/c9ENo+TvlTDbYFvuKPtQ9+l052rXrgKoWkDAFnvh0wTOmYn8R5f4k/jN/fZiCM1tQx9jQQ4ANhqG4hiL0qIFTGViG9DKB7GYzgubnpofgYRwO+DFjh0Zin2m4b/97EDkXkc+f6xYAPX0KK2I/7fUQuwzuwo/L3AkcjugPNixC8cHf0FyPjWlItmLxWw4Ou9YsQCr5fijMGoD/zpdRy95HRysyXA74MWOnscpO4j2y3HAVisw85hX5+AFBRSHt4ShfLFkIMXTqyKFc46xdzQM6XbAi702a7sy04J0+feReMFKp5q9esYLCqAZYw/k14E/xcLLsFElaornTuJB0svMuJINy8xkIYuL+xPAlWRceH6+HX7THJ0djLUom46zREu7tTkxwmf/FdOZ/sh6Q8qvEAiHpm4PJ4a/doJe0gH1t+aHRgCzOvBvJedEK5OFE5jpm4AGP2a8Dxe3gGJ/pAutug9Gp6he92CsSsWBaEcxGx0FHytmIpuqGkOpldqNYQK8cSoXvd+xLxXADw0kf6UkJNFtdo5MOgaLjiQOQHcn+A6h5NuL2s0qsC2LOM75PcF3yr5STuBSAcGG+meA14K/CI21HcS4LBT6tv0QAh8Dr5l93AhZzG5ZJ4VxAqdZUEl9z7WJ4aN+svMvwHHL21UKTd1mqvChH7/Za5xzXBBKrUcB0TQ+Ulgkfbi/H/YT5EptrGzsEK7tR1B7ln9BBwckYfMiuSqklSznIuoIIOM42MQO+QnduCoFCI0bpkzjCjddHPN/F+2Yu+sd9bKNpVwHhbS3LluK/0zgfwD0xYI5dXuzlQAAAABJRU5ErkJggg==');
  }

  div.callout-warning.callout-style-default .callout-title {
    background-color: #fcefdc
  }

  div.callout-tip {
    border-left-color: #02b875 !important;
  }

  div.callout-tip .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAADr0lEQVRYCe1XTWgTQRj9ZjZV8a9SPIkKgj8I1bMHsUWrqYLVg4Ue6v9BwZOxSYsIerFao7UiUryIqJcqgtpimhbBXoSCVxUFe9CTiogUrUp2Pt+3aUI2u5vdNh4dmMzOzHvvezuz8xNFM0mjnbXaNu1MvFWRXkXEyE6aYOYJpdW4IXuA4r0fo8qqSMDBU0v1HJUgVieAXxzCsdE/YJTdFcVIZQNMyhruOMJKXYFoLfIfIvVIMWdsrd+Rpd86ZmyzzjJmLStqRn0v8lzkb4rVIXvnpScOJuAn2ACC65FkPzEdEy4TPWRLJ2h7z4cArXzzaOdKlbOvKKX25Wl00jSnrwVxAg3o4dRxhO13RBSdNvH0xSARv3adTXbBdTf64IWO2vH0LT+cv4GR1DJt+DUItaQogeBX/chhbTBxEiZ6gftlDNXTrvT7co4ub5A6gp9HIcHvzTa46OS5fBeP87Qm0fQkr4FsYgVQ7Qg+ZayaDg9jhg1GkWj8RG6lkeSacrrHgDaxdoBiZPg+NXV/KifMuB6//JmYH4CntVEHy/keA6x4h4CU5oFy8GzrBS18cLJMXcljAKB6INjWsRcuZBWVaS3GDrqB7rdapVIeA+isQ57Eev9eCqzqOa81CY05VLd6SamW2wA2H3SiTbnbSxmzfp7WtKZkqy4mdyAlGx7ennghYf8voqp9cLSgKdqNfa6RdRsAAkPwRuJZNbpByn+RrJi1RXTwdi8RQF6ymDwGMAtZ6TVE+4uoKh+MYkcLsT0Hk8eAienbiGdjJHZTpmNjlbFJNKDVAp2fJlYju6IreQxQ08UJDNYdoLSl6AadO+fFuCQqVMB1NJwPm69T04Wv5WhfcWyfXQB+wXRs1pt+nCknRa0LVzSA/2B+a9+zQJadb7IyyV24YAxKp2Jqs3emZTuNnKxsah+uabKbMk7CbTgJx/zIgQYErIeTKRQ9yD9wxVof5YolPHqaWo7TD6tJlh7jQnK5z2n3+fGdggIOx2kaa2YI9QWarc5Ce1ipNWMKeSG4DysFF52KBmTNMmn5HqCFkwy34rDg05gDwgH3bBi+sgFhN/e8QvRn8kbamCOhgrZ9GJhFDgfcMHzFb6BAtjKpFhzTjwv1KCVuxHvCbsSiEz4CANnj84cwHdFXAbAOJ4LTSAawGWFn5tDhLMYz6nWeU2wJfIhmIJBefcd/A5FWQWGgrWzyORZ3Q6HuV+Jf0Bj+BTX69fm1zWgK7By1YTXchFDORywnfQ7GpzOo6S+qECrsx2ifVQAAAABJRU5ErkJggg==');
  }

  div.callout-tip.callout-style-default .callout-title {
    background-color: #ccf1e3
  }

  div.callout-caution {
    border-left-color: #fd7e14 !important;
  }

  div.callout-caution .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAACV0lEQVRYCdVWzWoUQRCuqp2ICBLJXgITZL1EfQDBW/bkzUMUD7klD+ATSHBEfAIfQO+iXsWDxJsHL96EHAwhgzlkg8nBg25XWb0zIb0zs9muYYWkoKeru+vn664fBqElyZNuyh167NXJ8Ut8McjbmEraKHkd7uAnAFku+VWdb3reSmRV8PKSLfZ0Gjn3a6Xlcq9YGb6tADjn+lUfTXtVmaZ1KwBIvFI11rRXlWlatwIAAv2asaa9mlB9wwygiDX26qaw1yYPzFXg2N1GgG0FMF8Oj+VIx7E/03lHx8UhvYyNZLN7BwSPgekXXLribw7w5/c8EF+DBK5idvDVYtEEwMeYefjjLAdEyQ3M9nfOkgnPTEkYU+sxMq0BxNR6jExrAI31H1rzvLEfRIdgcv1XEdj6QTQAS2wtstEALLG1yEZ3QhH6oDX7ExBSFEkFINXH98NTrme5IOaaA7kIfiu2L8A3qhH9zRbukdCqdsA98TdElyeMe5BI8Rs2xHRIsoTSSVFfCFCWGPn9XHb4cdobRIWABNf0add9jakDjQJpJ1bTXOJXnnRXHRf+dNL1ZV1MBRCXhMbaHqGI1JkKIL7+i8uffuP6wVQAzO7+qVEbF6NbS0LJureYcWXUUhH66nLR5rYmva+2tjRFtojkM2aD76HEGAD3tPtKM309FJg5j/K682ywcWJ3PASCcycH/22u+Bh7Aa0ehM2Fu4z0SAE81HF9RkB21c5bEn4Dzw+/qNOyXr3DCTQDMBOdhi4nAgiFDGCinIa2owCEChUwD8qzd03PG+qdW/4fDzjUMcE1ZpIAAAAASUVORK5CYII=');
  }

  div.callout-caution.callout-style-default .callout-title {
    background-color: #ffe5d0
  }

  </style>
  <style type="text/css">
    .reveal div.sourceCode {
      margin: 0;
      overflow: auto;
    }
    .reveal div.hanging-indent {
      margin-left: 1em;
      text-indent: -1em;
    }
    .reveal .slide:not(.center) {
      height: 100%;
    }
    .reveal .slide.scrollable {
      overflow-y: auto;
    }
    .reveal .footnotes {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide .absolute {
      position: absolute;
      display: block;
    }
    .reveal .footnotes ol {
      counter-reset: ol;
      list-style-type: none; 
      margin-left: 0;
    }
    .reveal .footnotes ol li:before {
      counter-increment: ol;
      content: counter(ol) ". "; 
    }
    .reveal .footnotes ol li > p:first-child {
      display: inline-block;
    }
    .reveal .slide ul,
    .reveal .slide ol {
      margin-bottom: 0.5em;
    }
    .reveal .slide ul li,
    .reveal .slide ol li {
      margin-top: 0.4em;
      margin-bottom: 0.2em;
    }
    .reveal .slide ul[role="tablist"] li {
      margin-bottom: 0;
    }
    .reveal .slide ul li > *:first-child,
    .reveal .slide ol li > *:first-child {
      margin-block-start: 0;
    }
    .reveal .slide ul li > *:last-child,
    .reveal .slide ol li > *:last-child {
      margin-block-end: 0;
    }
    .reveal .slide .columns:nth-child(3) {
      margin-block-start: 0.8em;
    }
    .reveal blockquote {
      box-shadow: none;
    }
    .reveal .tippy-content>* {
      margin-top: 0.2em;
      margin-bottom: 0.7em;
    }
    .reveal .tippy-content>*:last-child {
      margin-bottom: 0.2em;
    }
    .reveal .slide > img.stretch.quarto-figure-center,
    .reveal .slide > img.r-stretch.quarto-figure-center {
      display: block;
      margin-left: auto;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-left,
    .reveal .slide > img.r-stretch.quarto-figure-left  {
      display: block;
      margin-left: 0;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-right,
    .reveal .slide > img.r-stretch.quarto-figure-right  {
      display: block;
      margin-left: auto;
      margin-right: 0; 
    }
  </style>
<meta property="og:title" content="Biologia della Conservazione – Biologia della Conservazione 24/25">
<meta property="og:description" content="Class 6 - \(N_{e}\) and Population structure">
<meta property="og:site_name" content="Biologia della Conservazione 24/25">
<meta property="og:locale" content="en_US">
</head>
<body class="quarto-light">
  <div class="reveal">
    <div class="slides">

<section id="title-slide" class="quarto-title-block center">
  <h1 class="title">Biologia della Conservazione</h1>
  <p class="subtitle">Class 6 - <span class="math inline">\(N_{e}\)</span> and Population structure</p>

<div class="quarto-title-authors">
</div>

  <p class="date">2024-11-21</p>
</section>
<section id="genetic-drift" class="slide level2">
<h2>Genetic Drift</h2>
<h3 id="simulating-the-average-time-of-allele-fixation-in-our-example">Simulating the average time of allele fixation in our example</h3>
<div class="cell" data-layout-align="center">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a></a><span class="co"># simulare il tempo medio di fissazione degli alleli</span></span>
<span id="cb1-2"><a></a>pop.ls <span class="ot">&lt;-</span> <span class="fu">list</span>(<span class="at">geneA =</span> <span class="fu">c</span>(<span class="fu">rep</span>(<span class="st">"A"</span>,<span class="dv">50</span>), <span class="fu">rep</span>(<span class="st">"a"</span>,<span class="dv">50</span>)), </span>
<span id="cb1-3"><a></a>               <span class="at">geneB =</span> <span class="fu">c</span>(<span class="fu">rep</span>(<span class="st">"B"</span>,<span class="dv">50</span>), <span class="fu">rep</span>(<span class="st">"b"</span>,<span class="dv">50</span>)),</span>
<span id="cb1-4"><a></a>               <span class="at">geneC =</span> <span class="fu">c</span>(<span class="fu">rep</span>(<span class="st">"C"</span>,<span class="dv">50</span>), <span class="fu">rep</span>(<span class="st">"c"</span>,<span class="dv">50</span>)))</span>
<span id="cb1-5"><a></a></span>
<span id="cb1-6"><a></a></span>
<span id="cb1-7"><a></a><span class="co"># pop.ls  =  lista contenente i geni della popolazione</span></span>
<span id="cb1-8"><a></a><span class="co"># num.sim = numero di volte la simulazione deve girare</span></span>
<span id="cb1-9"><a></a>sim.gen2fix <span class="ot">&lt;-</span> <span class="cf">function</span> (pop.ls, num.sims) {</span>
<span id="cb1-10"><a></a>  </span>
<span id="cb1-11"><a></a>  <span class="co"># numero totale di loci per ogni gene   </span></span>
<span id="cb1-12"><a></a>  n <span class="ot">&lt;-</span> <span class="fu">length</span>(pop.ls[[<span class="dv">1</span>]])</span>
<span id="cb1-13"><a></a>  <span class="co"># numero di geni differenti nella popolazione</span></span>
<span id="cb1-14"><a></a>  n.genes <span class="ot">&lt;-</span> <span class="fu">length</span>(pop.ls)</span>
<span id="cb1-15"><a></a>  <span class="co"># lista vuota per i risultati della simulazione</span></span>
<span id="cb1-16"><a></a>  generations <span class="ot">&lt;-</span> <span class="fu">vector</span>(<span class="at">mode =</span> <span class="st">"list"</span>, <span class="at">length =</span> <span class="fu">length</span>(pop.ls))</span>
<span id="cb1-17"><a></a>  <span class="cf">for</span> (h <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span>n.genes){</span>
<span id="cb1-18"><a></a>    generations[[h]] <span class="ot">&lt;-</span> <span class="fu">rep</span>(<span class="dv">0</span>, num.sims)</span>
<span id="cb1-19"><a></a>  }</span>
<span id="cb1-20"><a></a>  <span class="fu">names</span>(generations) <span class="ot">&lt;-</span> <span class="fu">names</span>(pop.ls)</span>
<span id="cb1-21"><a></a>  </span>
<span id="cb1-22"><a></a>  <span class="co"># ciclo di simulazione </span></span>
<span id="cb1-23"><a></a>  <span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span>num.sims) {</span>
<span id="cb1-24"><a></a>    temp.pop.ls <span class="ot">&lt;-</span> pop.ls</span>
<span id="cb1-25"><a></a>    <span class="cf">for</span> (y <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span>n.genes) {</span>
<span id="cb1-26"><a></a>      <span class="cf">while</span> (<span class="fu">max</span>(<span class="fu">table</span>(temp.pop.ls[[y]])) <span class="sc">&lt;</span> n) {</span>
<span id="cb1-27"><a></a>        temp.pop.ls[[y]] <span class="ot">&lt;-</span> <span class="fu">sample</span>(temp.pop.ls[[y]],</span>
<span id="cb1-28"><a></a>                                   <span class="dv">100</span>, <span class="at">replace =</span> T)</span>
<span id="cb1-29"><a></a>        generations[[y]][i] <span class="ot">&lt;-</span> generations[[y]][i] <span class="sc">+</span> <span class="dv">1</span></span>
<span id="cb1-30"><a></a>      }</span>
<span id="cb1-31"><a></a>    }</span>
<span id="cb1-32"><a></a>  }</span>
<span id="cb1-33"><a></a>  </span>
<span id="cb1-34"><a></a>    <span class="co"># statistiche descrittive dei risultati</span></span>
<span id="cb1-35"><a></a>  df <span class="ot">&lt;-</span> <span class="fu">rbind</span>(<span class="fu">data.frame</span>(<span class="fu">lapply</span>(generations, mean)),</span>
<span id="cb1-36"><a></a>        <span class="fu">data.frame</span>(<span class="fu">lapply</span>(generations, sd))) </span>
<span id="cb1-37"><a></a>  <span class="fu">rownames</span>(df) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"mean"</span>, <span class="st">"sd"</span>)</span>
<span id="cb1-38"><a></a></span>
<span id="cb1-39"><a></a>  <span class="co"># ciclo per plottare i risultati</span></span>
<span id="cb1-40"><a></a>  <span class="co"># modificare se i geni da plottare sono &gt; 4</span></span>
<span id="cb1-41"><a></a>  <span class="fu">par</span>(<span class="at">mfrow =</span> <span class="fu">c</span>(<span class="dv">2</span>,<span class="dv">2</span>))</span>
<span id="cb1-42"><a></a>  <span class="cf">for</span> (z <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span>n.genes) {</span>
<span id="cb1-43"><a></a>    graph <span class="ot">&lt;-</span> generations[[z]]</span>
<span id="cb1-44"><a></a>    <span class="fu">hist</span>(graph, </span>
<span id="cb1-45"><a></a>         <span class="at">main =</span> <span class="fu">paste</span>(<span class="st">"Num. of generations to fix "</span>,</span>
<span id="cb1-46"><a></a>                             <span class="fu">names</span>(generations[z])),</span>
<span id="cb1-47"><a></a>         <span class="at">xlab =</span> <span class="st">"Generations"</span>)</span>
<span id="cb1-48"><a></a>    <span class="fu">abline</span>(<span class="at">v=</span><span class="fu">mean</span>(graph), <span class="at">lwd=</span><span class="dv">2</span>, <span class="at">lty=</span><span class="dv">2</span>, <span class="at">col=</span><span class="st">"red"</span>)</span>
<span id="cb1-49"><a></a>  }</span>
<span id="cb1-50"><a></a>  </span>
<span id="cb1-51"><a></a>  <span class="co">#return(df)</span></span>
<span id="cb1-52"><a></a>}</span>
<span id="cb1-53"><a></a></span>
<span id="cb1-54"><a></a><span class="fu">sim.gen2fix</span>(pop.ls, <span class="dv">5000</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>

</div>
<img data-src="class_6_files/figure-revealjs/drift4-1.png" class="quarto-figure quarto-figure-center r-stretch" width="960"></section>
<section id="size-matters" class="slide level2">
<h2>Size matters</h2>

<img data-src="class_6_files/figure-revealjs/drift5-1.png" width="960" class="r-stretch"></section>
<section id="population-size" class="slide level2">
<h2>Population size</h2>
<blockquote>
<p>There can be no doubt that genetic variation is related to population size, as <span class="citation" data-cites="Soule1973">Soule (<a href="#/references" role="doc-biblioref" onclick="">1973</a>)</span> proposed. Small population size reduces the evolutionary potential of wildlife species <span class="citation" data-cites="Frankham1996">(<a href="#/references" role="doc-biblioref" onclick="">Frankham, 1996</a>)</span>.</p>
</blockquote>
<ul>
<li>The importance of maintaining a large population size is pretty intuitive, but may not be enough!</li>
<li>Many populations of insects and small mammals naturally fluctuate in their “census” population size, but it is when they are at their minimum that the genetic processes affect them the most</li>
<li>Often the census population counts juveniles and senescent individuals that do not really contribute to the reproductive output of next generations</li>
<li>Within adult and reproductive individuals there may be significant variance in the number of offspring sired by males and females</li>
</ul>
</section>
<section class="slide level2">

<ul>
<li>Different reproductive strategies (monogamy VS. polygamy) may as well influence the genetic output generation after generation</li>
<li>Therefore it is necessary to distinguish between census population and an hypothetical idealized population</li>
<li>We can standardize this concept of idealized population by describing it in term of <strong>effective population size</strong> (<span class="math inline">\(N_e\)</span>), and it can be defined as the actual number of breeding individuals needed in order to see the same variation of a critical parameter (such as genetic diversity) measured in the real population</li>
<li>The effective size of a population determines the rate of change in composition of a population caused by genetic drift (the random sampling of genetic variants) <span class="citation" data-cites="Charlesworth2009a">(<a href="#/references" role="doc-biblioref" onclick="">Charlesworth, 2009, p. 195</a>)</span></li>
<li>We hereby introduce another model that is extremely useful in population and conservation genetics: the Wright-Fisher population model <span class="citation" data-cites="Wright1931">(<a href="#/references" role="doc-biblioref" onclick="">Wright, 1931</a>)</span></li>
</ul>
</section>
<section id="wright-fisher-population-model" class="slide level2">
<h2>Wright-Fisher population model</h2>
<p>Assumptions of the model:</p>
<ul>
<li>All individuals in the population are considered adults and capable of breeding</li>
<li>Individuals are hermafroditic (self fertilization is possible)</li>
<li>The number of breeding individuals is constant in all generations and they breed randomly</li>
<li>New individuals are formed each generation by random sampling, with replacement, of the gametes from the parents</li>
<li>The parents die after mating so each generation is only composed by new individuals (discrete generations)</li>
<li>There is no selection at any life stage and mutation is ignored</li>
<li>When <span class="math inline">\(N\)</span> is sufficiently large this means that each individual contributes to the next generation a number of offspring that is approximated by a Poisson distribution</li>
</ul>
</section>
<section id="the-wright-fisher-model-used-in-wild-populations" class="slide level2">
<h2>The Wright-Fisher model used in wild populations</h2>
<ul>
<li>Any characteristic of a real population that deviates from the characteristic of the ideal population will make the census size <span class="math inline">\(N\)</span> to be different than the effective population size <span class="math inline">\(N_e\)</span></li>
<li>An important measure in conservation is the ratio <span class="math inline">\(\frac{N_e}{N}\)</span></li>
<li>There are multiple evidence that in most natural population this ratio is usually much less than 1</li>
<li>The primary factors in natural populations that are influencing this ratio are:
<ul>
<li>fluctuating population size over multiple generations</li>
<li>high variance in family size</li>
<li>unequal ratios between sexes</li>
</ul></li>
<li>Demographic and genetic approaches to estimate <span class="math inline">\(N_e\)</span></li>
</ul>
</section>
<section id="estimating-n_e-unequal-sex-ratio" class="slide level2">
<h2>Estimating <span class="math inline">\(N_e\)</span>: Unequal sex ratio</h2>
<ul>
<li>What is the size of the ideal population that will lose heterozygosity at the same rate of the real population that has unequal sex ratio?</li>
<li>When selfing is not permitted, gametes in the progeny cannot come from the same individual, therefore the effective population size in a population with separate sexes is equal to the probability that the gametes in the progeny comes from the same grandparent.</li>
<li>The probability that the two uniting gametes in an individual will come from the same male (or female) grandparent is <span class="math inline">\(\frac{1}{4}\)</span>.</li>
<li>One-half of the time uniting gametes will come from a grandmother and a grandfather, and 1/4 of the time both gametes will come from the same grandparent <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</li>
<li>Therefore, the combined probability that both gametes come from the same grandparent is <span class="math inline">\(\frac{1}{4}*\frac{1}{N_m}= \frac{1}{4N_m}\)</span> for grandfathers and <span class="math inline">\(\frac{1}{4}*\frac{1}{N_f}= \frac{1}{4N_f}\)</span> for grandmothers.</li>
</ul>
</section>
<section class="slide level2">

<ul>
<li>Putting everything together, we can conclude that the combined probability of inheriting both gametes from the same grandparent is</li>
</ul>
<p><span class="math display">\[
\label{uneqsex}
\tag{6.1}
\frac{1}{N_e}=\frac{1}{4N_f}+\frac{1}{4N_m}  
\]</span></p>
<ul>
<li>We can solve this equation and get to <span class="math inline">\(N_e\)</span>:</li>
</ul>
<p><span class="math display">\[
\label{uneqsex_bis}
\tag{6.2}
N_e=\frac{4N_{f}N_{m}}{N_{f}+N_{m}}
\]</span></p>
<ul>
<li>In many natural populations of species that are harvested for food one of the two sexes is generally more abundantly hunted, and this can produce a highly skewed sex ratio <span class="citation" data-cites="Lamb2010">(<a href="#/references" role="doc-biblioref" onclick="">Lamb, 2010</a>)</span>.</li>
</ul>
</section>
<section class="slide level2">

<ul>
<li>According to <span class="citation" data-cites="Donald2007">Donald (<a href="#/references" role="doc-biblioref" onclick="">2007</a>)</span> there usually is an excess of females in mammals and a deficiency in males</li>
</ul>

<img data-src="./unequalsexratio.jpg" width="250" class="r-stretch quarto-figure-center"><p class="caption">Effects of unequal sex ratio on <span class="math inline">\(N_e/N\)</span>. After <span class="citation" data-cites="Frankham2010">Frankham, Ballou, &amp; Briscoe (<a href="#/references" role="doc-biblioref" onclick="">2010</a>)</span>, p.249</p></section>
<section id="estimating-n_e-unequal-probability-of-producing-offspring" class="slide level2">
<h2>Estimating <span class="math inline">\(N_e\)</span>: Unequal probability of producing offspring</h2>
<ul>
<li>In real populations parents very rarely have the same probability of producing offspring.</li>
<li>This variation results in a greater proportion of the progeny being produced by a smaller group of individuals compared to the number of adults potentially able to reproduce <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</li>
<li>We can model the non-random progeny contribution using <span class="citation" data-cites="Wright1939">Wright (<a href="#/references" role="doc-biblioref" onclick="">1939</a>)</span>.</li>
<li>Lets imagine a population of <span class="math inline">\(N\)</span> individuals each contributing a varying number <span class="math inline">\(k\)</span> of gametes to the next generation of the same size (<span class="math inline">\(N\)</span>). The mean number of gametes contributed by individual is <span class="math inline">\(\bar{k}=2\)</span>. We can calculate the variance associated to this number using</li>
</ul>
<p><span class="math display">\[
\label{varOffsp}
\tag{6.3}
V_k = \frac{\sum_{i=1}^N (k_i-2)^2}{N}
\]</span></p>
</section>
<section class="slide level2">

<ul>
<li>The proportion of cases in which two random gametes will come from the same parent is then</li>
</ul>
<p><span class="math display">\[
\label{varOffsp2}
\tag{6.4}
\frac{\sum_{i=1}^N k_i(k_i-1)}{2N(2N-1)}=\frac{2+V_k}{4N-2}
\]</span></p>
<ul>
<li>By now we know that the effective population size can be intended as the reciprocal of the probability that two gametes come from the same parent, and we can rewrite this equation accordingly</li>
</ul>
<p><span class="math display">\[
\label{varOffsp3}
\tag{6.5}
N_e=\frac{4N-2}{2+V_k}
\]</span></p>
</section>
<section class="slide level2">

<ul>
<li>Random variation of <span class="math inline">\(k\)</span> will produce a distribution that approximates the Poisson’s (i.e., with mean equal to the variance)</li>
<li>Thus, <span class="math inline">\(V_k = \bar{k} = 2\)</span> and <span class="math inline">\(N_e = N\)</span> for the idealized population</li>
<li>As <span class="math inline">\(V_k\)</span> increases, the effective population size decreases</li>
<li>An interesting result is that the effective population size will be larger than the actual population size if <span class="math inline">\(V_k &lt; 2\)</span></li>
<li>In captive breeding where we can control reproduction, we may nearly double the effective population size by making sure that all individuals contribute equal number of progeny <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span></li>
</ul>
</section>
<section class="slide level2">

<div class="columns">
<div class="column" style="width:50%;">
<ul>
<li>Three hypothetical populations of constant size N = 10. In <strong>A</strong>, only one pair of mates reproduces successfully. In <strong>B</strong>, each of the five pairs produces two offspring (no variance in reproductive success). <strong>C</strong> presents intermediate variance</li>
</ul>
<p><img data-src="./class_8_2.png"></p>
</div><div class="column" style="width:50%;">
<ul>
<li>Estimation of effective population size for populations in previous table with high, low, and intermediate variability in family size. <span class="math inline">\(N_e\)</span> is estimated using equation <span class="math inline">\(\ref{varOffsp3}\)</span></li>
</ul>
<p><img data-src="./class_8_3.png"></p>
</div></div>
</section>
<section id="estimating-n_e-fluctuating-population-size" class="slide level2">
<h2>Estimating <span class="math inline">\(N_e\)</span>: Fluctuating population size</h2>
<ul>
<li>The rate of loss of heterozygosity (<span class="math inline">\(\frac{1}{2N}\)</span>) is the reciprocal of the population size (<span class="math inline">\(\frac{1}{N}\)</span>)</li>
<li>Many generations with reduced population size will influence greatly the loss of heterozygosity and influcence <span class="math inline">\(N_e\)</span>.</li>
<li>Consider a population that fluctuates in the following way: <span class="math inline">\(N_1 = 100\)</span>, <span class="math inline">\(N_2 = 2\)</span>, and <span class="math inline">\(N_3 = 100\)</span></li>
<li>We can use the armonic mean to calculate the loss of heterozygosity and the effective population size in this situation</li>
</ul>
<p><span class="math display">\[
\label{fluctPop}
\tag{6.6}
N_e=\frac{t}{\sum_{i=1}^t{(\frac{1}{N_i})}}
\]</span></p>
<p><span class="math display">\[
N_e=\frac{3}{(\frac{1}{100}+\frac{1}{2}+\frac{1}{100})}=5.77
\]</span></p>
</section>
<section id="estimating-n_e-genetic-approaches-and-the-coalescent-theory" class="slide level2">
<h2>Estimating <span class="math inline">\(N_e\)</span>: Genetic approaches and the coalescent theory</h2>
<ul>
<li>DNA retains information on the evolutionary history of individuals</li>
<li>We can use basic probability theory to extrapolate information, including information on <span class="math inline">\(N_e\)</span> from DNA markers</li>
<li>The coalescent allows to study the loss of genetic diversity and to use this information to infer effective population size</li>
<li>To coalesce means “unite” or “fuse”</li>
<li>To better understand this approach we can look at the example of mitochondrial DNA (mtDNA) which is haploid, maternally inherited and it does not undergo recombination</li>
<li>Coalescent theory provides a powerful framework to study the effects of genetic drift, natural selection, mutation, and gene flow in natural populations <span class="citation" data-cites="Allendorf2022 Rosenberg2002">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>; <a href="#/references" role="doc-biblioref" onclick="">Rosenberg &amp; Nordborg, 2002</a>)</span></li>
</ul>
</section>
<section class="slide level2">


<img data-src="class_6_files/figure-revealjs/unnamed-chunk-1-1.png" width="960" class="r-stretch quarto-figure-center"><p class="caption">Population of constant size (<span class="math inline">\(N\)</span>) and discrete generations (<span class="math inline">\(G\{1,2, ..., i-1, i\}\)</span>). We further simplify the situation by assuming all individuals are potentially reproducing (<span class="math inline">\(N = N_e\)</span>).</p></section>
<section class="slide level2">


<img data-src="class_6_files/figure-revealjs/unnamed-chunk-2-1.png" width="960" class="r-stretch quarto-figure-center"><p class="caption">Population of constant size (<span class="math inline">\(N\)</span>) and discrete generations (<span class="math inline">\(G\{1,2, ..., i-1, i\}\)</span>). We further simplify the situation by assuming all individuals are potentially reproducing (<span class="math inline">\(N = N_e\)</span>).</p></section>
<section class="slide level2">


<img data-src="class_6_files/figure-revealjs/unnamed-chunk-3-1.png" width="960" class="r-stretch quarto-figure-center"><p class="caption">Population of constant size (<span class="math inline">\(N\)</span>) and discrete generations (<span class="math inline">\(G\{1,2, ..., i-1, i\}\)</span>). We further simplify the situation by assuming all individuals are potentially reproducing (<span class="math inline">\(N = N_e\)</span>).</p></section>
<section class="slide level2">


<img data-src="class_6_files/figure-revealjs/unnamed-chunk-4-1.png" width="960" class="r-stretch quarto-figure-center"><p class="caption">Population of constant size (<span class="math inline">\(N\)</span>) and discrete generations (<span class="math inline">\(G\{1,2, ..., i-1, i\}\)</span>). We further simplify the situation by assuming all individuals are potentially reproducing (<span class="math inline">\(N = N_e\)</span>).</p></section>
<section class="slide level2">


<img data-src="class_6_files/figure-revealjs/unnamed-chunk-5-1.png" width="960" class="r-stretch quarto-figure-center"><p class="caption">Population of constant size (<span class="math inline">\(N\)</span>) and discrete generations (<span class="math inline">\(G\{1,2, ..., i-1, i\}\)</span>). We further simplify the situation by assuming all individuals are potentially reproducing (<span class="math inline">\(N = N_e\)</span>).</p></section>
<section class="slide level2">


<img data-src="class_6_files/figure-revealjs/unnamed-chunk-6-1.png" width="960" class="r-stretch quarto-figure-center"><p class="caption">Population of constant size (<span class="math inline">\(N\)</span>) and discrete generations (<span class="math inline">\(G\{1,2, ..., i-1, i\}\)</span>). We further simplify the situation by assuming all individuals are potentially reproducing (<span class="math inline">\(N = N_e\)</span>).</p></section>
<section class="slide level2">


<img data-src="class_6_files/figure-revealjs/unnamed-chunk-7-1.png" width="960" class="r-stretch quarto-figure-center"><p class="caption">Tracing back the Most Recent Common Ancestor</p></section>
<section class="slide level2">

<ul>
<li>In a diploid population with <span class="math inline">\(k\)</span> alleles at a neutral locus the average time (expressed in generations) back to the previous coalescent event when there were <span class="math inline">\(k-1\)</span> alleles is expressed with:</li>
</ul>
<p><span class="math display">\[
\label{coaleNe}
\tag{6.7}
T_k = \frac{4N_e}{k(k-1)}generations
\]</span></p>
<ul>
<li>The time for all <span class="math inline">\(k\)</span> alleles in the population to coalesce back to one allele is: <span class="math inline">\(4N_e[1-(\frac{1}{k})]\)</span>.</li>
<li>In a population of <span class="math inline">\(N_e = 50\)</span> with three alleles the expected time to its previous coalescence is:</li>
</ul>
<p><span class="math display">\[
T_3 = \frac{4*50}{3*2} = 33 \ generations
\]</span></p>
</section>
<section id="population-structure" class="slide level2">
<h2>Population structure</h2>
<ul>
<li>In most species individuals are scattered in local conglomerates that are called <em>demes</em> (<em>i.e.</em>, local populations or subpopulations)</li>
</ul>
<div class="fragment">
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_13_1.png"></p>
<figcaption>After <span class="citation" data-cites="Akakaya2007">Akçakaya, Mills, &amp; Doncaster (<a href="#/references" role="doc-biblioref" onclick="">2007</a>)</span></figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_13_2.jpg"></p>
<figcaption>After <span class="citation" data-cites="Frankham2010">Frankham et al. (<a href="#/references" role="doc-biblioref" onclick="">2010</a>)</span></figcaption>
</figure>
</div>
</div></div>
</div>
</section>
<section id="population-structure-1" class="slide level2">
<h2>Population structure</h2>
<ul>
<li><p>A metapopulation is defined as a set of discrete populations (demes) of the same species, in the same general geographical area, potentially able to exchange individuals through migration, dispersal, or human-mediated movement <span class="citation" data-cites="Akakaya2007 Hanski1997">(<a href="#/references" role="doc-biblioref" onclick="">Akçakaya et al., 2007</a>; <a href="#/references" role="doc-biblioref" onclick="">Hanski &amp; Simberloff, 1997</a>)</span></p></li>
<li><p>This subdivision (or structure) of populations can be natural and/or human driven</p></li>
<li><p>Regardless of the cause, the existence of population structure in many species forces us to think at the genetic variation within a species at two primary levels:</p>
<ol type="1">
<li>Genetic variation within local demes.</li>
<li>Genetic variation bewteen local demes.</li>
</ol></li>
<li><p>Not all species are scattered!</p></li>
</ul>
</section>
<section id="population-structure-2" class="slide level2">
<h2>Population structure</h2>

<img data-src="./class_13_3.png" class="r-stretch quarto-figure-center"><p class="caption">After <span class="citation" data-cites="Allendorf2022">Allendorf et al. (<a href="#/references" role="doc-biblioref" onclick="">2022</a>)</span>, p.&nbsp;184</p></section>
<section id="population-structure-3" class="slide level2">
<h2>Population structure</h2>

<img data-src="./class_13_4.png" class="r-stretch quarto-figure-center"><p class="caption">Boreal forest distibution in North America. Available <a href="https://canadiangeographic.ca/articles/new-framework-identifies-climate-change-refugia-in-boreal-forest/">here</a></p></section>
<section id="population-structure-4" class="slide level2">
<h2>Population structure</h2>
<ul>
<li>Even in the absence of barriers to gene flow, genetic differences can accrue so that individuals that are very distant from each other may present significant differences at the genetic level: isolation by distance</li>
</ul>

<img data-src="./class_13_5.png" class="r-stretch quarto-figure-center"><p class="caption">Variability in the correlation between species genetic diversity and geographic distance. ranges. Modified from <span class="citation" data-cites="Forbes1999">Forbes &amp; Hogg (<a href="#/references" role="doc-biblioref" onclick="">1999</a>)</span> with additional unpublished data from M.K. Schwartz <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</p></section>
<section id="population-structure-5" class="slide level2">
<h2>Population structure</h2>
<ul>
<li>Describing the patterns and amount of differentiation among demes is of the uttermost importance to develop effective conservation strategies <span class="citation" data-cites="Thatte2018">(see for example <a href="#/references" role="doc-biblioref" onclick="">Thatte, Joshi, Vaidyanathan, Landguth, &amp; Ramakrishnan, 2018</a>)</span>.</li>
</ul>

<img data-src="./class_13_6.png" class="r-stretch quarto-figure-center"><p class="caption">Study area in <span class="citation" data-cites="Thatte2018">Thatte et al. (<a href="#/references" role="doc-biblioref" onclick="">2018</a>)</span>. Genetically identified tigers as red dots</p></section>
<section id="population-structure-6" class="slide level2">
<h2>Population structure</h2>
<ul>
<li><span class="citation" data-cites="Frankham2011">Frankham et al. (<a href="#/references" role="doc-biblioref" onclick="">2011</a>)</span> describe the risks of outbreeding depression when translocating individuals from genetically distinct populations to supplement imperiled ones.</li>
</ul>
<div class="fragment">
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_13_7.png" style="width:85.0%"></p>
<figcaption>After <span class="citation" data-cites="Welch2017">Welch et al. (<a href="#/references" role="doc-biblioref" onclick="">2017</a>)</span>.</figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_13_8.png" style="width:83.0%"></p>
<figcaption>After <span class="citation" data-cites="Welch2017">Welch et al. (<a href="#/references" role="doc-biblioref" onclick="">2017</a>)</span>.</figcaption>
</figure>
</div>
</div></div>
</div>
</section>
<section id="population-structure-7" class="slide level2">
<h2>Population structure</h2>

<img data-src="./class_13_9.png" class="r-stretch quarto-figure-center"><p class="caption">After <span class="citation" data-cites="Colosimo2021b">Colosimo et al. (<a href="#/references" role="doc-biblioref" onclick="">2021</a>)</span></p></section>
<section id="population-structure-8" class="slide level2">
<h2>Population structure</h2>
<ul>
<li>Developing priorities for the conservation of a species often requires an understanding of adaptive genetic differentiation among populations <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</li>
<li>To understand how genetic variation is distributed at different loci within subdivided populations it is necessary to look at different interacting forces like gene flow, genetic drift and natural selection.</li>
<li><span class="math inline">\(F\)</span> statistics <span class="citation" data-cites="Wright1931 Wright1951 Malecot1948">(<a href="#/references" role="doc-biblioref" onclick="">Malécot, 1948</a>; <a href="#/references" role="doc-biblioref" onclick="">Wright, 1931</a>, <a href="#/references" role="doc-biblioref" onclick="">1951</a>)</span> are undoubtedly the most used indexes to describe genetic differentiation in and between demes.</li>
<li><span class="math inline">\(F\)</span> statistics are, in short, inbreeding coefficients measuring different types of inbreeding at different levels.</li>
</ul>
</section>
<section id="f-statistics" class="slide level2">
<h2><span class="math inline">\(F\)</span> statistics</h2>
<ul>
<li>We recognize three primary coefficients: <span class="math inline">\(F_{IS}\)</span>, <span class="math inline">\(F_{IT}\)</span>, and <span class="math inline">\(F_{ST}\)</span>.</li>
<li><span class="math inline">\(F_{IS}\)</span> measures Hardy–Weinberg departures within local demes.</li>
<li><span class="math inline">\(F_{ST}\)</span> measures divergence in allele frequencies among demes.</li>
<li><span class="math inline">\(F_{IT}\)</span> measures Hardy–Weinberg departures in the metapopulation due to <span class="math inline">\(F_{IS}\)</span> (nonrandom mating within local demes) and <span class="math inline">\(F_{ST}\)</span> (allele frequency divergence among demes).</li>
<li>The original <span class="math inline">\(F\)</span> statistics developed by wright focused on loci with only two alleles.</li>
<li><span class="citation" data-cites="Nei1977">Nei (<a href="#/references" role="doc-biblioref" onclick="">1977</a>)</span> extended these measures to multiple alleles and used a different notation to name what he called <em>analysis of gene diversity</em>: <span class="math inline">\(G_{IS}\)</span>, <span class="math inline">\(G_{IT}\)</span>, and <span class="math inline">\(G_{ST}\)</span>. Often the two groups of indexes are used interchangeably.</li>
</ul>
</section>
<section id="f_is" class="slide level2">
<h2><span class="math inline">\(F_{IS}\)</span></h2>
<p>This is a measure of departure from HW proportions within local demes.</p>
<p><span class="math display">\[
\label{fis}
\tag{13.1}
F_{IS} = 1-\frac{H_O}{H_S}
\]</span></p>
<p>where <span class="math inline">\(H_O\)</span> is the observed heterozygosity averaged over all demes while <span class="math inline">\(H_S\)</span> is the expected heterozygosity averaged over all demes. <span class="math inline">\(F_{IS}\)</span> values can range from <span class="math inline">\(-1 \leq 0 \leq 1\)</span>, with positive values indicating an excess of homozygotes and negative values indicating an excess of heterozygotes <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</p>
</section>
<section id="f_st" class="slide level2">
<h2><span class="math inline">\(F_{ST}\)</span></h2>
<p>Often called <em>fixation index</em>, it is a measure of genetic divergence among demes.</p>
<p><span class="math display">\[
\label{fst}
\tag{13.2}
F_{ST} = 1-\frac{H_S}{H_T}
\]</span></p>
<p>with <span class="math inline">\(H_T\)</span> as the expected HW heterozygosity if the metapopulation were panmictic, and <span class="math inline">\(H_S\)</span> is the HW proportion of heterozygotes in each separate deme and then averaged over all demes. This measure varies from <span class="math inline">\(0\)</span> (demes all have equal allele frequencies) to <span class="math inline">\(1\)</span> (demes are all fixed for different alleles).</p>
</section>
<section id="f_it" class="slide level2">
<h2><span class="math inline">\(F_{IT}\)</span></h2>
<p>Measures the departure from HW proportions due to departures from HW proportions within local demes and divergence among demes <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</p>
<p><span class="math display">\[
\label{fit}
\tag{13.3}
F_{IT} = 1-\frac{H_O}{H_T}
\]</span></p>
<p>where <span class="math inline">\(H_O\)</span> is the observed heterozygosity averaged over all demes while <span class="math inline">\(H_T\)</span> is the expected HW heterozygosity if the metapopulation were panmictic.</p>
<div class="fragment">
<ul>
<li>The fixation indexes described in the previous slides are related by the following expression:</li>
</ul>
<p><span class="math display">\[
\label{findexrelation}
\tag{13.4}
F_{IT}= F_{IS}+F_{ST}-(F_{IS}*F_{ST})
\]</span></p>
</div>
</section>
<section id="population-structure-and-habitat-fragmentation" class="slide level2">
<h2>Population structure and habitat fragmentation</h2>
<ul>
<li>Habitat fragmentation is a combination of two processes: reduction in total area, and the creation of separate islands.</li>
<li>Human induced fragmentation and habitat loss is recognized as the main cause of biodiversity loss <span class="citation" data-cites="deBaan2013">(<a href="#/references" role="doc-biblioref" onclick="">Baan, Alkemade, &amp; Koellner, 2013</a>)</span>.</li>
<li>Habitat fragmentation causes an overall reduction in population size.</li>
<li>Habitat fragmentation also causes an overall reduction in gene flow among the patches it creates.</li>
<li>Inhospitable, cleared habitat among fragments typically inhibits gene flow. The same goes for roads rivers, water bodies, mountains, or any other artificial or natural barrier <span class="citation" data-cites="Colosimo2014">Riley et al. (<a href="#/references" role="doc-biblioref" onclick="">2006</a>)</span>.</li>
<li>With restricted gene flow fragmentation is highly deleterious in the long term.</li>
<li>Single Large or Several Small <span class="citation" data-cites="Frankham2010">(<a href="#/references" role="doc-biblioref" onclick="">Frankham et al., 2010</a>)</span>.</li>
</ul>
</section>
<section id="population-structure-and-habitat-fragmentation-1" class="slide level2">
<h2>Population structure and habitat fragmentation</h2>

<img data-src="./class_13_10.png" class="r-stretch quarto-figure-center"><p class="caption">Fragmentation of Atlantic forest. After <span class="citation" data-cites="Luci2011">Nunes (<a href="#/references" role="doc-biblioref" onclick="">2011</a>)</span></p></section>
<section id="population-structure-and-habitat-fragmentation-2" class="slide level2">
<h2>Population structure and habitat fragmentation</h2>

<img data-src="./class_13_11.jpg" class="r-stretch quarto-figure-center"><p class="caption">Single large (SL) VS several small (SS). After <span class="citation" data-cites="Frankham2010">Frankham et al. (<a href="#/references" role="doc-biblioref" onclick="">2010</a>)</span></p></section>
<section id="population-structure-and-habitat-fragmentation-3" class="slide level2">
<h2>Population structure and habitat fragmentation</h2>
<p>Fragmentation happens in two steps:</p>
<ol type="1">
<li>Initial genetic subdivision of the population.</li>
<li>Cumulative diversification through genetic drift, selection, inbreeding etc. etc. etc.</li>
</ol>
<p>After an initial fragmentation, different fragments will have different initial allele frequencies. This diversification can be described by the binomial sampling variance (<span class="math inline">\(\sigma_p^2 = \frac{pq}{2N}\)</span>) and the smaller the fragment, the higher the variance.</p>
<div class="fragment">
<p>The degree of differentiation among fragments can be described by partitioning the overall inbreeding into components within and among populations (<span class="math inline">\(F\)</span> statistics).</p>
</div>
<div class="fragment">
<p>We will look at some examples on estimating <span class="math inline">\(F\)</span> statistics from genotype data.</p>
</div>
</section>
<section id="population-structure-and-habitat-fragmentation-4" class="slide level2">
<h2>Population structure and habitat fragmentation</h2>
<h3 id="gene-flow">Gene flow</h3>
<p>What is gene flow?</p>
<ul>
<li>Movement of alleles between population due to migration.</li>
</ul>
<p>The effects of fragmentation on gene flow depend on:</p>
<ul>
<li>Number of population fragments.</li>
<li>Distribution of population sizes in the fragments.</li>
<li>Distance between fragments.</li>
<li>Dispersal ability of the species.</li>
<li>Migration rates among fragments.</li>
<li>Environment of the matrix among fragments.</li>
<li>Times since fragmentation and extinction and recolonization rates across fragments.</li>
</ul>
</section>
<section id="the-black-footed-rock-wallaby" class="slide level2">
<h2>The black footed rock wallaby</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_13_12.jpg" class="quarto-figure quarto-figure-center" style="width:20.0%"></p>
</figure>
</div>
</div><div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_13_14.jpg" class="quarto-figure quarto-figure-center" style="width:50.0%"></p>
</figure>
</div>
</div></div>
<table class="caption-top">
<colgroup>
<col style="width: 24%">
<col style="width: 26%">
<col style="width: 24%">
<col style="width: 24%">
</colgroup>
<thead>
<tr class="header">
<th>Location</th>
<th>Prop. of polymorphic loci</th>
<th>Mean # of alleles</th>
<th>Average heterozygosity</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Barrow Island (1)</td>
<td>0.1</td>
<td>1.2</td>
<td>0.05</td>
</tr>
<tr class="even">
<td>Exmouth (2)</td>
<td>1.0</td>
<td>3.4</td>
<td>0.62</td>
</tr>
<tr class="odd">
<td>Wheatbelt (3)</td>
<td>1.0</td>
<td>4.4</td>
<td>0.56</td>
</tr>
</tbody>
</table>
<ul>
<li>Data from <span class="citation" data-cites="Eldridge1999">Eldridge et al. (<a href="#/references" role="doc-biblioref" onclick="">1999</a>)</span>, images after <span class="citation" data-cites="Frankham2010">Frankham et al. (<a href="#/references" role="doc-biblioref" onclick="">2010</a>)</span>.</li>
</ul>
</section>
<section id="example---1" class="slide level2">
<h2>Example - 1</h2>
<ul>
<li>Populations with same allele frequencies. One with random mating the other one with inbreeding.</li>
</ul>
<table class="caption-top">
<colgroup>
<col style="width: 18%">
<col style="width: 15%">
<col style="width: 15%">
<col style="width: 15%">
<col style="width: 20%">
<col style="width: 7%">
<col style="width: 7%">
</colgroup>
<thead>
<tr class="header">
<th>Population</th>
<th><span class="math inline">\(A_1A_1\)</span></th>
<th><span class="math inline">\(A_1A_2\)</span></th>
<th><span class="math inline">\(A_2A_2\)</span></th>
<th>Allele Freq</th>
<th>F</th>
<th>He</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1</td>
<td>0.25</td>
<td>0.5</td>
<td>0.25</td>
<td><span class="math inline">\(p=0.5\)</span></td>
<td>0</td>
<td>0.5</td>
</tr>
<tr class="even">
<td></td>
<td></td>
<td></td>
<td></td>
<td><span class="math inline">\(q=0.5\)</span></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2</td>
<td>0.4</td>
<td>0.2</td>
<td>0.4</td>
<td><span class="math inline">\(p=0.5\)</span></td>
<td>0.6</td>
<td>0.5</td>
</tr>
<tr class="even">
<td></td>
<td></td>
<td></td>
<td></td>
<td><span class="math inline">\(q=0.5\)</span></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>
<ul>
<li><span class="math inline">\(H_O\)</span> observed heterozygosity averaged over all demes.</li>
<li><span class="math inline">\(H_S\)</span> expected heterozygosity averaged over all demes.</li>
<li><span class="math inline">\(H_T\)</span> expected heterozygosity of the metapopulation.</li>
<li><span class="math inline">\(F_{IS} = 1-\frac{H_O}{H_S}\)</span>; <span class="math inline">\(F_{IT} = 1-\frac{H_O}{H_T}\)</span>; <span class="math inline">\(F_{ST} = 1-\frac{H_S}{H_T}\)</span>.</li>
</ul>
</section>
<section id="example---2" class="slide level2">
<h2>Example - 2</h2>
<ul>
<li>Random mating populations with different allele frequencies.</li>
</ul>
<table class="caption-top">
<colgroup>
<col style="width: 18%">
<col style="width: 15%">
<col style="width: 15%">
<col style="width: 15%">
<col style="width: 19%">
<col style="width: 7%">
<col style="width: 9%">
</colgroup>
<thead>
<tr class="header">
<th>Population</th>
<th><span class="math inline">\(A_1A_1\)</span></th>
<th><span class="math inline">\(A_1A_2\)</span></th>
<th><span class="math inline">\(A_2A_2\)</span></th>
<th>Allele Freq</th>
<th>F</th>
<th>He</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1</td>
<td>0.25</td>
<td>0.5</td>
<td>0.25</td>
<td><span class="math inline">\(p=0.5\)</span></td>
<td>0</td>
<td>0.5</td>
</tr>
<tr class="even">
<td></td>
<td></td>
<td></td>
<td></td>
<td><span class="math inline">\(q=0.5\)</span></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2</td>
<td>0.04</td>
<td>0.32</td>
<td>0.64</td>
<td><span class="math inline">\(p=0.2\)</span></td>
<td>0</td>
<td>0.32</td>
</tr>
<tr class="even">
<td></td>
<td></td>
<td></td>
<td></td>
<td><span class="math inline">\(q=0.8\)</span></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>
<ul>
<li><span class="math inline">\(H_O\)</span> observed heterozygosity averaged over all demes.</li>
<li><span class="math inline">\(H_S\)</span> expected heterozygosity averaged over all demes.</li>
<li><span class="math inline">\(H_T\)</span> expected heterozygosity of the metapopulation.</li>
<li><span class="math inline">\(F_{IS} = 1-\frac{H_O}{H_S}\)</span>; <span class="math inline">\(F_{IT} = 1-\frac{H_O}{H_T}\)</span>; <span class="math inline">\(F_{ST} = 1-\frac{H_S}{H_T}\)</span>.</li>
</ul>
</section>
<section id="example---3" class="slide level2">
<h2>Example - 3</h2>
<ul>
<li>Populations with different allele frequencies, one random mating and one with inbreeding.</li>
</ul>
<table class="caption-top">
<colgroup>
<col style="width: 18%">
<col style="width: 15%">
<col style="width: 15%">
<col style="width: 15%">
<col style="width: 19%">
<col style="width: 7%">
<col style="width: 9%">
</colgroup>
<thead>
<tr class="header">
<th>Population</th>
<th><span class="math inline">\(A_1A_1\)</span></th>
<th><span class="math inline">\(A_1A_2\)</span></th>
<th><span class="math inline">\(A_2A_2\)</span></th>
<th>Allele Freq</th>
<th>F</th>
<th>He</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1</td>
<td>0.25</td>
<td>0.5</td>
<td>0.25</td>
<td><span class="math inline">\(p=0.5\)</span></td>
<td>0</td>
<td>0.5</td>
</tr>
<tr class="even">
<td></td>
<td></td>
<td></td>
<td></td>
<td><span class="math inline">\(q=0.5\)</span></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2</td>
<td>0.14</td>
<td>0.13</td>
<td>0.74</td>
<td><span class="math inline">\(p=0.2\)</span></td>
<td>0.6</td>
<td>0.32</td>
</tr>
<tr class="even">
<td></td>
<td></td>
<td></td>
<td></td>
<td><span class="math inline">\(q=0.8\)</span></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>
<ul>
<li><span class="math inline">\(H_O\)</span> observed heterozygosity averaged over all demes.</li>
<li><span class="math inline">\(H_S\)</span> expected heterozygosity averaged over all demes.</li>
<li><span class="math inline">\(H_T\)</span> expected heterozygosity of the metapopulation.</li>
<li><span class="math inline">\(F_{IS} = 1-\frac{H_O}{H_S}\)</span>; <span class="math inline">\(F_{IT} = 1-\frac{H_O}{H_T}\)</span>; <span class="math inline">\(F_{ST} = 1-\frac{H_S}{H_T}\)</span>.</li>
</ul>
</section>
<section id="example---4" class="slide level2">
<h2>Example - 4</h2>

<img data-src="./class_13_15.jpg" class="r-stretch quarto-figure-center"><p class="caption"><span class="math inline">\(F_{ST}\)</span> increases over generations in fragmented populations at a rate inversely dependent on population size. Values that are <span class="math inline">\(\ge 0.15\)</span> indicate significant differentiation between fragments <span class="citation" data-cites="Frankham2010">(<a href="#/references" role="doc-biblioref" onclick="">Frankham et al., 2010</a>)</span>.</p></section>
<section id="migration-and-gene-flow" class="slide level2">
<h2>Migration and gene flow</h2>
<ul>
<li>Migration reduces the impact of fragmentation in a way that is dependent on the rate of gene flow.</li>
<li>When is gene flow sufficiently high to prevent the genetic impact of fragmentation?</li>
</ul>
<div class="fragment">
<div class="columns">
<div class="column" style="width:50%;">
<ul>
<li>A single migrant per generation is considered sufficient to prevent differentiation between demes.</li>
<li>The migration rate <span class="math inline">\(m\)</span> is the proportion of genes in a population derived from migrants per generation.</li>
<li><span class="math inline">\(Nm\)</span> is the number of migrants per generation <span class="citation" data-cites="Frankham2010">(<a href="#/references" role="doc-biblioref" onclick="">Frankham et al., 2010</a>)</span>.</li>
</ul>
</div><div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_13_16.jpg" class="quarto-figure quarto-figure-center" style="width:65.0%"></p>
</figure>
</div>
</div></div>
</div>
</section>
<section id="references" class="slide level2 smaller scrollable">
<h2>References</h2>

<div class="quarto-auto-generated-content">
<p><img src="../../figures/Tor-Vergata_logo_cmyk.png" class="slide-logo"></p>
<div class="footer footer-default">
<p><a href="https://giulianocolosimo.github.io/biologia_della_conservazione_24-25/">Biologia della conservazione 24/25 - Giuliano Colosimo, PhD</a></p>
</div>
</div>
<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" data-line-spacing="2" role="list">
<div id="ref-Akakaya2007" class="csl-entry" role="listitem">
Akçakaya, H. R., Mills, G., &amp; Doncaster, C. P. (2007). The role of metapopulations in conservation.
</div>
<div id="ref-Allendorf2022" class="csl-entry" role="listitem">
Allendorf, F. W., Funk, W. C., Aitken, S. N., Byrne, M., Luikart, G., &amp; Antunes, A. (2022). <em><span class="nocase">Conservation and the Genomics of Populations</span></em>. Oxford University Press. Retrieved from <a href="https://doi.org/10.1093/oso/9780198856566.001.0001">https://doi.org/10.1093/oso/9780198856566.001.0001</a>
</div>
<div id="ref-deBaan2013" class="csl-entry" role="listitem">
Baan, L. de, Alkemade, R., &amp; Koellner, T. (2013). Land use impacts on biodiversity in LCA: A global approach. <em>The International Journal of Life Cycle Assessment</em>, <em>18</em>(6), 1216–1230. Retrieved from <a href="https://link.springer.com/content/pdf/10.1007/s11367-012-0412-0.pdf">https://link.springer.com/content/pdf/10.1007/s11367-012-0412-0.pdf</a>
</div>
<div id="ref-Charlesworth2009a" class="csl-entry" role="listitem">
Charlesworth, B. (2009). <span class="nocase">Fundamental concepts in genetics: Effective population size and patterns of molecular evolution and variation</span>. <em>Nature Reviews Genetics</em>, <em>10</em>(3), 195–205. Retrieved from <a href="http://www.nature.com/doifinder/10.1038/nrg2526">http://www.nature.com/doifinder/10.1038/nrg2526</a>
</div>
<div id="ref-Colosimo2021b" class="csl-entry" role="listitem">
Colosimo, G., Jackson, A. C., Benton, A., Varela-Stokes, A., Iverson, J., Knapp, C. R., &amp; Welch, M. (2021). <span class="nocase">Correlated Population Genetic Structure in a Three-Tiered Host-Parasite System: The Potential for Coevolution and Adaptive Divergence</span>. <em>Journal of Heredity</em>, <em>112</em>(7), 590–601. Retrieved from <a href="https://doi.org/10.1093/jhered/esab058">https://doi.org/10.1093/jhered/esab058</a>
</div>
<div id="ref-Colosimo2014" class="csl-entry" role="listitem">
Colosimo, G., Knapp, C. R., Wallace, L. E., &amp; Welch, M. E. (2014). <span class="nocase">Inferred vs realized patterns of gene flow: an analysis of population structure in the Andros Island Rock Iguana</span>. (V. Ketmaier, Ed.)<em>PLoS ONE</em>, <em>9</em>(9), e106963. Retrieved from <a href="http://dx.plos.org/10.1371/journal.pone.0106963">http://dx.plos.org/10.1371/journal.pone.0106963</a>
</div>
<div id="ref-Donald2007" class="csl-entry" role="listitem">
Donald, P. F. (2007). Adult sex ratios in wild bird populations. <em>Ibis</em>, <em>149</em>(4), 671–692. Retrieved from <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1474-919X.2007.00724.x">https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1474-919X.2007.00724.x</a>
</div>
<div id="ref-Eldridge1999" class="csl-entry" role="listitem">
Eldridge, M. D. B., King, J. M., Loupis, A. K., Spencer, P. B. S., Taylor, A. C., Pope, L. C., &amp; Hall, G. P. (1999). Unprecedented low levels of genetic variation and inbreeding depression in an island population of the black-footed rock-wallaby. <em>Conservation Biology</em>, <em>13</em>(3), 531–541. Retrieved from <a href="https://conbio.onlinelibrary.wiley.com/doi/abs/10.1046/j.1523-1739.1999.98115.x">https://conbio.onlinelibrary.wiley.com/doi/abs/10.1046/j.1523-1739.1999.98115.x</a>
</div>
<div id="ref-Forbes1999" class="csl-entry" role="listitem">
Forbes, S. H., &amp; Hogg, J. T. (1999). Assessing population structure at high levels of differentiation: Microsatellite comparisons of bighorn sheep and large carnivores. <em>Animal Conservation</em>, <em>2</em>(3), 223–233. Retrieved from <a href="https://zslpublications.onlinelibrary.wiley.com/doi/abs/10.1111/j.1469-1795.1999.tb00068.x">https://zslpublications.onlinelibrary.wiley.com/doi/abs/10.1111/j.1469-1795.1999.tb00068.x</a>
</div>
<div id="ref-Frankham1996" class="csl-entry" role="listitem">
Frankham, R. (1996). <a href="https://doi.org/10.1046/j.1523-1739.1996.10061500.x"><span class="nocase">Relationship of genetic variation to population size in wildlife</span></a>. <em>Conserv Biol</em>, <em>10</em>(6), 1500–1508.
</div>
<div id="ref-Frankham2010" class="csl-entry" role="listitem">
Frankham, R., Ballou, J. D., &amp; Briscoe, D. A. (2010). <em>Introduction to conservation genetics</em>. Cambridge University Press. Retrieved from <a href="https://doi.org/10.1017%2Fcbo9780511809002">https://doi.org/10.1017%2Fcbo9780511809002</a>
</div>
<div id="ref-Frankham2011" class="csl-entry" role="listitem">
Frankham, R., Ballou, J. D., Eldridge, M. D. B., Lacy, R. C., Ralls, K., Dudash, M. R., &amp; Fenster, C. B. (2011). <a href="https://doi.org/10.1111/j.1523-1739.2011.01662.x"><span class="nocase">Predicting the probability of outbreeding depression</span></a>. <em>Conservation Biology</em>, <em>25</em>(3), 465–475.
</div>
<div id="ref-Hanski1997" class="csl-entry" role="listitem">
Hanski, I., &amp; Simberloff, D. (1997). The metapopulation approach, its history, conceptual domain, and application to conservation. In I. Hanski &amp; M. E. Gilpin (Eds.), <em>Metapopulation biology</em> (pp. 5–26). San Diego: Academic Press. Retrieved from <a href="https://www.sciencedirect.com/science/article/pii/B9780123234452500031">https://www.sciencedirect.com/science/article/pii/B9780123234452500031</a>
</div>
<div id="ref-Lamb2010" class="csl-entry" role="listitem">
Lamb, L. (2010). Living up to its name: How controversial hunting regulations restored big bulls to the elkhorn mountains. <em>Living up to its name: how controversial hunting regulations restored big bulls to the Elkhorn Mountains</em>, <em>Nov/Dec</em>, 14–19. Retrieved from <a href="https://fwp.mt.gov/binaries/content/assets/fwp/montana-outdoors/2010/elkhorns.pdf">https://fwp.mt.gov/binaries/content/assets/fwp/montana-outdoors/2010/elkhorns.pdf</a>
</div>
<div id="ref-Malecot1948" class="csl-entry" role="listitem">
Malécot, G. (1948). <em>Les math<span>é</span>matiques de l’h<span>é</span>r<span>é</span>dit<span>é</span></em>. Masson. Retrieved from <a href="https://books.google.fi/books?id=g7FjAQAACAAJ">https://books.google.fi/books?id=g7FjAQAACAAJ</a>
</div>
<div id="ref-Nei1977" class="csl-entry" role="listitem">
Nei, M. (1977). F-statistics and analysis of gene diversity in subdivided populations. <em>Annals of Human Genetics</em>, <em>41</em>(2), 225–233. Wiley. Retrieved from <a href="https://doi.org/10.1111%2Fj.1469-1809.1977.tb01918.x">https://doi.org/10.1111%2Fj.1469-1809.1977.tb01918.x</a>
</div>
<div id="ref-Luci2011" class="csl-entry" role="listitem">
Nunes, L. (2011). <a href="https://doi.org/10.22498/pages.19.2.60">Landslides in s<span>ã</span>o paulo, brazil: An integrated historical perspective</a>. <em>Pages News</em>, <em>19</em>, 60–63.
</div>
<div id="ref-Riley2006" class="csl-entry" role="listitem">
Riley, S. P. D., Pollinger, J. P., Sauvajot, R. M., York, E. C., BROMLEY, C., FULLER, T. K., &amp; WAYNE, R. K. (2006). A southern california freeway is a physical and social barrier to gene flow in carnivores. <em>Molecular Ecology</em>, <em>15</em>(7), 1733–1741. Wiley. Retrieved from <a href="https://doi.org/10.1111%2Fj.1365-294x.2006.02907.x">https://doi.org/10.1111%2Fj.1365-294x.2006.02907.x</a>
</div>
<div id="ref-Rosenberg2002" class="csl-entry" role="listitem">
Rosenberg, N. A., &amp; Nordborg, M. (2002). Genealogical trees, coalescent theory and the analysis of genetic polymorphisms. <em>Nature Reviews Genetics</em>, <em>3</em>(5), 380–390. Springer Science; Business Media <span>LLC</span>. Retrieved from <a href="https://doi.org/10.1038%2Fnrg795">https://doi.org/10.1038%2Fnrg795</a>
</div>
<div id="ref-Soule1973" class="csl-entry" role="listitem">
Soule, M. (1973). The epistasis cycle: A theory of marginal populations. <em>Annual Review of Ecology and Systematics</em>, <em>4</em>, 165–187. Annual Reviews. Retrieved April 5, 2023, from <a href="http://www.jstor.org/stable/2096809">http://www.jstor.org/stable/2096809</a>
</div>
<div id="ref-Thatte2018" class="csl-entry" role="listitem">
Thatte, P., Joshi, A., Vaidyanathan, S., Landguth, E., &amp; Ramakrishnan, U. (2018). Maintaining tiger connectivity and minimizing extinction into the next century: Insights from landscape genetics and spatially-explicit simulations. <em>Biological Conservation</em>, <em>218</em>, 181–191. Retrieved from <a href="https://www.sciencedirect.com/science/article/pii/S0006320717307346">https://www.sciencedirect.com/science/article/pii/S0006320717307346</a>
</div>
<div id="ref-Welch2017" class="csl-entry" role="listitem">
Welch, M. E., Colosimo, G., Pasachnik, S. A., Malone, C. L., Hilton, J., Long, J., Getz, A. H., et al. (2017). <span class="nocase">Molecular variation and population structure in critically endangered Turks and Caicos Rock Iguanas: identifying intraspecific conservation units and revising subspecific taxonomy</span>. <em>Conservation Genetics</em>, <em>18</em>(2), 479–493. Retrieved from <a href="http://link.springer.com/10.1007/s10592-016-0922-6">http://link.springer.com/10.1007/s10592-016-0922-6</a>
</div>
<div id="ref-Wright1931" class="csl-entry" role="listitem">
Wright, S. (1931). <a href="https://doi.org/10.1016/S0092-8240(05)80011-4"><span class="nocase">Evolution in Mendelian populations</span></a>. <em>Bulletin of Mathematical Biology</em>, <em>52</em>(1-2), 241–295.
</div>
<div id="ref-Wright1939" class="csl-entry" role="listitem">
Wright, S. (1939). <em>Statistical genetics in relation to evolution</em>. Actualit<span>é</span>s scientifiques et industrielles. Hermann. Retrieved from <a href="https://books.google.com.au/books?id=RQqMoAEACAAJ">https://books.google.com.au/books?id=RQqMoAEACAAJ</a>
</div>
<div id="ref-Wright1951" class="csl-entry" role="listitem">
Wright, S. (1951). <a href="https://doi.org/10.2307/2407273"><span class="nocase">The genetical structure of populations</span></a>. <em>Annals of eugenics</em>, <em>15</em>, 323–354.
</div>
</div>
</section>
    </div>
  </div>

  <script>window.backupDefine = window.define; window.define = undefined;</script>
  <script src="../../site_libs/revealjs/dist/reveal.js"></script>
  <!-- reveal.js plugins -->
  <script src="../../site_libs/revealjs/plugin/quarto-line-highlight/line-highlight.js"></script>
  <script src="../../site_libs/revealjs/plugin/pdf-export/pdfexport.js"></script>
  <script src="../../site_libs/revealjs/plugin/reveal-menu/menu.js"></script>
  <script src="../../site_libs/revealjs/plugin/reveal-menu/quarto-menu.js"></script>
  <script src="../../site_libs/revealjs/plugin/reveal-chalkboard/plugin.js"></script>
  <script src="../../site_libs/revealjs/plugin/multiplex/socket.io.js"></script>
  <script src="../../site_libs/revealjs/plugin/multiplex/multiplex.js"></script>
  <script src="../../site_libs/revealjs/plugin/quarto-support/support.js"></script>
  

  <script src="../../site_libs/revealjs/plugin/notes/notes.js"></script>
  <script src="../../site_libs/revealjs/plugin/search/search.js"></script>
  <script src="../../site_libs/revealjs/plugin/zoom/zoom.js"></script>
  <script src="../../site_libs/revealjs/plugin/math/math.js"></script>
  <script>window.define = window.backupDefine; window.backupDefine = undefined;</script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
'controlsAuto': true,
'previewLinksAuto': false,
'pdfSeparateFragments': false,
'autoAnimateEasing': "ease",
'autoAnimateDuration': 1,
'autoAnimateUnmatched': true,
'menu': {"side":"left","useTextContentForMissingTitles":true,"markers":false,"loadIcons":false,"custom":[{"title":"Tools","icon":"<i class=\"fas fa-gear\"></i>","content":"<ul class=\"slide-menu-items\">\n<li class=\"slide-tool-item active\" data-item=\"0\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.fullscreen(event)\"><kbd>f</kbd> Fullscreen</a></li>\n<li class=\"slide-tool-item\" data-item=\"1\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.speakerMode(event)\"><kbd>s</kbd> Speaker View</a></li>\n<li class=\"slide-tool-item\" data-item=\"2\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>o</kbd> Slide Overview</a></li>\n<li class=\"slide-tool-item\" data-item=\"3\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.togglePdfExport(event)\"><kbd>e</kbd> PDF Export Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"4\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleChalkboard(event)\"><kbd>b</kbd> Toggle Chalkboard</a></li>\n<li class=\"slide-tool-item\" data-item=\"5\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleNotesCanvas(event)\"><kbd>c</kbd> Toggle Notes Canvas</a></li>\n<li class=\"slide-tool-item\" data-item=\"6\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.downloadDrawings(event)\"><kbd>d</kbd> Download Drawings</a></li>\n<li class=\"slide-tool-item\" data-item=\"7\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.keyboardHelp(event)\"><kbd>?</kbd> Keyboard Help</a></li>\n</ul>"}],"openButton":true},
'chalkboard': {"buttons":true,"boardmarkerWidth":5},
'multiplex': {"secret":"17320572135651460717","id":"8399d6483dc4d337","url":"https://reveal-multiplex.glitch.me/"},
'smaller': false,
 
        // Display controls in the bottom right corner
        controls: false,

        // Help the user learn the controls by providing hints, for example by
        // bouncing the down arrow when they first encounter a vertical slide
        controlsTutorial: false,

        // Determines where controls appear, "edges" or "bottom-right"
        controlsLayout: 'edges',

        // Visibility rule for backwards navigation arrows; "faded", "hidden"
        // or "visible"
        controlsBackArrows: 'faded',

        // Display a presentation progress bar
        progress: true,

        // Display the page number of the current slide
        slideNumber: 'c/t',

        // 'all', 'print', or 'speaker'
        showSlideNumber: 'all',

        // Add the current slide number to the URL hash so that reloading the
        // page/copying the URL will return you to the same slide
        hash: true,

        // Start with 1 for the hash rather than 0
        hashOneBasedIndex: false,

        // Flags if we should monitor the hash and change slides accordingly
        respondToHashChanges: true,

        // Push each slide change to the browser history
        history: true,

        // Enable keyboard shortcuts for navigation
        keyboard: true,

        // Enable the slide overview mode
        overview: true,

        // Disables the default reveal.js slide layout (scaling and centering)
        // so that you can use custom CSS layout
        disableLayout: false,

        // Vertical centering of slides
        center: false,

        // Enables touch navigation on devices with touch input
        touch: true,

        // Loop the presentation
        loop: false,

        // Change the presentation direction to be RTL
        rtl: false,

        // see https://revealjs.com/vertical-slides/#navigation-mode
        navigationMode: 'linear',

        // Randomizes the order of slides each time the presentation loads
        shuffle: false,

        // Turns fragments on and off globally
        fragments: true,

        // Flags whether to include the current fragment in the URL,
        // so that reloading brings you to the same fragment position
        fragmentInURL: false,

        // Flags if the presentation is running in an embedded mode,
        // i.e. contained within a limited portion of the screen
        embedded: false,

        // Flags if we should show a help overlay when the questionmark
        // key is pressed
        help: true,

        // Flags if it should be possible to pause the presentation (blackout)
        pause: true,

        // Flags if speaker notes should be visible to all viewers
        showNotes: false,

        // Global override for autoplaying embedded media (null/true/false)
        autoPlayMedia: null,

        // Global override for preloading lazy-loaded iframes (null/true/false)
        preloadIframes: null,

        // Number of milliseconds between automatically proceeding to the
        // next slide, disabled when set to 0, this value can be overwritten
        // by using a data-autoslide attribute on your slides
        autoSlide: 0,

        // Stop auto-sliding after user input
        autoSlideStoppable: true,

        // Use this method for navigation when auto-sliding
        autoSlideMethod: null,

        // Specify the average time in seconds that you think you will spend
        // presenting each slide. This is used to show a pacing timer in the
        // speaker view
        defaultTiming: null,

        // Enable slide navigation via mouse wheel
        mouseWheel: false,

        // The display mode that will be used to show slides
        display: 'block',

        // Hide cursor if inactive
        hideInactiveCursor: true,

        // Time before the cursor is hidden (in ms)
        hideCursorTime: 5000,

        // Opens links in an iframe preview overlay
        previewLinks: true,

        // Transition style (none/fade/slide/convex/concave/zoom)
        transition: 'fade',

        // Transition speed (default/fast/slow)
        transitionSpeed: 'default',

        // Transition style for full page slide backgrounds
        // (none/fade/slide/convex/concave/zoom)
        backgroundTransition: 'none',

        // Number of slides away from the current that are visible
        viewDistance: 3,

        // Number of slides away from the current that are visible on mobile
        // devices. It is advisable to set this to a lower number than
        // viewDistance in order to save resources.
        mobileViewDistance: 2,

        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1600,

        height: 900,

        // Factor of the display size that should remain empty around the content
        margin: 0.1,

        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [QuartoLineHighlight, PdfExport, RevealMenu, RevealChalkboard, QuartoSupport,

          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    
    <script>
      // htmlwidgets need to know to resize themselves when slides are shown/hidden.
      // Fire the "slideenter" event (handled by htmlwidgets.js) when the current
      // slide changes (different for each slide format).
      (function () {
        // dispatch for htmlwidgets
        function fireSlideEnter() {
          const event = window.document.createEvent("Event");
          event.initEvent("slideenter", true, true);
          window.document.dispatchEvent(event);
        }

        function fireSlideChanged(previousSlide, currentSlide) {
          fireSlideEnter();

          // dispatch for shiny
          if (window.jQuery) {
            if (previousSlide) {
              window.jQuery(previousSlide).trigger("hidden");
            }
            if (currentSlide) {
              window.jQuery(currentSlide).trigger("shown");
            }
          }
        }

        // hookup for slidy
        if (window.w3c_slidy) {
          window.w3c_slidy.add_observer(function (slide_num) {
            // slide_num starts at position 1
            fireSlideChanged(null, w3c_slidy.slides[slide_num - 1]);
          });
        }

      })();
    </script>

    <script id="quarto-html-after-body" type="application/javascript">
    window.document.addEventListener("DOMContentLoaded", function (event) {
      const toggleBodyColorMode = (bsSheetEl) => {
        const mode = bsSheetEl.getAttribute("data-mode");
        const bodyEl = window.document.querySelector("body");
        if (mode === "dark") {
          bodyEl.classList.add("quarto-dark");
          bodyEl.classList.remove("quarto-light");
        } else {
          bodyEl.classList.add("quarto-light");
          bodyEl.classList.remove("quarto-dark");
        }
      }
      const toggleBodyColorPrimary = () => {
        const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
        if (bsSheetEl) {
          toggleBodyColorMode(bsSheetEl);
        }
      }
      toggleBodyColorPrimary();  
      const tabsets =  window.document.querySelectorAll(".panel-tabset-tabby")
      tabsets.forEach(function(tabset) {
        const tabby = new Tabby('#' + tabset.id);
      });
      const isCodeAnnotation = (el) => {
        for (const clz of el.classList) {
          if (clz.startsWith('code-annotation-')) {                     
            return true;
          }
        }
        return false;
      }
      const onCopySuccess = function(e) {
        // button target
        const button = e.trigger;
        // don't keep focus
        button.blur();
        // flash "checked"
        button.classList.add('code-copy-button-checked');
        var currentTitle = button.getAttribute("title");
        button.setAttribute("title", "Copied!");
        let tooltip;
        if (window.bootstrap) {
          button.setAttribute("data-bs-toggle", "tooltip");
          button.setAttribute("data-bs-placement", "left");
          button.setAttribute("data-bs-title", "Copied!");
          tooltip = new bootstrap.Tooltip(button, 
            { trigger: "manual", 
              customClass: "code-copy-button-tooltip",
              offset: [0, -8]});
          tooltip.show();    
        }
        setTimeout(function() {
          if (tooltip) {
            tooltip.hide();
            button.removeAttribute("data-bs-title");
            button.removeAttribute("data-bs-toggle");
            button.removeAttribute("data-bs-placement");
          }
          button.setAttribute("title", currentTitle);
          button.classList.remove('code-copy-button-checked');
        }, 1000);
        // clear code selection
        e.clearSelection();
      }
      const getTextToCopy = function(trigger) {
          const codeEl = trigger.previousElementSibling.cloneNode(true);
          for (const childEl of codeEl.children) {
            if (isCodeAnnotation(childEl)) {
              childEl.remove();
            }
          }
          return codeEl.innerText;
      }
      const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
        text: getTextToCopy
      });
      clipboard.on('success', onCopySuccess);
      if (window.document.getElementById('quarto-embedded-source-code-modal')) {
        // For code content inside modals, clipBoardJS needs to be initialized with a container option
        // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
        const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
          text: getTextToCopy,
          container: window.document.getElementById('quarto-embedded-source-code-modal')
        });
        clipboardModal.on('success', onCopySuccess);
      }
        var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
        var mailtoRegex = new RegExp(/^mailto:/);
          var filterRegex = new RegExp("https:\/\/giulianocolosimo\.github\.com");
        var isInternal = (href) => {
            return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
        }
        // Inspect non-navigation links and adorn them if external
     	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
        for (var i=0; i<links.length; i++) {
          const link = links[i];
          if (!isInternal(link.href)) {
            // undo the damage that might have been done by quarto-nav.js in the case of
            // links that we want to consider external
            if (link.dataset.originalHref !== undefined) {
              link.href = link.dataset.originalHref;
            }
          }
        }
      function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
        const config = {
          allowHTML: true,
          maxWidth: 500,
          delay: 100,
          arrow: false,
          appendTo: function(el) {
              return el.closest('section.slide') || el.parentElement;
          },
          interactive: true,
          interactiveBorder: 10,
          theme: 'light-border',
          placement: 'bottom-start',
        };
        if (contentFn) {
          config.content = contentFn;
        }
        if (onTriggerFn) {
          config.onTrigger = onTriggerFn;
        }
        if (onUntriggerFn) {
          config.onUntrigger = onUntriggerFn;
        }
          config['offset'] = [0,0];
          config['maxWidth'] = 700;
        window.tippy(el, config); 
      }
      const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
      for (var i=0; i<noterefs.length; i++) {
        const ref = noterefs[i];
        tippyHover(ref, function() {
          // use id or data attribute instead here
          let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
          try { href = new URL(href).hash; } catch {}
          const id = href.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note) {
            return note.innerHTML;
          } else {
            return "";
          }
        });
      }
      const findCites = (el) => {
        const parentEl = el.parentElement;
        if (parentEl) {
          const cites = parentEl.dataset.cites;
          if (cites) {
            return {
              el,
              cites: cites.split(' ')
            };
          } else {
            return findCites(el.parentElement)
          }
        } else {
          return undefined;
        }
      };
      var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
      for (var i=0; i<bibliorefs.length; i++) {
        const ref = bibliorefs[i];
        const citeInfo = findCites(ref);
        if (citeInfo) {
          tippyHover(citeInfo.el, function() {
            var popup = window.document.createElement('div');
            citeInfo.cites.forEach(function(cite) {
              var citeDiv = window.document.createElement('div');
              citeDiv.classList.add('hanging-indent');
              citeDiv.classList.add('csl-entry');
              var biblioDiv = window.document.getElementById('ref-' + cite);
              if (biblioDiv) {
                citeDiv.innerHTML = biblioDiv.innerHTML;
              }
              popup.appendChild(citeDiv);
            });
            return popup.innerHTML;
          });
        }
      }
    });
    </script>
    

</body></html>