<!DOCTYPE html>
<html lang="en"><head>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-html/tabby.min.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/light-border.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting-e26003cea8cd680ca0c55a263523d882.css" rel="stylesheet" id="quarto-text-highlighting-styles"><meta charset="utf-8">
  <meta name="generator" content="quarto-1.6.39">

  <meta name="dcterms.date" content="2024-11-28">
  <title>Biologia della Conservazione 24/25 – Biologia della Conservazione</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="../../site_libs/revealjs/dist/reset.css">
  <link rel="stylesheet" href="../../site_libs/revealjs/dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
      vertical-align: middle;
    }
    /* CSS for citations */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
      margin-bottom: 0em;
    }
    .hanging-indent div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }  </style>
  <link rel="stylesheet" href="../../site_libs/revealjs/dist/theme/quarto-fbff88913fe25870c86f77c0ed158baa.css">
  <link href="../../site_libs/revealjs/plugin/quarto-line-highlight/line-highlight.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-menu/menu.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-menu/quarto-menu.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-chalkboard/font-awesome/css/all.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-chalkboard/style.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">
  <style type="text/css">
    .reveal div.sourceCode {
      margin: 0;
      overflow: auto;
    }
    .reveal div.hanging-indent {
      margin-left: 1em;
      text-indent: -1em;
    }
    .reveal .slide:not(.center) {
      height: 100%;
    }
    .reveal .slide.scrollable {
      overflow-y: auto;
    }
    .reveal .footnotes {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide .absolute {
      position: absolute;
      display: block;
    }
    .reveal .footnotes ol {
      counter-reset: ol;
      list-style-type: none; 
      margin-left: 0;
    }
    .reveal .footnotes ol li:before {
      counter-increment: ol;
      content: counter(ol) ". "; 
    }
    .reveal .footnotes ol li > p:first-child {
      display: inline-block;
    }
    .reveal .slide ul,
    .reveal .slide ol {
      margin-bottom: 0.5em;
    }
    .reveal .slide ul li,
    .reveal .slide ol li {
      margin-top: 0.4em;
      margin-bottom: 0.2em;
    }
    .reveal .slide ul[role="tablist"] li {
      margin-bottom: 0;
    }
    .reveal .slide ul li > *:first-child,
    .reveal .slide ol li > *:first-child {
      margin-block-start: 0;
    }
    .reveal .slide ul li > *:last-child,
    .reveal .slide ol li > *:last-child {
      margin-block-end: 0;
    }
    .reveal .slide .columns:nth-child(3) {
      margin-block-start: 0.8em;
    }
    .reveal blockquote {
      box-shadow: none;
    }
    .reveal .tippy-content>* {
      margin-top: 0.2em;
      margin-bottom: 0.7em;
    }
    .reveal .tippy-content>*:last-child {
      margin-bottom: 0.2em;
    }
    .reveal .slide > img.stretch.quarto-figure-center,
    .reveal .slide > img.r-stretch.quarto-figure-center {
      display: block;
      margin-left: auto;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-left,
    .reveal .slide > img.r-stretch.quarto-figure-left  {
      display: block;
      margin-left: 0;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-right,
    .reveal .slide > img.r-stretch.quarto-figure-right  {
      display: block;
      margin-left: auto;
      margin-right: 0; 
    }
  </style>
<meta property="og:title" content="Biologia della Conservazione – Biologia della Conservazione 24/25">
<meta property="og:description" content="Class 7 - Inbreeding">
<meta property="og:site_name" content="Biologia della Conservazione 24/25">
<meta property="og:locale" content="en_US">
</head>
<body class="quarto-light">
  <div class="reveal">
    <div class="slides">

<section id="title-slide" class="quarto-title-block center">
  <h1 class="title">Biologia della Conservazione</h1>
  <p class="subtitle">Class 7 - Inbreeding</p>

<div class="quarto-title-authors">
</div>

  <p class="date">2024-11-28</p>
</section>
<section id="inbreeding" class="slide level2">
<h2>Inbreeding</h2>
<div class="columns">
<div class="column" style="width:50%;">
<p><img data-src="./class_10_1.jpg"></p>
</div><div class="column" style="width:50%;">
<p><img data-src="./class_10_2.jpg"></p>
<ul>
<li>There cannot be a class on inbreeding without these pictures. I am not sure who should receive credit, but if you type “inbreeding” in your browser and go to image results they immediately pop out.</li>
</ul>
</div></div>
</section>
<section id="what-is-inbreeding" class="slide level2">
<h2>What is inbreeding?</h2>
<ul>
<li>In its most simple definition inbreeding is the mating between related individuals. An individual is considered inbred if its parents share a common ancestor</li>
</ul>
<div class="fragment">
<ul>
<li>In large population inbreeding can happen because of the non-random assortment of gametes</li>
</ul>
</div>
<div class="fragment">
<ul>
<li>In many tree species spatially closer trees are more likely to be related than trees further apart</li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_10_3.png"></p>
<figcaption>Image after <span class="citation" data-cites="Allendorf2022">Allendorf et al. (<a href="#/references" role="doc-biblioref" onclick="">2022</a>)</span>, pg. 178</figcaption>
</figure>
</div>
</div>
</section>
<section class="slide level2">

<ul>
<li>Can inbreeding happen when there is random mating?</li>
</ul>
<div class="fragment">
<ul>
<li>Imagine the case of small populations. In the case of a population with 10 breeders, after a few generations the most distantly related individuals will be cousins, and even if the population mates at random, individuals will be highly inbred</li>
<li>This has been called <strong>inbreeding effect of small populations</strong> <span class="citation" data-cites="Crow1970">(<a href="#/references" role="doc-biblioref" onclick="">Crow &amp; Kimura, 1970</a>, pg. 101)</span></li>
</ul>
</div>
<div class="fragment">
<ul>
<li>So, we can consider an individual inbred if its parents share a common ancestor and its genes (one, two, or big chunks of the genome) are Identical by Descent (IBD)</li>
<li>What we know from studying evolution and from the examples on the colaescence theory is that eventually all parents of an individual share a common ancestor</li>
<li>We must, therefore, define inbreeding relative to some “base” population in which we assume all individuals are unrelated to one another <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>, pg. 371)</span></li>
</ul>
</div>
</section>
<section class="slide level2">

<ul>
<li>The concept of inbreeding is relatively easy to understand, but can also be confusing as the same term is associated with different phenomena (gentic drift, non random mating within populations, increase in genome wide homozygosity)</li>
<li>All these concepts are measured by the same quantity, inbreeding coefficients <span class="citation" data-cites="Templeton1994">(<a href="#/references" role="doc-biblioref" onclick="">Templeton &amp; Read, 1994</a>)</span>, and all these concepts can, to certain degree, be used interchangeably as they all refer to a situation in which relatives mate with each other <span class="citation" data-cites="Keller2002a">(<a href="#/references" role="doc-biblioref" onclick="">Keller &amp; Waller, 2002</a>)</span></li>
<li>As inbred individuals receive two copies of the same allele from its parents, they will present higher homozygosity and lower heterozygosity throughout the entire genome</li>
<li>We can, then, define the inbreeding coefficient (<span class="math inline">\(F\)</span>) as that proportion of the genome in an individual that is IBD (sometime an alternative word is used, <em>i.e.</em>, autozygous)</li>
<li>The alternative to IBD is IBS (Identical by State), which means that an individual possesses two alleles that are descendant from different ancestral alleles in the base population</li>
</ul>
</section>
<section class="slide level2">


<img data-src="./class_10_4.png" class="r-stretch quarto-figure-center"><p class="caption">Figure from <span class="citation" data-cites="Allendorf2022">Allendorf et al. (<a href="#/references" role="doc-biblioref" onclick="">2022</a>)</span>, pg. 374</p></section>
<section id="the-pedigree-inbreeding-coefficient" class="slide level2">
<h2>The pedigree inbreeding coefficient</h2>
<ul>
<li>The pedigree inbreeding coefficient (<span class="math inline">\(F_P\)</span>) refers to the expected increase in homozigosity of the inbred individual relative to the population founders or founder</li>
<li>The value ranges from <span class="math inline">\(0\)</span> (indicating no inbreeding) to <span class="math inline">\(1\)</span> (indicating completely inbred)</li>
</ul>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_10_5.png"></p>
<figcaption>Classical representation of a pedigree. Squares are males and rounds are females <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_10_6.png"></p>
<figcaption>Path diagram representing the pedigree. Approach used to calculate the inbreeding coefficient <span class="math inline">\(F_p\)</span> <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</figcaption>
</figure>
</div>
</div></div>
</section>
<section class="slide level2">


<img data-src="./class_10_11.png" class="r-stretch quarto-figure-center"><p class="caption">Did you know that Charles Darwin and his wife were cousins?</p></section>
<section id="how-to-estimate-the-pedigree" class="slide level2">
<h2>How to estimate the pedigree?</h2>
<ol type="1">
<li>Draw the pedigree as shown in previous figure where each individual appears only one</li>
<li>Identify the ancestors of the male and female that generated X. If the father and the mother share no ancestors <span class="math inline">\(F_p = 0\)</span></li>
<li>Trace all of the paths that lead from one of X’s parents, through the common ancestor, and then back again to the other parent of X (in our example there is only one path: from D to A and from A to E. The notation to indicate such path is D<u>A</u>E)</li>
<li>Determine <span class="math inline">\(N\)</span>, the number of individuals in the path, excluding our target individual X. In our case <span class="math inline">\(N = 3\)</span></li>
<li>With a single path we can use</li>
</ol>
<p><span class="math display">\[
\label{fp}
\tag{7.1}
F_P = (\frac{1}{2})^N(1+F_{P(CA)})
\]</span></p>
<p>where <span class="math inline">\(F_{P(CA)}\)</span> is the pedigree inbreeding coefficient of the common ancestor.</p>
</section>
<section class="slide level2">

<ol start="6" type="1">
<li>If there is more than just one path, <span class="math inline">\(F_P\)</span> becomes the sum of all <span class="math inline">\(F_P\)</span> values obtained from each single paths.</li>
</ol>
<p><span class="math display">\[
\label{fp2}
\tag{7.2}
F_P = \sum[(\frac{1}{2})^N(1+F_{P(CA)})]
\]</span></p>
<p>So, in our previous example, taken from <span class="citation" data-cites="Allendorf2022">Allendorf et al. (<a href="#/references" role="doc-biblioref" onclick="">2022</a>)</span>, we can use equation <span class="math inline">\(\ref{fp}\)</span> and the estimated inbreeding coefficient is</p>
<p><span class="math display">\[
F_{P(X)}= (\frac{1}{2})^3(1+0) =0.125
\]</span></p>
<p>We can read this result in two complementary ways:</p>
<ol type="1">
<li>Individual X is expected to have 12.5% of his genes IBD</li>
<li>X is expected to have its heterozygosity reduced by 12.5% compared to the ancestral population</li>
</ol>
</section>
<section class="slide level2">


<img data-src="./class_10_7.png" class="r-stretch quarto-figure-center"><p class="caption">What is <span class="math inline">\(F_{p(K)}\)</span> based on the above path diagram representation? After <span class="citation" data-cites="Allendorf2022">Allendorf et al. (<a href="#/references" role="doc-biblioref" onclick="">2022</a>)</span>.</p></section>
<section class="slide level2">

<ul>
<li>There is one loop that contains a common ancestor of both parents of K (I<u>G</u>J). Therefore, using equation <span class="math inline">\(\ref{fp}\)</span>, <span class="math inline">\(F_{P(K)} = (1/2)^3 (1 + F_{P(G)})\)</span>.</li>
<li>Individual G is also inbred; there is one loop with three individuals through a common ancestor for individual G (D<u>B</u>E). Therefore, <span class="math inline">\(F_{P(G)} = (1/2)^3 (1 + F_{P(B)})\)</span>. Individual B is not inbred (<span class="math inline">\(F_{P(B)} = 0\)</span>) since she is a founder in this pedigree.</li>
<li>Solving the equations: <span class="math inline">\(F_{P(G)} = 0.125\)</span>, and <span class="math inline">\(F_{P(K)} = (1/2)^3 (1.125) = 0.141\)</span>.</li>
</ul>
</section>
<section class="slide level2">


<img data-src="./class10_pedigreeBonus.png" class="r-stretch quarto-figure-center"><p class="caption">After <span class="citation" data-cites="Allendorf2022">Allendorf et al. (<a href="#/references" role="doc-biblioref" onclick="">2022</a>)</span></p></section>
<section class="slide level2">


<img data-src="./class10_pedigreeBonus2.png" class="r-stretch quarto-figure-center"><p class="caption">Calculation of the pedigree inbreeding coefficient for the previous slide. After <span class="citation" data-cites="Allendorf2022">Allendorf et al. (<a href="#/references" role="doc-biblioref" onclick="">2022</a>)</span></p></section>
<section class="slide level2">

<ul>
<li><p>Two major short comings while using <span class="math inline">\(F_P\)</span>:</p>
<ol type="1">
<li>It does not account for the inbreeding level of individuals that are not included in the reference pedigree</li>
<li>This metric is sub-optimal to obtain an accurate estimate of the proportion of the genome that is IBD, because there is a much variation in the proportion of genes that are IBD imputable to chance alone</li>
</ol></li>
<li><p>In general, it is often very complicated to obtain accurate pedigrees for natural populations, and while this approach is still massively used in all species for which we can maintain appropriate pedigree databases, coefficient of inbreeding in wild animals is often obtained through the analysis of molecular markers</p></li>
<li><p>Molecular marker-based measures of <span class="math inline">\(F\)</span> range from simple estimates of individual heterozygosity to more advanced methods that use mapped loci to estimate <span class="math inline">\(F\)</span> via identification of IBD chromosome segments as stretches of homozygous genotypes at mapped single nucleotide polymorphisms <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>, pg. 381)</span>.</p></li>
</ul>
</section>
<section id="inbreeding-coefficient-using-unmapped-loci" class="slide level2">
<h2>Inbreeding coefficient using unmapped loci</h2>
<ul>
<li>Estimates of <span class="math inline">\(F\)</span> based on individual heterozygosity rely on the concept that individuals whose parents are more closely related will have lower heterozygosity on average across the genome due to the presence of IBD chromosome segments</li>
</ul>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./ellegren1.png" style="width:50.0%"></p>
<figcaption>After <span class="citation" data-cites="Ellegren1999">Ellegren (<a href="#/references" role="doc-biblioref" onclick="">1999</a>)</span></figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./ellegren2.png" style="width:60.0%"></p>
<figcaption>After <span class="citation" data-cites="Ellegren1999">Ellegren (<a href="#/references" role="doc-biblioref" onclick="">1999</a>)</span></figcaption>
</figure>
</div>
</div></div>
</section>
<section id="heterozygosity-inbreeding" class="slide level2">
<h2>Heterozygosity inbreeding</h2>
<ul>
<li><p>The availability of thousands of genetic markers or more now makes it possible to use this approach in many taxa</p></li>
<li><p><span class="math inline">\(F_H\)</span> measures the excess in the observed number of homozygous genotypes within an individual relative to the mean number of homozygous genotypes expected under random mating</p></li>
</ul>
<p><span class="math display">\[
\label{fh}
\tag{7.3}
F_H = \frac{O(Hom)-E(Hom)}{L-E(Hom)}
\]</span></p>
<ul>
<li><p>Where <span class="math inline">\(O(Hom)\)</span> and <span class="math inline">\(E(Hom)\)</span> are the observed and expected Hardy–Weinberg (HW) numbers of homozygous loci in an individual and <span class="math inline">\(L\)</span> is the number of loci examined</p></li>
<li><p>The average <span class="math inline">\(O(Hom)\)</span> is expected to equal <span class="math inline">\(E(Hom)\)</span> in random mating populations. <span class="math inline">\(F_H\)</span> can thus be negative when the parents are less related than expected with random mating</p></li>
</ul>
</section>
<section id="inbreeding-coefficient-using-mapped-loci" class="slide level2">
<h2>Inbreeding coefficient using mapped loci</h2>

<img data-src="./F_ROH.png" class="r-stretch quarto-figure-center"><p class="caption">Heterozygosity across chromosomes 10-14 of scandinavian Wolves as analyzed by <span class="citation" data-cites="Kardos2017">Kardos et al. (<a href="#/references" role="doc-biblioref" onclick="">2017</a>)</span>.</p></section>
<section id="inbreeding-depression" class="slide level2">
<h2>Inbreeding depression</h2>
<ul>
<li><p>Inbred individuals generally have reduced fitness in comparison to noninbred individuals from the same population because of their increased homozygosity</p></li>
<li><p>Inbreeding depression is the reduction in fitness (or phenotype value) of progeny from matings between related individuals relative to the fitness of progeny between unrelated individuals</p></li>
<li><p>Inbreeding depression in natural populations contributes to the extinction of populations under some circumstances <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span></p></li>
<li><p>The importance of inbreeding depression has been debated since the time of Darwin. Even today, the importance of inbreeding depression is sometimes questioned by some conservationists and policy-makers</p></li>
</ul>
</section>
<section id="inbreeding-depression-a-common-garden-example" class="slide level2">
<h2>Inbreeding depression: a common garden example</h2>

<img data-src="./Willis_fig1_1993.png" class="r-stretch quarto-figure-center"><p class="caption">Common garden experiment of <em>Mimulus guttatus</em></p></section>
<section id="inbreeding-depression-an-example-from-a-zoo-population" class="slide level2">
<h2>Inbreeding depression: an example from a zoo population</h2>

<img data-src="./class10_callimico.jpg" class="r-stretch quarto-figure-center"><p class="caption">Callimico monkeys from the <a href="https://www.beardsleyzoo.org">Connecticut Zoo</a></p></section>
<section class="slide level2">


<img data-src="./class_10_12.png" class="r-stretch quarto-figure-center"><p class="caption">Relationship between inbreeding (<span class="math inline">\(F_P\)</span>) and infant survival in captive callimico monkeys <span class="citation" data-cites="Lacy1997">(<a href="#/references" role="doc-biblioref" onclick="">Lacy, 1997</a>)</span>.</p></section>
<section class="slide level2">

<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_10_9.jpg"></p>
<figcaption>The concomitant contribution of different factors in determining a vortex effect leading the populations to extinction. Image downloaded <a href="https://bio1152.nicerweb.com/Locked/media/ch56/vortex.html">here</a>.</figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_10_10.png"></p>
<figcaption>A more recent take on extinction vortex <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</figcaption>
</figure>
</div>
</div></div>
</section>
<section class="slide level2">

<ul>
<li>Inbreeding depression can result from either increased homozygosity or reduced heterozygosity</li>
<li>Aren’t increased homozygosity or reduced heterozygosity two sides of the same coin?</li>
</ul>
<div class="fragment">
<ul>
<li>Yes, but the mechanism related to a fitness reduction is distinct! Increased homozygosity leads to expression of a greater number of deleterious recessive alleles in inbred individuals, thereby lowering their fitness. Reduced heterozygosity reduces fitness of inbred individuals at loci where the heterozygotes have a selective advantage over homozygotes (heterozygous advantage or overdominance)</li>
<li>Most evidence show support for the primary problem of inbreeding depression is increased homozygosity <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span></li>
</ul>
</div>
</section>
<section id="different-mechanisms-of-fitness-reduction" class="slide level2">
<h2>Different mechanisms of fitness reduction</h2>

<img data-src="./class_15_2.jpg" class="r-stretch quarto-figure-center"><p class="caption">After <span class="citation" data-cites="Frankham2010">Frankham, Ballou, &amp; Briscoe (<a href="#/references" role="doc-biblioref" onclick="">2010</a>)</span></p></section>
<section class="slide level2">

<ul>
<li>It is crucial to know the mechanisms causing inbreeding depression because it affects the ability of a population to “adapt” to inbreeding.</li>
<li>A population could adapt to inbreeding if inbreeding depression is caused by deleterious recessive alleles that potentially could be removed (purged by selection).</li>
<li>However, inbreeding depression caused by heterozygous advantage cannot be purged because overdominant loci will always suffer reduced fitness as homozygosity increases due to increased inbreeding <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</li>
<li>Deleterious recessive alleles are present in the genome of all species because they are continually introduced by mutation, and natural selection is inefficient at removing them because most copies are “hidden” phenotypically in heterozygotes that do not have reduced fitness <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</li>
<li>We, therefore, expect all species to show some inbreeding depression due to the increase in homozygosity of recessive deleterious alleles.</li>
</ul>
</section>
<section class="slide level2">

<ul>
<li>The effects of inbreeding depression on survival are often measured by the mean number of “lethal equivalents” (LEs) per diploid genome.</li>
<li>A lethal equivalent is a set of deleterious alleles that would cause death if homozygous.</li>
<li>One lethal equivalent may be a single allele that is lethal when homozygous, two alleles each with a probability of 0.5 of causing death when homozygous, or 10 alleles each with a probability of 0.10 of causing death when homozygous.</li>
<li>The number of LEs present in a species or population is generally estimated by regressing survival on the inbreeding coefficient.</li>
<li>The effects of inbreeding on the probability of survival, S, can be expressed as a function of <span class="math inline">\(F_P\)</span> following <span class="citation" data-cites="Morton1956">Morton, Crow, &amp; Muller (<a href="#/references" role="doc-biblioref" onclick="">1956</a>)</span>.</li>
</ul>
<p><span class="math display">\[
\label{le}
\tag{7.4}
S = e^{-(A+BF_P)} \\ ln \ S= -A-BF_P
\]</span></p>
<p>where <span class="math inline">\(e^{-A}\)</span> is survival in an outbred population, and B is the inbreeding load.</p>
</section>
<section class="slide level2">


<img data-src="./class_10_13.png" class="r-stretch quarto-figure-center"><p class="caption">Relationship between inbreeding coefficient (F) and reduction in fitness. Inbreeding depression is the reduction in fitness of inbred individuals and measured by the number of LEs per gamete (B). The bracket shows the reduction in fitness (inbreeding depression) predicted for individuals with an F of 0.5. After <span class="citation" data-cites="Keller2002a">Keller &amp; Waller (<a href="#/references" role="doc-biblioref" onclick="">2002</a>)</span> and <span class="citation" data-cites="Allendorf2022">Allendorf et al. (<a href="#/references" role="doc-biblioref" onclick="">2022</a>)</span>.</p></section>
<section class="slide level2">


<img data-src="./LE_.png" class="r-stretch quarto-figure-center"><p class="caption">After <span class="citation" data-cites="Allendorf2022">Allendorf et al. (<a href="#/references" role="doc-biblioref" onclick="">2022</a>)</span></p></section>
<section id="the-concept-of-genetic-load" class="slide level2">
<h2>The concept of genetic load</h2>
<ul>
<li><p>Genetic load is the relative difference in fitness between the theoretically fittest genotype within a population and the average genotype in that population.</p></li>
<li><p>The term “load” was first used by <span class="citation" data-cites="Muller1950">Muller (<a href="#/references" role="doc-biblioref" onclick="">1950</a>)</span> in his consideration of the possible effects of increased mutation rates because of nuclear weapons.</p>
<ol type="1">
<li>Mutation load: The decrease in fitness caused by the accumulation of deleterious mutations.</li>
<li>Segregation load: The decrease in fitness caused by heterozygous advantage.</li>
<li>Drift load: The decrease in fitness caused by the increase in frequency of deleterious alleles resulting from genetic drift.</li>
<li>Migration load: The reduction in fitness caused by the migration into a population of individuals that are less adapted to the local environment than native individuals.</li>
</ol></li>
</ul>
</section>
<section id="references" class="slide level2 smaller scrollable">
<h2>References</h2>


<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" data-line-spacing="2" role="list">
<div id="ref-Allendorf2022" class="csl-entry" role="listitem">
Allendorf, F. W., Funk, W. C., Aitken, S. N., Byrne, M., Luikart, G., &amp; Antunes, A. (2022). <em><span class="nocase">Conservation and the Genomics of Populations</span></em>. Oxford University Press. Retrieved from <a href="https://doi.org/10.1093/oso/9780198856566.001.0001">https://doi.org/10.1093/oso/9780198856566.001.0001</a>
</div>
<div id="ref-Crow1970" class="csl-entry" role="listitem">
Crow, J. F., &amp; Kimura, M. (1970). <em><span class="nocase">An Introduction to Population Genetics Theory</span></em> (p. 591). New York. Retrieved from <a href="http://www.amazon.de/Introduction-Population-Genetics-Theory-James/dp/0060414383/ref=sr{\_}1{\_}3?ie=UTF8{\&amp;}qid=1321741251{\&amp;}sr=8-3">http://www.amazon.de/Introduction-Population-Genetics-Theory-James/dp/0060414383/ref=sr{\_}1{\_}3?ie=UTF8{\&amp;}qid=1321741251{\&amp;}sr=8-3</a>
</div>
<div id="ref-Ellegren1999" class="csl-entry" role="listitem">
Ellegren, H. (1999). Inbreeding and relatedness in scandinavian grey wolves canis lupus. <em>Hereditas</em>, <em>130</em>(3), 239–244. Retrieved from <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1601-5223.1999.00239.x">https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1601-5223.1999.00239.x</a>
</div>
<div id="ref-Frankham2010" class="csl-entry" role="listitem">
Frankham, R., Ballou, J. D., &amp; Briscoe, D. A. (2010). <em>Introduction to conservation genetics</em>. Cambridge University Press. Retrieved from <a href="https://doi.org/10.1017%2Fcbo9780511809002">https://doi.org/10.1017%2Fcbo9780511809002</a>
</div>
<div id="ref-Kardos2017" class="csl-entry" role="listitem">
Kardos, M., Åkesson, M., Fountain, T., Flagstad, Øystein, Liberg, O., Olason, P., Sand, H., et al. (2017). Genomic consequences of intensive inbreeding in an isolated wolf population. <em>Nature Ecology <span>&amp;</span> Evolution</em>, <em>2</em>(1), 124–131. Springer Science; Business Media <span>LLC</span>. Retrieved from <a href="https://doi.org/10.1038%2Fs41559-017-0375-4">https://doi.org/10.1038%2Fs41559-017-0375-4</a>
</div>
<div id="ref-Keller2002a" class="csl-entry" role="listitem">
Keller, L. F., &amp; Waller, D. M. (2002). <a href="https://doi.org/10.1016/S0169-5347(02)02489-8"><span class="nocase">Inbreeding effects in wild populations</span></a>. <em>Trends in Ecology and Evolution</em>, <em>17</em>(5), 230–241.
</div>
<div id="ref-Lacy1997" class="csl-entry" role="listitem">
Lacy, R. C. (1997). <span class="nocase">Importance of Genetic Variation to the Viability of Mammalian Populations</span>. <em>Journal of Mammalogy</em>, <em>78</em>(2), 320–335. Retrieved from <a href="http://www.jstor.org/stable/1382885?origin=crossref">http://www.jstor.org/stable/1382885?origin=crossref</a>
</div>
<div id="ref-Morton1956" class="csl-entry" role="listitem">
Morton, N. E., Crow, J. F., &amp; Muller, H. J. (1956). <a href="https://doi.org/10.1073/pnas.42.11.855"><span class="nocase">An Estimate of the Mutational Damage in Man From Data on Consanguineous Marriages.</span></a> <em>Proceedings of the National Academy of Sciences of the United States of America</em>, <em>42</em>(11), 855–863.
</div>
<div id="ref-Muller1950" class="csl-entry" role="listitem">
Muller, H. J. (1950). <a href="https://www.ncbi.nlm.nih.gov/pubmed/14771033">Our load of mutations.</a> <em>Am J Hum Genet</em>, <em>2</em>(2), 111–176.
</div>
<div id="ref-Templeton1994" class="csl-entry" role="listitem">
Templeton, A. R., &amp; Read, B. (1994). Inbreeding: One word, several meanings, much confusion. <em>Conservation genetics</em> (pp. 91–105). Birkh<span>ä</span>user Basel. Retrieved from <a href="https://doi.org/10.1007%2F978-3-0348-8510-2_9">https://doi.org/10.1007%2F978-3-0348-8510-2_9</a>
</div>
</div>
</section>
    </div>
  <div class="quarto-auto-generated-content" style="display: none;">
<p><img src="../../figures/Tor-Vergata_logo_cmyk.png" class="slide-logo"></p>
<div class="footer footer-default">
<p><a href="https://giulianocolosimo.github.io/biologia_della_conservazione_24-25/">Biologia della conservazione 24/25 - Giuliano Colosimo, PhD</a></p>
</div>
</div></div>

  <script>window.backupDefine = window.define; window.define = undefined;</script>
  <script src="../../site_libs/revealjs/dist/reveal.js"></script>
  <!-- reveal.js plugins -->
  <script src="../../site_libs/revealjs/plugin/quarto-line-highlight/line-highlight.js"></script>
  <script src="../../site_libs/revealjs/plugin/pdf-export/pdfexport.js"></script>
  <script src="../../site_libs/revealjs/plugin/reveal-menu/menu.js"></script>
  <script src="../../site_libs/revealjs/plugin/reveal-menu/quarto-menu.js"></script>
  <script src="../../site_libs/revealjs/plugin/reveal-chalkboard/plugin.js"></script>
  <script src="../../site_libs/revealjs/plugin/multiplex/socket.io.js"></script>
  <script src="../../site_libs/revealjs/plugin/multiplex/multiplex.js"></script>
  <script src="../../site_libs/revealjs/plugin/quarto-support/support.js"></script>
  

  <script src="../../site_libs/revealjs/plugin/notes/notes.js"></script>
  <script src="../../site_libs/revealjs/plugin/search/search.js"></script>
  <script src="../../site_libs/revealjs/plugin/zoom/zoom.js"></script>
  <script src="../../site_libs/revealjs/plugin/math/math.js"></script>
  <script>window.define = window.backupDefine; window.backupDefine = undefined;</script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
'controlsAuto': true,
'previewLinksAuto': false,
'pdfSeparateFragments': false,
'autoAnimateEasing': "ease",
'autoAnimateDuration': 1,
'autoAnimateUnmatched': true,
'jumpToSlide': true,
'menu': {"side":"left","useTextContentForMissingTitles":true,"markers":false,"loadIcons":false,"custom":[{"title":"Tools","icon":"<i class=\"fas fa-gear\"></i>","content":"<ul class=\"slide-menu-items\">\n<li class=\"slide-tool-item active\" data-item=\"0\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.fullscreen(event)\"><kbd>f</kbd> Fullscreen</a></li>\n<li class=\"slide-tool-item\" data-item=\"1\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.speakerMode(event)\"><kbd>s</kbd> Speaker View</a></li>\n<li class=\"slide-tool-item\" data-item=\"2\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>o</kbd> Slide Overview</a></li>\n<li class=\"slide-tool-item\" data-item=\"3\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.togglePdfExport(event)\"><kbd>e</kbd> PDF Export Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"4\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleScrollView(event)\"><kbd>r</kbd> Scroll View Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"5\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleChalkboard(event)\"><kbd>b</kbd> Toggle Chalkboard</a></li>\n<li class=\"slide-tool-item\" data-item=\"6\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleNotesCanvas(event)\"><kbd>c</kbd> Toggle Notes Canvas</a></li>\n<li class=\"slide-tool-item\" data-item=\"7\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.downloadDrawings(event)\"><kbd>d</kbd> Download Drawings</a></li>\n<li class=\"slide-tool-item\" data-item=\"8\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.keyboardHelp(event)\"><kbd>?</kbd> Keyboard Help</a></li>\n</ul>"}],"openButton":true},
'chalkboard': {"buttons":true,"boardmarkerWidth":5},
'multiplex': {"secret":"17325250810937488326","id":"3837b2721fabdaa5","url":"https://reveal-multiplex.glitch.me/"},
'smaller': false,
 
        // Display controls in the bottom right corner
        controls: false,

        // Help the user learn the controls by providing hints, for example by
        // bouncing the down arrow when they first encounter a vertical slide
        controlsTutorial: false,

        // Determines where controls appear, "edges" or "bottom-right"
        controlsLayout: 'edges',

        // Visibility rule for backwards navigation arrows; "faded", "hidden"
        // or "visible"
        controlsBackArrows: 'faded',

        // Display a presentation progress bar
        progress: true,

        // Display the page number of the current slide
        slideNumber: 'c/t',

        // 'all', 'print', or 'speaker'
        showSlideNumber: 'all',

        // Add the current slide number to the URL hash so that reloading the
        // page/copying the URL will return you to the same slide
        hash: true,

        // Start with 1 for the hash rather than 0
        hashOneBasedIndex: false,

        // Flags if we should monitor the hash and change slides accordingly
        respondToHashChanges: true,

        // Push each slide change to the browser history
        history: true,

        // Enable keyboard shortcuts for navigation
        keyboard: true,

        // Enable the slide overview mode
        overview: true,

        // Disables the default reveal.js slide layout (scaling and centering)
        // so that you can use custom CSS layout
        disableLayout: false,

        // Vertical centering of slides
        center: false,

        // Enables touch navigation on devices with touch input
        touch: true,

        // Loop the presentation
        loop: false,

        // Change the presentation direction to be RTL
        rtl: false,

        // see https://revealjs.com/vertical-slides/#navigation-mode
        navigationMode: 'linear',

        // Randomizes the order of slides each time the presentation loads
        shuffle: false,

        // Turns fragments on and off globally
        fragments: true,

        // Flags whether to include the current fragment in the URL,
        // so that reloading brings you to the same fragment position
        fragmentInURL: false,

        // Flags if the presentation is running in an embedded mode,
        // i.e. contained within a limited portion of the screen
        embedded: false,

        // Flags if we should show a help overlay when the questionmark
        // key is pressed
        help: true,

        // Flags if it should be possible to pause the presentation (blackout)
        pause: true,

        // Flags if speaker notes should be visible to all viewers
        showNotes: false,

        // Global override for autoplaying embedded media (null/true/false)
        autoPlayMedia: null,

        // Global override for preloading lazy-loaded iframes (null/true/false)
        preloadIframes: null,

        // Number of milliseconds between automatically proceeding to the
        // next slide, disabled when set to 0, this value can be overwritten
        // by using a data-autoslide attribute on your slides
        autoSlide: 0,

        // Stop auto-sliding after user input
        autoSlideStoppable: true,

        // Use this method for navigation when auto-sliding
        autoSlideMethod: null,

        // Specify the average time in seconds that you think you will spend
        // presenting each slide. This is used to show a pacing timer in the
        // speaker view
        defaultTiming: null,

        // Enable slide navigation via mouse wheel
        mouseWheel: false,

        // The display mode that will be used to show slides
        display: 'block',

        // Hide cursor if inactive
        hideInactiveCursor: true,

        // Time before the cursor is hidden (in ms)
        hideCursorTime: 5000,

        // Opens links in an iframe preview overlay
        previewLinks: true,

        // Transition style (none/fade/slide/convex/concave/zoom)
        transition: 'fade',

        // Transition speed (default/fast/slow)
        transitionSpeed: 'default',

        // Transition style for full page slide backgrounds
        // (none/fade/slide/convex/concave/zoom)
        backgroundTransition: 'none',

        // Number of slides away from the current that are visible
        viewDistance: 3,

        // Number of slides away from the current that are visible on mobile
        // devices. It is advisable to set this to a lower number than
        // viewDistance in order to save resources.
        mobileViewDistance: 2,

        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1600,

        height: 900,

        // Factor of the display size that should remain empty around the content
        margin: 0.1,

        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [QuartoLineHighlight, PdfExport, RevealMenu, RevealChalkboard, QuartoSupport,

          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    
    <script>
      // htmlwidgets need to know to resize themselves when slides are shown/hidden.
      // Fire the "slideenter" event (handled by htmlwidgets.js) when the current
      // slide changes (different for each slide format).
      (function () {
        // dispatch for htmlwidgets
        function fireSlideEnter() {
          const event = window.document.createEvent("Event");
          event.initEvent("slideenter", true, true);
          window.document.dispatchEvent(event);
        }

        function fireSlideChanged(previousSlide, currentSlide) {
          fireSlideEnter();

          // dispatch for shiny
          if (window.jQuery) {
            if (previousSlide) {
              window.jQuery(previousSlide).trigger("hidden");
            }
            if (currentSlide) {
              window.jQuery(currentSlide).trigger("shown");
            }
          }
        }

        // hookup for slidy
        if (window.w3c_slidy) {
          window.w3c_slidy.add_observer(function (slide_num) {
            // slide_num starts at position 1
            fireSlideChanged(null, w3c_slidy.slides[slide_num - 1]);
          });
        }

      })();
    </script>

    <script id="quarto-html-after-body" type="application/javascript">
    window.document.addEventListener("DOMContentLoaded", function (event) {
      const toggleBodyColorMode = (bsSheetEl) => {
        const mode = bsSheetEl.getAttribute("data-mode");
        const bodyEl = window.document.querySelector("body");
        if (mode === "dark") {
          bodyEl.classList.add("quarto-dark");
          bodyEl.classList.remove("quarto-light");
        } else {
          bodyEl.classList.add("quarto-light");
          bodyEl.classList.remove("quarto-dark");
        }
      }
      const toggleBodyColorPrimary = () => {
        const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
        if (bsSheetEl) {
          toggleBodyColorMode(bsSheetEl);
        }
      }
      toggleBodyColorPrimary();  
      const tabsets =  window.document.querySelectorAll(".panel-tabset-tabby")
      tabsets.forEach(function(tabset) {
        const tabby = new Tabby('#' + tabset.id);
      });
      const isCodeAnnotation = (el) => {
        for (const clz of el.classList) {
          if (clz.startsWith('code-annotation-')) {                     
            return true;
          }
        }
        return false;
      }
      const onCopySuccess = function(e) {
        // button target
        const button = e.trigger;
        // don't keep focus
        button.blur();
        // flash "checked"
        button.classList.add('code-copy-button-checked');
        var currentTitle = button.getAttribute("title");
        button.setAttribute("title", "Copied!");
        let tooltip;
        if (window.bootstrap) {
          button.setAttribute("data-bs-toggle", "tooltip");
          button.setAttribute("data-bs-placement", "left");
          button.setAttribute("data-bs-title", "Copied!");
          tooltip = new bootstrap.Tooltip(button, 
            { trigger: "manual", 
              customClass: "code-copy-button-tooltip",
              offset: [0, -8]});
          tooltip.show();    
        }
        setTimeout(function() {
          if (tooltip) {
            tooltip.hide();
            button.removeAttribute("data-bs-title");
            button.removeAttribute("data-bs-toggle");
            button.removeAttribute("data-bs-placement");
          }
          button.setAttribute("title", currentTitle);
          button.classList.remove('code-copy-button-checked');
        }, 1000);
        // clear code selection
        e.clearSelection();
      }
      const getTextToCopy = function(trigger) {
          const codeEl = trigger.previousElementSibling.cloneNode(true);
          for (const childEl of codeEl.children) {
            if (isCodeAnnotation(childEl)) {
              childEl.remove();
            }
          }
          return codeEl.innerText;
      }
      const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
        text: getTextToCopy
      });
      clipboard.on('success', onCopySuccess);
      if (window.document.getElementById('quarto-embedded-source-code-modal')) {
        const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
          text: getTextToCopy,
          container: window.document.getElementById('quarto-embedded-source-code-modal')
        });
        clipboardModal.on('success', onCopySuccess);
      }
        var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
        var mailtoRegex = new RegExp(/^mailto:/);
          var filterRegex = new RegExp("https:\/\/giulianocolosimo\.github\.com");
        var isInternal = (href) => {
            return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
        }
        // Inspect non-navigation links and adorn them if external
     	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
        for (var i=0; i<links.length; i++) {
          const link = links[i];
          if (!isInternal(link.href)) {
            // undo the damage that might have been done by quarto-nav.js in the case of
            // links that we want to consider external
            if (link.dataset.originalHref !== undefined) {
              link.href = link.dataset.originalHref;
            }
          }
        }
      function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
        const config = {
          allowHTML: true,
          maxWidth: 500,
          delay: 100,
          arrow: false,
          appendTo: function(el) {
              return el.closest('section.slide') || el.parentElement;
          },
          interactive: true,
          interactiveBorder: 10,
          theme: 'light-border',
          placement: 'bottom-start',
        };
        if (contentFn) {
          config.content = contentFn;
        }
        if (onTriggerFn) {
          config.onTrigger = onTriggerFn;
        }
        if (onUntriggerFn) {
          config.onUntrigger = onUntriggerFn;
        }
          config['offset'] = [0,0];
          config['maxWidth'] = 700;
        window.tippy(el, config); 
      }
      const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
      for (var i=0; i<noterefs.length; i++) {
        const ref = noterefs[i];
        tippyHover(ref, function() {
          // use id or data attribute instead here
          let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
          try { href = new URL(href).hash; } catch {}
          const id = href.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note) {
            return note.innerHTML;
          } else {
            return "";
          }
        });
      }
      const findCites = (el) => {
        const parentEl = el.parentElement;
        if (parentEl) {
          const cites = parentEl.dataset.cites;
          if (cites) {
            return {
              el,
              cites: cites.split(' ')
            };
          } else {
            return findCites(el.parentElement)
          }
        } else {
          return undefined;
        }
      };
      var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
      for (var i=0; i<bibliorefs.length; i++) {
        const ref = bibliorefs[i];
        const citeInfo = findCites(ref);
        if (citeInfo) {
          tippyHover(citeInfo.el, function() {
            var popup = window.document.createElement('div');
            citeInfo.cites.forEach(function(cite) {
              var citeDiv = window.document.createElement('div');
              citeDiv.classList.add('hanging-indent');
              citeDiv.classList.add('csl-entry');
              var biblioDiv = window.document.getElementById('ref-' + cite);
              if (biblioDiv) {
                citeDiv.innerHTML = biblioDiv.innerHTML;
              }
              popup.appendChild(citeDiv);
            });
            return popup.innerHTML;
          });
        }
      }
    });
    </script>
    

</body></html>