<!DOCTYPE html>
<html lang="en"><head>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-html/tabby.min.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/light-border.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting-e26003cea8cd680ca0c55a263523d882.css" rel="stylesheet" id="quarto-text-highlighting-styles"><meta charset="utf-8">
  <meta name="generator" content="quarto-1.6.39">

  <meta name="dcterms.date" content="2024-12-02">
  <title>Biologia della Conservazione 24/25 – Biologia della Conservazione</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="../../site_libs/revealjs/dist/reset.css">
  <link rel="stylesheet" href="../../site_libs/revealjs/dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
      vertical-align: middle;
    }
    /* CSS for syntax highlighting */
    pre > code.sourceCode { white-space: pre; position: relative; }
    pre > code.sourceCode > span { line-height: 1.25; }
    pre > code.sourceCode > span:empty { height: 1.2em; }
    .sourceCode { overflow: visible; }
    code.sourceCode > span { color: inherit; text-decoration: inherit; }
    div.sourceCode { margin: 1em 0; }
    pre.sourceCode { margin: 0; }
    @media screen {
    div.sourceCode { overflow: auto; }
    }
    @media print {
    pre > code.sourceCode { white-space: pre-wrap; }
    pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
    }
    pre.numberSource code
      { counter-reset: source-line 0; }
    pre.numberSource code > span
      { position: relative; left: -4em; counter-increment: source-line; }
    pre.numberSource code > span > a:first-child::before
      { content: counter(source-line);
        position: relative; left: -1em; text-align: right; vertical-align: baseline;
        border: none; display: inline-block;
        -webkit-touch-callout: none; -webkit-user-select: none;
        -khtml-user-select: none; -moz-user-select: none;
        -ms-user-select: none; user-select: none;
        padding: 0 4px; width: 4em;
        color: #aaaaaa;
      }
    pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
    div.sourceCode
      { color: #003b4f; background-color: #f1f3f5; }
    @media screen {
    pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
    }
    code span { color: #003b4f; } /* Normal */
    code span.al { color: #ad0000; } /* Alert */
    code span.an { color: #5e5e5e; } /* Annotation */
    code span.at { color: #657422; } /* Attribute */
    code span.bn { color: #ad0000; } /* BaseN */
    code span.bu { } /* BuiltIn */
    code span.cf { color: #003b4f; font-weight: bold; } /* ControlFlow */
    code span.ch { color: #20794d; } /* Char */
    code span.cn { color: #8f5902; } /* Constant */
    code span.co { color: #5e5e5e; } /* Comment */
    code span.cv { color: #5e5e5e; font-style: italic; } /* CommentVar */
    code span.do { color: #5e5e5e; font-style: italic; } /* Documentation */
    code span.dt { color: #ad0000; } /* DataType */
    code span.dv { color: #ad0000; } /* DecVal */
    code span.er { color: #ad0000; } /* Error */
    code span.ex { } /* Extension */
    code span.fl { color: #ad0000; } /* Float */
    code span.fu { color: #4758ab; } /* Function */
    code span.im { color: #00769e; } /* Import */
    code span.in { color: #5e5e5e; } /* Information */
    code span.kw { color: #003b4f; font-weight: bold; } /* Keyword */
    code span.op { color: #5e5e5e; } /* Operator */
    code span.ot { color: #003b4f; } /* Other */
    code span.pp { color: #ad0000; } /* Preprocessor */
    code span.sc { color: #5e5e5e; } /* SpecialChar */
    code span.ss { color: #20794d; } /* SpecialString */
    code span.st { color: #20794d; } /* String */
    code span.va { color: #111111; } /* Variable */
    code span.vs { color: #20794d; } /* VerbatimString */
    code span.wa { color: #5e5e5e; font-style: italic; } /* Warning */
    /* CSS for citations */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
      margin-bottom: 0em;
    }
    .hanging-indent div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }  </style>
  <link rel="stylesheet" href="../../site_libs/revealjs/dist/theme/quarto-fbff88913fe25870c86f77c0ed158baa.css">
  <link href="../../site_libs/revealjs/plugin/quarto-line-highlight/line-highlight.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-menu/menu.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-menu/quarto-menu.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-chalkboard/font-awesome/css/all.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-chalkboard/style.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">
  <style type="text/css">
    .reveal div.sourceCode {
      margin: 0;
      overflow: auto;
    }
    .reveal div.hanging-indent {
      margin-left: 1em;
      text-indent: -1em;
    }
    .reveal .slide:not(.center) {
      height: 100%;
    }
    .reveal .slide.scrollable {
      overflow-y: auto;
    }
    .reveal .footnotes {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide .absolute {
      position: absolute;
      display: block;
    }
    .reveal .footnotes ol {
      counter-reset: ol;
      list-style-type: none; 
      margin-left: 0;
    }
    .reveal .footnotes ol li:before {
      counter-increment: ol;
      content: counter(ol) ". "; 
    }
    .reveal .footnotes ol li > p:first-child {
      display: inline-block;
    }
    .reveal .slide ul,
    .reveal .slide ol {
      margin-bottom: 0.5em;
    }
    .reveal .slide ul li,
    .reveal .slide ol li {
      margin-top: 0.4em;
      margin-bottom: 0.2em;
    }
    .reveal .slide ul[role="tablist"] li {
      margin-bottom: 0;
    }
    .reveal .slide ul li > *:first-child,
    .reveal .slide ol li > *:first-child {
      margin-block-start: 0;
    }
    .reveal .slide ul li > *:last-child,
    .reveal .slide ol li > *:last-child {
      margin-block-end: 0;
    }
    .reveal .slide .columns:nth-child(3) {
      margin-block-start: 0.8em;
    }
    .reveal blockquote {
      box-shadow: none;
    }
    .reveal .tippy-content>* {
      margin-top: 0.2em;
      margin-bottom: 0.7em;
    }
    .reveal .tippy-content>*:last-child {
      margin-bottom: 0.2em;
    }
    .reveal .slide > img.stretch.quarto-figure-center,
    .reveal .slide > img.r-stretch.quarto-figure-center {
      display: block;
      margin-left: auto;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-left,
    .reveal .slide > img.r-stretch.quarto-figure-left  {
      display: block;
      margin-left: 0;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-right,
    .reveal .slide > img.r-stretch.quarto-figure-right  {
      display: block;
      margin-left: auto;
      margin-right: 0; 
    }
  </style>
<meta property="og:title" content="Biologia della Conservazione – Biologia della Conservazione 24/25">
<meta property="og:description" content="Class 8 - Hybridization and heritability">
<meta property="og:site_name" content="Biologia della Conservazione 24/25">
<meta property="og:locale" content="en_US">
</head>
<body class="quarto-light">
  <div class="reveal">
    <div class="slides">

<section id="title-slide" class="quarto-title-block center">
  <h1 class="title">Biologia della Conservazione</h1>
  <p class="subtitle">Class 8 - Hybridization and heritability</p>

<div class="quarto-title-authors">
</div>

  <p class="date">2024-12-02</p>
</section>
<section id="assignment-tests" class="slide level2">
<h2>Assignment tests</h2>
<div class="columns">
<div class="column" style="width:50%;">
<ul>
<li><p>Assignment tests offer an attractive alternative to the traditional capture–mark–recapture approach to estimating <span class="math inline">\(Nm\)</span> directly.</p></li>
<li><p>For example, we can genotype many individuals in a single population sample, and then determine the proportion of “immigrant” individuals; that is, individuals with a foreign genotype that are unlikely to have originated locally.</p></li>
</ul>
</div><div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_13_19.png"></p>
<figcaption>After <span class="citation" data-cites="Allendorf2022">Allendorf et al. (<a href="#/references" role="doc-biblioref" onclick="">2022</a>)</span></figcaption>
</figure>
</div>
</div></div>
</section>
<section id="assignment-tests-1" class="slide level2">
<h2>Assignment tests</h2>

<img data-src="./galaStru.png" class="r-stretch quarto-figure-center"><p class="caption">After Paradiso et al., in preparation</p></section>
<section id="hybridization" class="slide level2">
<h2>Hybridization</h2>
<ul>
<li><p>Rates of hybridization and introgression have increased dramatically worldwide because of widespread intentional and incidental translocations of organisms and habitat modifications by humans <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</p></li>
<li><p>The increasing pace of the three interacting human activities that contribute most to increased rates of hybridization (introductions of plants and animals, fragmentation, and habitat modification) suggests that this problem will become even more serious in the future <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span></p></li>
<li><p>For example, increased turbidity in Lake Victoria, Africa, has reduced color perception of cichlid fishes and has interfered with the mate choice that produced reproductive isolation among species <span class="citation" data-cites="Seehausen1997 Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>; <a href="#/references" role="doc-biblioref" onclick="">Seehausen, Alphen, &amp; Witte, 1997</a>)</span></p></li>
</ul>
</section>
<section id="hybridization-1" class="slide level2">
<h2>Hybridization</h2>
<ul>
<li><p>On the other hand, hybridization is a natural part of the evolutionary process <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</p></li>
<li><p>Hybridization has long been recognized as playing an important role in the evolution of plants.</p></li>
<li><p>In addition, recent studies have found that hybridization has also played an important role in the evolution of animals</p></li>
<li><p>Many early conservation policies generally did not allow protection of hybrids. However, increased appreciation of the important role of hybridization as an evolutionary process has caused a re-evaluation of these policies. Determining whether hybridization is natural or anthropogenic is crucial for conservation, but is often difficult <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</p></li>
</ul>
</section>
<section id="hybridization-2" class="slide level2">
<h2>Hybridization</h2>

<img data-src="./class_13_20.png" class="r-stretch quarto-figure-center"><p class="caption">Microsatellite gel image showing genotype profiles (locus Lc106) for 10 lynx, 10 bobcat, and three putative hybrids. After <span class="citation" data-cites="Schwartz2004">Schwartz et al. (<a href="#/references" role="doc-biblioref" onclick="">2004</a>)</span></p></section>
<section id="hybridization-3" class="slide level2">
<h2>Hybridization</h2>

<img data-src="./class_13_21.png" class="r-stretch quarto-figure-center"><p class="caption">Which is which? Photo credit: G. Colosimo.</p></section>
<section id="hybridization-4" class="slide level2">
<h2>Hybridization</h2>

<img data-src="./class_13_22.png" class="r-stretch quarto-figure-center"><p class="caption">Demographic (a) and genetic (b) swamping. After <span class="citation" data-cites="Allendorf2022">Allendorf et al. (<a href="#/references" role="doc-biblioref" onclick="">2022</a>)</span></p></section>
<section class="slide level2">

<p>How can we use all the genetic data we have learnt to gather and analyse?</p>
<p><br></p>
<div class="fragment">
<ol type="1">
<li>Clarification of relatedness, taxonomy and phylogeny among populations.</li>
</ol>
</div>
<div class="fragment">
<ol start="2" type="1">
<li>Determination of population management units based on genetic criteria.</li>
</ol>
</div>
<div class="fragment">
<ol start="3" type="1">
<li>Estimation of gene flow and dispersal among populations.</li>
</ol>
</div>
<div class="fragment">
<ol start="4" type="1">
<li>Determination of the time since past genetic bottlenecks in a population or the time since significant differentiation between populations.</li>
</ol>
</div>
<div class="fragment">
<ol start="5" type="1">
<li>Understanding patterns of reproductive ecology.</li>
</ol>
</div>
<div class="fragment">
<ol start="6" type="1">
<li>Locating original sources of wildlife products.</li>
</ol>
</div>
</section>
<section id="clarification-of-relatedness-taxonomy-and-phylogeny-among-populations." class="slide level2">
<h2>1. Clarification of relatedness, taxonomy and phylogeny among populations.</h2>
<ul>
<li>The most basic question in conservation biology: how many species are there?</li>
<li>Assessing the taxonomy of one population or species still remains one of the most important applications of conservation genetics.</li>
<li>Until 1970 various taxonomic units were classified using morphological features.</li>
<li>In 1973 the Endangered Species Act (ESA, US Gov.) extended legal protection to species and subspecies.</li>
</ul>
<blockquote>
<p>When taxonomic distinction became the basis for legal protection…the innocence was lost forever <span class="citation" data-cites="Obrien1996">(<a href="#/references" role="doc-biblioref" onclick="">O’Brien, 1996</a>)</span>.</p>
</blockquote>
</section>
<section id="determination-of-population-management-units-based-on-genetic-criteria." class="slide level2">
<h2>2. Determination of population management units based on genetic criteria.</h2>
<ul>
<li><p>On what basis we delimit meaningful management units in the wild?</p></li>
<li><p>The genetic differences as measured by the <span class="math inline">\(F_{ST}\)</span> statistics provide an objective criterion for defining population management units.</p></li>
<li><p>Combining nuclear and mtDNA markers it was possible to identify 4 demographically autonomous population units of <em>Elephas maximus</em>, a northeastern India unit, a central India unit, and 2 genetically distinct unit in the south of India to be administrered separately because of their genetic uniqueness. These units were radically different from what had been proposed solely on the political considerations <span class="citation" data-cites="Vidya2005">(<a href="#/references" role="doc-biblioref" onclick="">Vidya, Fernando, Melnick, &amp; Sukumar, 2005</a>)</span>.</p></li>
</ul>
</section>
<section id="estimation-of-gene-flow-and-dispersal-among-populations." class="slide level2">
<h2>3. Estimation of gene flow and dispersal among populations.</h2>
<ul>
<li><p>Managing gene flow between populations, through a detailed analysis of <span class="math inline">\(F_{ST}\)</span> patterns, can help to better understand and define the individuals that could be used in translocation projects, when to do so, and the duration of the whole translocation process <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</p></li>
<li><p>Consideration about gene flow rates are extremely important in the management of wild plant and animal populations.</p></li>
<li><p><span class="citation" data-cites="Dolan1999">Dolan, Yahr, Menges, &amp; Halfhill (<a href="#/references" role="doc-biblioref" onclick="">1999</a>)</span> indirectly estimated gene flow measuring the genetic differentiation of three rare species of perennial scrubs in Florida. In one of them, <em>Hypericum cumulicola</em>, gene flow was absent. The limited genetic variation detected in this species limits the adaptive response of it to future environmental changes and makes it more prone to extinction.</p></li>
</ul>
</section>
<section id="determination-of-the-time-since-past-genetic-bottlenecks." class="slide level2">
<h2>4. Determination of the time since past genetic bottlenecks.</h2>

<img data-src="./class_13_23.png" class="r-stretch quarto-figure-center"><p class="caption">Each point shown represents a population and the vertical segment represents the age of its associated founder event where the segment length is proportional to the founder age <span class="citation" data-cites="Tournebize2022">(<a href="#/references" role="doc-biblioref" onclick="">Tournebize, Chu, &amp; Moorjani, 2022</a>)</span>.</p></section>
<section id="understanding-patterns-of-reproductive-ecology." class="slide level2">
<h2>5. Understanding patterns of reproductive ecology.</h2>

<img data-src="./class_13_24.png" class="r-stretch quarto-figure-center"><p class="caption">Structure graphical output for two syntopic iguana species from the Galápagos islands <span class="citation" data-cites="DiGiambattista2018">(<a href="#/references" role="doc-biblioref" onclick="">Di Giambattista et al., 2018</a>)</span>.</p></section>
<section class="slide level2">


<img data-src="./class_13_25.png" class="r-stretch quarto-figure-center"><p class="caption">Adapted from <span class="citation" data-cites="MacLeod2015">MacLeod et al. (<a href="#/references" role="doc-biblioref" onclick="">2015</a>)</span>.</p></section>
<section id="locating-original-sources-of-wildlife-products." class="slide level2">
<h2>6. Locating original sources of wildlife products.</h2>
<ul>
<li><p>The International Whaling Commission (IWC) imposed a moratorium on commercial whaling in 1982 <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</p></li>
<li><p>A small quota of very specific species of whales was still available to countries like Japan and South Korea.</p></li>
</ul>
<blockquote>
<p>The source of these products was assumed to be undocumented `incidental takes’ from fisheries’ by-catch, although we cannot exclude the possibility of illegal hunting or smuggling. The demographic impact of this undocumented exploitation was evaluated using the model of population dynamics adopted by the Scientific Committee of the International Whaling Commission. For the range of exploitation consistent with the market sample, this protected stock was predicted to decline towards extinction over the next few decades <span class="citation" data-cites="Baker2000">(<a href="#/references" role="doc-biblioref" onclick="">Baker, Lento, Cipriano, &amp; Palumbi, 2000</a>)</span>.</p>
</blockquote>
</section>
<section id="managing-genetic-diversity-in-wild-populations" class="slide level2">
<h2>Managing genetic diversity in wild populations</h2>
<ul>
<li><p>The IUCN lists the common adder, <em>Vipera berus</em>, in the category <a href="https://www.iucnredlist.org/species/47756146/743903">Least Concern</a>. It has a very widespread distribution, although populations are fragmented in small and often isolated demes.</p></li>
<li><p>One isolated population of this species, in Sweden, declined in 1960s and then began to exhibit the a high proportion of deformed and stillborn offspring coupled with low genetic variability, i.e., signs of inbreeding depression <span class="citation" data-cites="Madsen1999">(<a href="#/references" role="doc-biblioref" onclick="">Madsen, Shine, Olsson, &amp; Wittzell, 1999</a>)</span>.</p></li>
<li><p><span class="citation" data-cites="Madsen1999">Madsen et al. (<a href="#/references" role="doc-biblioref" onclick="">1999</a>)</span> introduced 20 young adder individuals collected from larger and more genetically diverse population in this declining population. After only 4 years, the time needed for an adder to become reproductively active, started to document a reduction in still born babies and a reduction in deformities and an increase in genetic diversity.</p></li>
</ul>
</section>
<section class="slide level2">


<img data-src="./class_13_26.png" class="r-stretch quarto-figure-center"><p class="caption">Numbers of adult male Adders (<em>Vipera berus</em>) captured from 1981 to 2010 along the seashore at an isolated population just west of Smygehuk Village in southern Sweden. After <span class="citation" data-cites="Madsen2011">Madsen &amp; Ujvari (<a href="#/references" role="doc-biblioref" onclick="">2011</a>)</span></p></section>
<section class="slide level2">

<h3 id="hybridization-and-introgression-are-two-important-factors-to-monitor-in-wild-populations.">Hybridization and introgression are two important factors to monitor in wild populations.</h3>
<ul>
<li><p>When two closely related species coexist and one of them is more rare, the rare one risks to be genetically swamped due to hybridization and/or introgression (the incorporation of genetic material from a species/population/subpopulation into the genome of another) <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</p></li>
<li><p>A very well documented example of managing genetic problems in wildlife population comes from the red wolf (<em>Canis rufus</em>) from south-central United States.</p></li>
<li><p>In 1975 red wolf became extinct in the wild <span class="citation" data-cites="Avise1996">(<a href="#/references" role="doc-biblioref" onclick="">Avise &amp; Hamrick, 1996</a>)</span>. The united States Fish and Wildlife Services (USFWS) initiated a campaign to re-establish a population of this species using captive-bred individuals.</p></li>
<li><p>Released individuals became established and that area was designed as the Red Wolf Experimental Population Area (RWEP-A).</p></li>
</ul>
</section>
<section class="slide level2">

<ul>
<li>The biggest threat was not much the hybridization between red wolves and coyotes (<em>Canis latrans</em>), but the backcrossing between <span class="math inline">\(F_1\)</span> hybrids and adult red wolf individuals (i.e., introgression).</li>
</ul>

<img data-src="./class_13_27.png" class="r-stretch quarto-figure-center"><p class="caption">After <span class="citation" data-cites="Fredrickson2006">Fredrickson &amp; Hedrick (<a href="#/references" role="doc-biblioref" onclick="">2006</a>)</span></p></section>
<section id="managing-genetic-diversity-in-captive-populations" class="slide level2">
<h2>Managing genetic diversity in captive populations</h2>
<ul>
<li><p>Attempts at turkey reintroduction using captive bred individuals were a failure in 95% of cases in the united States <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</p></li>
<li><p>As <span class="citation" data-cites="Leopold1944">Leopold (<a href="#/references" role="doc-biblioref" onclick="">1944</a>)</span> said:</p></li>
</ul>
<blockquote>
<p>Wild turkey are wary and shy, which are advantageous characteristics in eluding natural and human enemies. The hens and young ones automatically react to threats in ways that are self protective…Birds of the domestic strain, on the other hand, are differently adapted. Many of their physiological reaction and psychological characteristics are favorable to existence in the barnyard but may perclude existence in the wild.</p>
</blockquote>
<ul>
<li>The problem stated above is present in every single population and it is known as <em>adaptation to captivity</em> <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</li>
</ul>
</section>
<section class="slide level2">

<ul>
<li><p>The more accustomed wild animals become to their artificial environment the less likely they are to survive when reintroduced in the wild.</p></li>
<li><p>Lifetime reproductive success of and overall fitness of stealhead trout (<em>Oncorhynchus mykiss</em>) declined precipitously in just two generations of captivity <span class="citation" data-cites="Araki2007">(<a href="#/references" role="doc-biblioref" onclick="">Araki, Cooper, &amp; Blouin, 2007</a>)</span>.</p></li>
<li><p>The data provided in <span class="citation" data-cites="Araki2007">Araki et al. (<a href="#/references" role="doc-biblioref" onclick="">2007</a>)</span> suggest that 37.5% fitness decline per captive reared generation demonstrating that genetic adaptation (GA) to captivity is something dangerous that should not be underestimated.</p></li>
<li><p>GA can be mathematically described:</p></li>
</ul>
<p><span class="math display">\[
\label{ga}
\tag{8.1}
GA = \frac{Sh^2}{L}\sum_{i=1}^{\frac{y}{L}}[1-\frac{1}{2N_e}]^{\frac{y}{L}}(1-m_i)
\]</span></p>
</section>
<section class="slide level2">

<ul>
<li>Where <span class="math inline">\(S\)</span> is the selection differential in captivity (the strength in selection for captive traits).</li>
<li><span class="math inline">\(h^2\)</span> is heritability (additive genetic variation for reproductive fitness).</li>
<li><span class="math inline">\(y/L\)</span> number of generations in captivity (<span class="math inline">\(y\)</span> = years, <span class="math inline">\(L\)</span> = generation length).</li>
<li><span class="math inline">\(N_e\)</span>, you should recognize this one….</li>
<li><span class="math inline">\(m_i\)</span> proportion of genetic material coming from immigrants.</li>
</ul>
</section>
<section id="what-could-we-do-to-minimize-ga" class="slide level2">
<h2>What could we do to minimize GA?</h2>
<ul>
<li>Reduce the amount of time in captivity (i.e., reduce the number of iterations <span class="math inline">\(\sum_{i=1}^{y/L}\)</span>).</li>
<li>Reduce the strength of selection, <span class="math inline">\(S\)</span>, in captivity. Very complicated, as it entails making the captive environment as similar as possible to the natural environment, both at the biotic and the abiotic level.</li>
<li>Add more immigrants (unrelated individuals) from wild or captive populations to slow the rate of genetic adaptation.</li>
</ul>
<p>Understanding the intentions, methods and scope of the management plans is essential if we want to grasp the way in which captive populations are integrated into worldwide conservation strategies today <span class="citation" data-cites="Allendorf2022">(<a href="#/references" role="doc-biblioref" onclick="">Allendorf et al., 2022</a>)</span>.</p>
</section>
<section id="heritability" class="slide level2">
<h2>Heritability</h2>
<ul>
<li><p>The heritability of a character is probably the most important determinant of individuals to evolve in response to selection!</p></li>
<li><p>Heritability is the proportion of phenotypic variance in a population due to genetic differences that parents can pass through offspring in the next generation.</p></li>
<li><p>Heritability is symbolized <span class="math inline">\(h^2\)</span>, but the superscript has no mathematical meaning!</p></li>
<li><p>Molecular measures of genetic diversity do not often answer some of the questions we try to address in conservation genetics as correlation between molecular and quantitative measures of genetic diversity are low.</p></li>
<li><p>The heritability associated with quantitative characters that are related to reproductive fitness are of great interests to conservation biologists.</p></li>
</ul>
</section>
<section class="slide level2">


<img data-src="./class_13_28.jpg" class="r-stretch quarto-figure-center"><p class="caption">Distribution of phenotypes for 5 quantitative characters that are very important for reproductive fitness. After <span class="citation" data-cites="Frankham2010">Frankham, Ballou, &amp; Briscoe (<a href="#/references" role="doc-biblioref" onclick="">2010</a>)</span></p></section>
<section class="slide level2">

<ul>
<li><p>Quantitative traits are influenced by mani loci plus environmental influences (feeding, exposure to sunlight, water availability et. etc. etc.).</p></li>
<li><p>It is not possible to directly infer genotype from phenotype only when we refer to quantitative characters, and individuals with the same phenotype may have different genotypes, or individuals with the same genotypes may have different phenotype.</p></li>
<li><p>Genetic diversity for quantitative characters in outbred populations is due to the segregation of multiple mendelian loci referred to as Quantitative Trait Loci (QTL).</p></li>
<li><p>QTLs possess alleles that add to or detract from the magnitude of the quantitative character.</p></li>
<li><p>How do we measure what portion of the phenotypic variance for quantitative traits derives from genetic variation among individuals?</p></li>
</ul>
</section>
<section class="slide level2">

<ol type="1">
<li>Variation within and among populations.</li>
<li>Comparison of inbred with outbred populations.</li>
<li>Resemblance among relatives.</li>
</ol>
<ul>
<li>Because of the great impact that the environment can have on the expression of quantitative traits characters it is extremely important that all comparisons are carried out in a controlled situation and synchronously (i.e., at the same time): common garden experiment.</li>
</ul>
</section>
<section class="slide level2">

<p>When talking about phenotypes we have mentioned that a measurable phenotype is the consequence of the underlying genotypes inherited by the parents plus the influence of the environment. We can represent this relation in mathematical terms in the following way:</p>
<p><span class="math display">\[
\label{p}
\tag{8.2}
P = G + E
\]</span></p>
<p>where <span class="math inline">\(P\)</span> = phenotye, <span class="math inline">\(G\)</span> = genotype, and <span class="math inline">\(E\)</span> = environment. Being able to distinguish between the genetic and the environmental component in this interaction is important to understand the evolutionary trajectory of a population or species <span class="citation" data-cites="Frankham2010">(<a href="#/references" role="doc-biblioref" onclick="">Frankham et al., 2010</a>)</span>.</p>
<p>Different phenotypes within a population, also known as phenotypic variance <span class="math inline">\(V_P\)</span>, represents the sum total of all the interactions between the genotypic variance, <span class="math inline">\(V_G\)</span>, the environmental variance, <span class="math inline">\(V_E\)</span>, and the interaction term between these two quantities, <span class="math inline">\(G*E\)</span>.</p>
<p><span class="math display">\[
\label{Vp}
\tag{8.3}
V_P = V_G + V_E + 2Cov_{GE}
\]</span></p>
</section>
<section class="slide level2">

<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_14_7.jpg" style="width:70.0%"></p>
<figcaption>Quantitative genetic variation for corolla length in tobacco. After <span class="citation" data-cites="Strickberger1985">Strickberger (<a href="#/references" role="doc-biblioref" onclick="">1985</a>)</span>.</figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<ul>
<li>Variances for corolla length in the two homozygous parent population are 48 and 32 respectively.</li>
<li>Variance in the <span class="math inline">\(F_1\)</span> population is 46, while variance in the <span class="math inline">\(F_2\)</span> population is 130.5.</li>
<li>The variances in the parental and <span class="math inline">\(F_1\)</span> populations are due solely on environmental variance, <span class="math inline">\(V_E\)</span>.</li>
<li>We can average the three separate estimates of <span class="math inline">\(V_E = \frac{48+32+46}{3}=42\)</span>.</li>
<li>In <span class="math inline">\(F_2\)</span> we need to account for both genetic variation and environmental variation: <span class="math inline">\(V_{F_2} = V_G+V_E = 130.5\)</span>, and we can rearrange the equation as <span class="math inline">\(V_G = V_{F_2} - V_E = 130.5- 42 = 88.5\)</span>.</li>
</ul>
</div></div>
</section>
<section class="slide level2">

<ul>
<li>Based on the previous example, we can conclude that in <span class="math inline">\(F_2\)</span> generation <span class="math inline">\(88/130.5 = 68\%\)</span> of variance is due to segregation of polymorphic loci while only <span class="math inline">\(100-68=32\%\)</span> of the total phenotypic variance is due to environmental factors (<span class="math inline">\(V_E\)</span>).</li>
<li>In common garden experiment the covariance term in equation <span class="math inline">\(\ref{p}\)</span> can be kept under control by standardizing the experimental condition for the animal or plant under study.</li>
<li>This is way more complicated to obtain in wild populations as the there is no easy way to keep under control the many environmental variables that play into the reproductive fitness of individuals.</li>
</ul>
<div class="fragment">
<h3 id="further-partitioning-v_g">Further partitioning <span class="math inline">\(V_G\)</span></h3>
<p>We can further divide the genetic variance in three subcomponents: additive genetic variance (<span class="math inline">\(V_A\)</span>), dominance variance (<span class="math inline">\(V_D\)</span>) and interacting variance (<span class="math inline">\(V_I\)</span>).</p>
<p><span class="math display">\[
\label{Vg}
\tag{8.4}
V_G= V_A+V_D+V_I
\]</span></p>
</div>
</section>
<section class="slide level2">

<ul>
<li><span class="math inline">\(V_A\)</span> reflects variation in the avergae effect of the many loci involved in the overall genotypic variance. This measure, and in particular the ratio <span class="math inline">\(V_A/V_P\)</span> (also known as heritability) reflects the evolutionary potential of the population or species under study <span class="citation" data-cites="Frankham2010">(<a href="#/references" role="doc-biblioref" onclick="">Frankham et al., 2010</a>)</span>.</li>
<li><span class="math inline">\(V_D\)</span> measures variation in dominance deviation.</li>
<li><span class="math inline">\(V_I\)</span> is a measure of epistasis, i.e.&nbsp;the variation in interaction deviations among genes. This can also be interpreted as a measure of the effects of outbreeding, whether beneficial or deleterious <span class="citation" data-cites="Frankham2010">(<a href="#/references" role="doc-biblioref" onclick="">Frankham et al., 2010</a>)</span>.</li>
</ul>
<div class="fragment">
<ul>
<li>To better understand the meaning of <span class="math inline">\(V_A\)</span> and <span class="math inline">\(V_D\)</span> we can work a simple model to help illustrate how we can partition the genetic variance in its component, in particular the additive and the dominant ones (<span class="math inline">\(V_A\)</span> and <span class="math inline">\(V_D\)</span>)!</li>
</ul>
</div>
</section>
<section class="slide level2">

<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_15_1.jpg"></p>
<figcaption>Genotypic values assigned to three genotypes at a locus along with their frequencies with random mating <span class="citation" data-cites="Frankham2010">(<a href="#/references" role="doc-biblioref" onclick="">Frankham et al., 2010</a>)</span></figcaption>
</figure>
</div>
<div class="fragment">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_15_2.jpg"></p>
<figcaption>Genotypic values for additive, dominant and overdominant loci <span class="citation" data-cites="Frankham2010">(<a href="#/references" role="doc-biblioref" onclick="">Frankham et al., 2010</a>)</span></figcaption>
</figure>
</div>
</div>
</section>
<section class="slide level2">

<ul>
<li>In Merino sheep the <span class="math inline">\(B\)</span> allele increases the litter size. Based on some experimental studies we can compute the following average litter size for the three distinct genotypes:</li>
</ul>
<table class="caption-top">
<thead>
<tr class="header">
<th>BB</th>
<th>Bb</th>
<th>bb</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>2.66</td>
<td>2.17</td>
<td>1.48</td>
</tr>
</tbody>
</table>
<ul>
<li>We can now compute the value of <span class="math inline">\(a\)</span> as:</li>
</ul>
<p><span class="math display">\[
a = \frac{BB \ value- bb \ value}{2} = \frac{2.66-1.48}{2}=0.59
\]</span></p>
<ul>
<li>Based on our model, the mid-point between the two homozygotes is the zero point on the scale of genotypic values, and we can use it to determine <span class="math inline">\(d\)</span>. It is calculated as <span class="math inline">\((2.66-0.59 = 2.07)\)</span>. We can now calculate the dominance deviation <span class="math inline">\(d\)</span> as:</li>
</ul>
<p><span class="math display">\[
d= Bb \ value - mid \ point = 2.17-2.07=0.10
\]</span></p>
</section>
<section class="slide level2">

<ul>
<li>Each copy of the <span class="math inline">\(B\)</span> allele increases the litter size by an average of 0.59 offspring and this allele presents partial dominance with the heterozygote having 0.10 offspring above the <span class="math inline">\(bb\)</span> homozygote.</li>
</ul>
<div class="fragment">
<h3 id="focusing-on-additive-genetic-variance-v_a">Focusing on additive genetic variance (<span class="math inline">\(V_A\)</span>)</h3>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_15_1.jpg"></p>
<figcaption>Genotypic values assigned to three genotypes at a locus along with their frequencies with random mating <span class="citation" data-cites="Frankham2010">(<a href="#/references" role="doc-biblioref" onclick="">Frankham et al., 2010</a>)</span></figcaption>
</figure>
</div>
<ul>
<li>For a polymorphic locus with addivie effects (<span class="math inline">\(d = 0\)</span>) and genotypic frequncies and values as in the above example, the additive genetic variation is defined as <span class="math inline">\(V_A = 2pqa^2\)</span>.</li>
</ul>
</div>
</section>
<section class="slide level2">

<ul>
<li>We can desume that <span class="math inline">\(V_A\)</span> is directly dependent on the heterozygosity (2pq) and half the difference in genotypic values between the two homozygotes (<span class="math inline">\(a\)</span>).</li>
<li>The larger the difference between the two homozygotes the larger <span class="math inline">\(V_A\)</span>.</li>
<li>On the other end, when <span class="math inline">\(d \ne 0\)</span> (there is dominance) the additive genetic variation can be computed as:</li>
</ul>
<p><span class="math display">\[
\label{vadom}
\tag{8.5}
V_A = 2pq[a+d(q-p)]^2
\]</span></p>
<ul>
<li><p>Thus, <span class="math inline">\(V_A\)</span> also depends on the dominance deviation!</p></li>
<li><p>As you can guess, <span class="math inline">\(V_A\)</span> of a specific character depends on the combined effect of all segregating loci in a population with effect on the specific quantitative character under study <span class="citation" data-cites="Frankham2010">(<a href="#/references" role="doc-biblioref" onclick="">Frankham et al., 2010</a>)</span>.</p></li>
</ul>
</section>
<section id="heritability-and-evolutionary-potential" class="slide level2">
<h2>Heritability and Evolutionary potential</h2>
<ul>
<li><p>Heritability determines the extent to which a character is passed on from parents to offspring. Only traits that are heritable will be passed on from generation to generation. A trait that is favorable, but it is not heritable will not be passed on!</p></li>
<li><p>Heritability (<span class="math inline">\(h^2\)</span>) is defined as the proportion of total phenotypic variation due to additive genetic variance:</p></li>
</ul>
<p><span class="math display">\[
\label{h2}
\tag{8.6}
h^2 = \frac{V_A}{V_P}
\]</span></p>
<ul>
<li><span class="math inline">\(h^2\)</span> value ranges from <span class="math inline">\(0\)</span> to <span class="math inline">\(1\)</span>, with <span class="math inline">\(0\)</span> characteristic of highly inbred populations with no genetic variation, and <span class="math inline">\(1\)</span> expected in traits with basically no environmental variance components and all additive genetic variance.</li>
<li>Heritability can not be extrapolated for environments that are not native to population or species under study.</li>
</ul>
</section>
<section class="slide level2">

<ul>
<li>Quantitative variation is characterized using statistical parameters including means, variances, covariances, regressions and correlations <span class="citation" data-cites="Frankham2010">(<a href="#/references" role="doc-biblioref" onclick="">Frankham et al., 2010</a>)</span>.</li>
</ul>
<div class="fragment">
<table class="caption-top">
<thead>
<tr class="header">
<th>Parent means (P)</th>
<th>Offspring means (O)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>6.8</td>
<td>7.3</td>
</tr>
<tr class="even">
<td>6.9</td>
<td>7.4</td>
</tr>
<tr class="odd">
<td>7.1</td>
<td>7.6</td>
</tr>
<tr class="even">
<td>7.3</td>
<td>7.5</td>
</tr>
<tr class="odd">
<td>7.3</td>
<td>7.3</td>
</tr>
<tr class="even">
<td>7.3</td>
<td>7.2</td>
</tr>
<tr class="odd">
<td>7.4</td>
<td>7.4</td>
</tr>
<tr class="even">
<td>7.5</td>
<td>7.7</td>
</tr>
<tr class="odd">
<td>7.5</td>
<td>7.6</td>
</tr>
</tbody>
</table>
<p>Data from <span class="citation" data-cites="Murray1968">(<a href="#/references" role="doc-biblioref" onclick="">Murray &amp; Clarke, 1968</a>)</span>.</p>
</div>
</section>
<section class="slide level2">

<ul>
<li><p>Mean: <span class="math inline">\(\bar{P}=\frac{\sum_{i=1}^n P_i}{n}\)</span> and <span class="math inline">\(\bar{O}=\frac{\sum_{i=1}^n O_i}{n}\)</span></p></li>
<li><p>Variance: <span class="math inline">\(V_{P}=\frac{\sum_{i=1}^n (P_i-\bar{P})^2}{n-1}\)</span> and <span class="math inline">\(V_{O}=\frac{\sum_{i=1}^n (O_i-\bar{O})^2}{n-1}\)</span></p></li>
<li><p>Standard deviations: <span class="math inline">\(sd_{P} = \sqrt{V_{P}}\)</span> and <span class="math inline">\(sd_{O} = \sqrt{V_{O}}\)</span></p></li>
<li><p>Covariance (<span class="math inline">\(Cov_{PO}\)</span>) measures the extent to which they vary in concert (<span class="math inline">\(+\)</span>), they vary independently (<span class="math inline">\(0\)</span>), or in opposition (<span class="math inline">\(-\)</span>): <span class="math inline">\(Cov_{PO} = \frac{\sum_{i=1}^n (P_i-\bar{P})(O_i-\bar{O})}{n-1}\)</span></p></li>
<li><p>Correlation (<span class="math inline">\(r_{PO}\)</span>) is a standardized measure of how two measurements vary in concert. Ranges from <span class="math inline">\(-1\)</span> to <span class="math inline">\(+1\)</span>: <span class="math inline">\(r_{PO} = \frac{Cov_{PO}}{\sqrt{(V_{P}*V_{O})}}\)</span></p></li>
<li><p>Regression of offspring on parents (<span class="math inline">\(b_{OP}\)</span>) is the slope of the line of best fit relating offspring and parents: <span class="math inline">\(b_{OP}= \frac{Cov_{PO}}{V_{P}}\)</span></p></li>
</ul>
</section>
<section class="slide level2">

<ul>
<li><p>Access the <a href="https://drive.proton.me/urls/5RKA9GC418#RwMAMxHGaHl2">data for excercise</a> and lets try to estimate the heritability in R.</p></li>
<li><p>Save the file in a specific folder and navigate to that folder within R.</p></li>
</ul>
<ul>
<li>Load the data file in your working environment.</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a></a><span class="fu">load</span>(<span class="st">"path_to_file.RData"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<ul>
<li>Lets take a look at the data.</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb2-1"><a></a><span class="co"># Parent means complete dataset</span></span>
<span id="cb2-2"><a></a>P</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code> [1] 6.8 6.9 6.9 7.1 7.3 7.3 7.3 7.4 7.5 7.5 7.5 7.5 7.5 7.5 7.5 7.6 7.6 7.6 7.6
[20] 7.6 7.6 7.7 7.7 7.8 7.8 7.8 7.8 7.8 7.9 7.9 7.9 7.9 7.9 8.0 8.0 8.0 8.1 8.1
[39] 8.1 8.5</code></pre>
</div>
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a></a><span class="co"># Offspring means complete dataset</span></span>
<span id="cb4-2"><a></a>O</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code> [1] 7.3 7.4 7.6 7.5 7.3 7.2 7.4 7.7 7.6 7.5 7.7 7.4 7.8 7.6 7.3 7.7 7.7 7.9 7.4
[20] 7.5 7.4 7.6 7.9 7.5 7.8 7.9 7.6 7.5 7.6 7.7 7.7 7.7 7.8 7.7 7.9 7.8 7.8 7.8
[39] 7.9 8.1</code></pre>
</div>
</div>
</section>
<section class="slide level2">

<ul>
<li>Mean</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a></a><span class="fu">mean</span>(P)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 7.645</code></pre>
</div>
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb8-1"><a></a><span class="fu">mean</span>(O)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 7.63</code></pre>
</div>
</div>
<ul>
<li>Variance</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb10"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb10-1"><a></a><span class="fu">var</span>(P)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 0.1251026</code></pre>
</div>
<div class="sourceCode cell-code" id="cb12"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb12-1"><a></a><span class="fu">var</span>(O)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 0.04266667</code></pre>
</div>
</div>
<ul>
<li>Standard deviation</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb14"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb14-1"><a></a><span class="fu">sd</span>(P)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 0.3536984</code></pre>
</div>
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb16-1"><a></a><span class="fu">sd</span>(O)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 0.2065591</code></pre>
</div>
</div>
</section>
<section class="slide level2">

<ul>
<li>Covariance</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb18"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb18-1"><a></a><span class="fu">cov</span>(P,O)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 0.04964103</code></pre>
</div>
</div>
<ul>
<li>Correlation</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb20"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb20-1"><a></a><span class="fu">cor</span>(P,O)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 0.679459</code></pre>
</div>
</div>
<ul>
<li>Regression</li>
</ul>
<div class="cell" data-layout-align="center">
<div class="sourceCode cell-code" id="cb22"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb22-1"><a></a><span class="fu">plot</span>(O <span class="sc">~</span> P, <span class="at">xlab =</span> <span class="st">"Parent means"</span>, <span class="at">ylab =</span> <span class="st">"Offspring means"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="class_8_files/figure-revealjs/unnamed-chunk-9-1.png" class="quarto-figure quarto-figure-center r-stretch" width="672"></section>
<section class="slide level2">

<div class="cell">
<div class="sourceCode cell-code" id="cb23"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb23-1"><a></a><span class="fu">summary</span>(<span class="fu">lm</span>(O <span class="sc">~</span> P))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Call:
lm(formula = O ~ P)

Residuals:
      Min        1Q    Median        3Q       Max 
-0.293103 -0.097863 -0.002623  0.094215  0.287856 

Coefficients:
            Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)  4.59644    0.53196   8.641 1.68e-10 ***
P            0.39680    0.06951   5.709 1.43e-06 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Residual standard error: 0.1535 on 38 degrees of freedom
Multiple R-squared:  0.4617,    Adjusted R-squared:  0.4475 
F-statistic: 32.59 on 1 and 38 DF,  p-value: 1.429e-06</code></pre>
</div>
</div>
<ol type="1">
<li>The estimate for the model parameters – the value of the y-intercept (in this case 4.596) and the estimated effect of parents on offspring (0.396).</li>
<li>The standard error of the estimated values (Std. Error).</li>
<li>The test statistic (t value, in this case the t statistic).</li>
<li>The p value ( Pr(&gt;| t | ) ), aka the probability of finding the given t statistic if the null hypothesis of no relationship were true.</li>
</ol>
</section>
<section class="slide level2">

<div class="cell">
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb25-1"><a></a><span class="fu">plot</span>(O <span class="sc">~</span> P, <span class="at">xlab =</span> <span class="st">"Parent means"</span>, <span class="at">ylab =</span> <span class="st">"Offspring means"</span>)</span>
<span id="cb25-2"><a></a><span class="fu">abline</span>(<span class="fu">lm</span>(O <span class="sc">~</span> P))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="class_8_files/figure-revealjs/unnamed-chunk-11-1.png" width="960" class="r-stretch"></section>
<section class="slide level2">


<img data-src="./class_15_3.jpg" class="r-stretch quarto-figure-center"><p class="caption">Hypothetical relationships between mean values of parents and mean values of offspring for three cases, representing (a) complete, (b) incomplete, and (c) zero relationships between parents and offspring <span class="citation" data-cites="Frankham2010">(<a href="#/references" role="doc-biblioref" onclick="">Frankham et al., 2010</a>)</span>.</p><ol type="a">
<li>Parents with larger values produce offspring with larger values, <span class="math inline">\(h^2 = 1\)</span>.</li>
<li>Parents with high phenotypic values produce offspring with slightly higher phenotypic values, but still close to the population mean, <span class="math inline">\(h^2\)</span> has an intermediate value.</li>
<li>Parents with high and low values have similar offspring.</li>
</ol>
</section>
<section id="predicting-response-to-selection" class="slide level2">
<h2>Predicting response to selection</h2>
<ul>
<li>The genetic change (response) produced by directional selection can be predicted from the intensity of selection and the heritability.</li>
</ul>
<div class="fragment">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_15_4.jpg" style="width:40.0%"></p>
<figcaption>Forms of selection operating on quantitative characters After <span class="citation" data-cites="Frankham2010">Frankham et al. (<a href="#/references" role="doc-biblioref" onclick="">2010</a>)</span> and <span class="citation" data-cites="Futuyma1979">Futuyma (<a href="#/references" role="doc-biblioref" onclick="">1979</a>)</span>.</figcaption>
</figure>
</div>
</div>
</section>
<section class="slide level2">


<img data-src="./class_15_5.jpg" class="r-stretch quarto-figure-center"><p class="caption">Predicting response to directional selection. Individuals with the highest values are selected to be parents (filled circles), and their mean phenotype value, <span class="math inline">\(S\)</span>, is higher than the mean of the general parental population After <span class="citation" data-cites="Frankham2010">(<a href="#/references" role="doc-biblioref" onclick="">Frankham et al., 2010</a>)</span>.</p></section>
<section class="slide level2">

<ul>
<li><span class="math inline">\(S\)</span> is termed selection differential.</li>
<li><span class="math inline">\(R\)</span> is termed response to selection (the difference in the offspring of selected parents against the offspring of all parents) can be measured as:</li>
</ul>
<p><span class="math display">\[
\label{R}
\tag{8.7}
R = Sh^2
\]</span></p>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="./class_15_6.jpg" style="width:30.0%"></p>
<figcaption>Darwin’s medium ground finches from Galápagos. After <span class="citation" data-cites="Frankham2010">Frankham et al. (<a href="#/references" role="doc-biblioref" onclick="">2010</a>)</span></figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<ul>
<li>Over 18 months of drought and no breeding the finches suffered 85% mortality.</li>
<li>Survivors had beaks that were 0.25mm wider than the original population (<span class="math inline">\(S\)</span>).</li>
<li>The heritability for bill width in this population is 74.5%.</li>
<li>Using this value we can predict the genetic change of the offspring of these survivors</li>
</ul>
</div></div>
</section>
<section id="references" class="slide level2 smaller scrollable">
<h2>References</h2>


<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" data-line-spacing="2" role="list">
<div id="ref-Allendorf2022" class="csl-entry" role="listitem">
Allendorf, F. W., Funk, W. C., Aitken, S. N., Byrne, M., Luikart, G., &amp; Antunes, A. (2022). <em><span class="nocase">Conservation and the Genomics of Populations</span></em>. Oxford University Press. Retrieved from <a href="https://doi.org/10.1093/oso/9780198856566.001.0001">https://doi.org/10.1093/oso/9780198856566.001.0001</a>
</div>
<div id="ref-Araki2007" class="csl-entry" role="listitem">
Araki, H., Cooper, B., &amp; Blouin, M. S. (2007). Genetic effects of captive breeding cause a rapid, cumulative fitness decline in the wild. <em>Science</em>, <em>318</em>(5847), 100–103. Retrieved from <a href="https://www.science.org/doi/abs/10.1126/science.1145621">https://www.science.org/doi/abs/10.1126/science.1145621</a>
</div>
<div id="ref-Avise1996" class="csl-entry" role="listitem">
Avise, J. C., &amp; Hamrick, J. L. (1996). <em>Conservation genetics : Case histories from nature</em>. (J. C. Avise &amp; J. L. Hamrick, Eds.)<em>Conservation genetics : case histories from nature /</em>. New York: Chapman &amp; Hall.
</div>
<div id="ref-Baker2000" class="csl-entry" role="listitem">
Baker, C. S., Lento, G. M., Cipriano, F., &amp; Palumbi, S. R. (2000). Predicted decline of protected whales based on molecular genetic monitoring of japanese and korean markets. <em>Proceedings of the Royal Society of London. Series B: Biological Sciences</em>, <em>267</em>(1449), 1191–1199. The Royal Society. Retrieved from <a href="https://doi.org/10.1098%2Frspb.2000.1128">https://doi.org/10.1098%2Frspb.2000.1128</a>
</div>
<div id="ref-DiGiambattista2018" class="csl-entry" role="listitem">
Di Giambattista, L., Fulvo, A., Fabiani, A., Bonanni, J., Carrión, J. E., &amp; Gentile, G. (2018). <span class="nocase">Molecular data exclude current hybridization between iguanas <em>Conolophus marthae</em> and <em>C. subcristatus</em> on Wolf Volcano (Gal<span class="nocase">á</span>pagos Islands)</span>. <em>Conservation Genetics</em>, <em>19</em>(6), 1461–1469. Springer Netherlands. Retrieved from <a href="http://link.springer.com/10.1007/s10592-018-1114-3">http://link.springer.com/10.1007/s10592-018-1114-3</a>
</div>
<div id="ref-Dolan1999" class="csl-entry" role="listitem">
Dolan, R. W., Yahr, R., Menges, E. S., &amp; Halfhill, M. D. (1999). Conservation implications of genetic variation in three rare species endemic to florida rosemary scrub. <em>American Journal of Botany</em>, <em>86</em>(11), 1556–1562. Botanical Society of America. Retrieved May 8, 2023, from <a href="http://www.jstor.org/stable/2656793">http://www.jstor.org/stable/2656793</a>
</div>
<div id="ref-Frankham2010" class="csl-entry" role="listitem">
Frankham, R., Ballou, J. D., &amp; Briscoe, D. A. (2010). <em>Introduction to conservation genetics</em>. Cambridge University Press. Retrieved from <a href="https://doi.org/10.1017%2Fcbo9780511809002">https://doi.org/10.1017%2Fcbo9780511809002</a>
</div>
<div id="ref-Fredrickson2006" class="csl-entry" role="listitem">
Fredrickson, R. J., &amp; Hedrick, P. W. (2006). Dynamics of hybridization and introgression in red wolves and coyotes. <em>Conservation Biology</em>, <em>20</em>(4), 1272–1283. Retrieved from <a href="https://conbio.onlinelibrary.wiley.com/doi/abs/10.1111/j.1523-1739.2006.00401.x">https://conbio.onlinelibrary.wiley.com/doi/abs/10.1111/j.1523-1739.2006.00401.x</a>
</div>
<div id="ref-Futuyma1979" class="csl-entry" role="listitem">
Futuyma, D. J. (1979). <em>Evolutionary biology</em>. Sinauer Associates, Inc.
</div>
<div id="ref-Leopold1944" class="csl-entry" role="listitem">
Leopold, A. S. (1944). <span class="nocase">The Nature of Heritable Wildness in Turkeys</span>. <em>The Condor</em>, <em>46</em>(4), 133–197. Retrieved from <a href="https://doi.org/10.2307/1364213">https://doi.org/10.2307/1364213</a>
</div>
<div id="ref-MacLeod2015" class="csl-entry" role="listitem">
MacLeod, A., Rodríguez, A., Vences, M., Orozco-terWengel, P., García, C., Trillmich, F., Gentile, G., et al. (2015). <span class="nocase">Hybridization masks speciation in the evolutionary history of the Gal<span class="nocase">á</span>pagos marine iguana</span>. <em>Proceedings of the Royal Society B: Biological Sciences</em>, <em>282</em>(1809), 20150425. Retrieved from <a href="http://rspb.royalsocietypublishing.org/lookup/doi/10.1098/rspb.2015.0425">http://rspb.royalsocietypublishing.org/lookup/doi/10.1098/rspb.2015.0425</a>
</div>
<div id="ref-Madsen1999" class="csl-entry" role="listitem">
Madsen, T., Shine, R., Olsson, M., &amp; Wittzell, H. (1999). <span class="nocase">Conservation biology: Restoration of an inbred adder population</span>. <em>Nature</em>, <em>402</em>(6757), 34–35. Retrieved from <a href="http://www.nature.com/nature/journal/v402/n6757/pdf/402034a0.pdf">http://www.nature.com/nature/journal/v402/n6757/pdf/402034a0.pdf</a>
</div>
<div id="ref-Madsen2011" class="csl-entry" role="listitem">
Madsen, T., &amp; Ujvari, B. (2011). <span class="nocase">The potential demise of a population of adders (<em>Vipera berus</em>) in Smygehuk, Sweden</span>. <em>Herpetological Conservation and Biology</em>, <em>6</em>(1), 72–74.
</div>
<div id="ref-Murray1968" class="csl-entry" role="listitem">
Murray, J., &amp; Clarke, B. (1968). Inheritance of shell size in partula. <em>Heredity</em>, <em>23</em>(2), 189–198. Springer Science; Business Media <span>LLC</span>. Retrieved from <a href="https://doi.org/10.1038%2Fhdy.1968.27">https://doi.org/10.1038%2Fhdy.1968.27</a>
</div>
<div id="ref-Obrien1996" class="csl-entry" role="listitem">
O’Brien, S. J. (1996). Conservation genetics of the felidae. In J. C. Avise &amp; J. L. Hamrick (Eds.), <em>Conservation genetics: Case histories from nature.</em> (pp. 50–74). New York: Chapman; Hall.
</div>
<div id="ref-Schwartz2004" class="csl-entry" role="listitem">
Schwartz, M. K., Pilgrim, K. L., McKelvey, K. S., Lindquist, E. L., Claar, J. J., Loch, S., &amp; Ruggiero, L. F. (2004). Hybridization between canada lynx and bobcats: Genetic results and management implications. <em>Conservation Genetics</em>, <em>5</em>(3), 349–355. Retrieved from <a href="https://link.springer.com/content/pdf/10.1023/B:COGE.0000031141.47148.8b.pdf">https://link.springer.com/content/pdf/10.1023/B:COGE.0000031141.47148.8b.pdf</a>
</div>
<div id="ref-Seehausen1997" class="csl-entry" role="listitem">
Seehausen, O., Alphen, J. J. M. van, &amp; Witte, F. (1997). Cichlid fish diversity threatened by eutrophication that curbs sexual selection. <em>Science</em>, <em>277</em>(5333), 1808–1811. Retrieved from <a href="https://www.science.org/doi/abs/10.1126/science.277.5333.1808">https://www.science.org/doi/abs/10.1126/science.277.5333.1808</a>
</div>
<div id="ref-Strickberger1985" class="csl-entry" role="listitem">
Strickberger, M. W. (1985). <em>Genetics</em>. Macmillan international editions in science. Macmillan. Retrieved from <a href="https://books.google.co.uk/books?id=IWrwAAAAMAAJ">https://books.google.co.uk/books?id=IWrwAAAAMAAJ</a>
</div>
<div id="ref-Tournebize2022" class="csl-entry" role="listitem">
Tournebize, R., Chu, G., &amp; Moorjani, P. (2022). Reconstructing the history of founder events using genome-wide patterns of allele sharing across individuals. (K. E. Lohmueller, Ed.)<em><span>PLOS</span> Genetics</em>, <em>18</em>(6), e1010243. Public Library of Science (<span>PLoS</span>). Retrieved from <a href="https://doi.org/10.1371%2Fjournal.pgen.1010243">https://doi.org/10.1371%2Fjournal.pgen.1010243</a>
</div>
<div id="ref-Vidya2005" class="csl-entry" role="listitem">
Vidya, T. N. C., Fernando, P., Melnick, D. J., &amp; Sukumar, R. (2005). <span class="nocase">Population genetic structure and conservation of Asian elephants (<em>Elephas maximus</em>) across India</span>. <em>Animal Conservation</em>, <em>8</em>(4), 377–388. Retrieved from <a href="https://zslpublications.onlinelibrary.wiley.com/doi/abs/10.1017/S1367943005002428">https://zslpublications.onlinelibrary.wiley.com/doi/abs/10.1017/S1367943005002428</a>
</div>
</div>
</section>
    </div>
  <div class="quarto-auto-generated-content" style="display: none;">
<p><img src="../../figures/Tor-Vergata_logo_cmyk.png" class="slide-logo"></p>
<div class="footer footer-default">
<p><a href="https://giulianocolosimo.github.io/biologia_della_conservazione_24-25/">Biologia della conservazione 24/25 - Giuliano Colosimo, PhD</a></p>
</div>
</div></div>

  <script>window.backupDefine = window.define; window.define = undefined;</script>
  <script src="../../site_libs/revealjs/dist/reveal.js"></script>
  <!-- reveal.js plugins -->
  <script src="../../site_libs/revealjs/plugin/quarto-line-highlight/line-highlight.js"></script>
  <script src="../../site_libs/revealjs/plugin/pdf-export/pdfexport.js"></script>
  <script src="../../site_libs/revealjs/plugin/reveal-menu/menu.js"></script>
  <script src="../../site_libs/revealjs/plugin/reveal-menu/quarto-menu.js"></script>
  <script src="../../site_libs/revealjs/plugin/reveal-chalkboard/plugin.js"></script>
  <script src="../../site_libs/revealjs/plugin/multiplex/socket.io.js"></script>
  <script src="../../site_libs/revealjs/plugin/multiplex/multiplex.js"></script>
  <script src="../../site_libs/revealjs/plugin/quarto-support/support.js"></script>
  

  <script src="../../site_libs/revealjs/plugin/notes/notes.js"></script>
  <script src="../../site_libs/revealjs/plugin/search/search.js"></script>
  <script src="../../site_libs/revealjs/plugin/zoom/zoom.js"></script>
  <script src="../../site_libs/revealjs/plugin/math/math.js"></script>
  <script>window.define = window.backupDefine; window.backupDefine = undefined;</script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
'controlsAuto': true,
'previewLinksAuto': false,
'pdfSeparateFragments': false,
'autoAnimateEasing': "ease",
'autoAnimateDuration': 1,
'autoAnimateUnmatched': true,
'jumpToSlide': true,
'menu': {"side":"left","useTextContentForMissingTitles":true,"markers":false,"loadIcons":false,"custom":[{"title":"Tools","icon":"<i class=\"fas fa-gear\"></i>","content":"<ul class=\"slide-menu-items\">\n<li class=\"slide-tool-item active\" data-item=\"0\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.fullscreen(event)\"><kbd>f</kbd> Fullscreen</a></li>\n<li class=\"slide-tool-item\" data-item=\"1\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.speakerMode(event)\"><kbd>s</kbd> Speaker View</a></li>\n<li class=\"slide-tool-item\" data-item=\"2\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>o</kbd> Slide Overview</a></li>\n<li class=\"slide-tool-item\" data-item=\"3\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.togglePdfExport(event)\"><kbd>e</kbd> PDF Export Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"4\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleScrollView(event)\"><kbd>r</kbd> Scroll View Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"5\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleChalkboard(event)\"><kbd>b</kbd> Toggle Chalkboard</a></li>\n<li class=\"slide-tool-item\" data-item=\"6\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleNotesCanvas(event)\"><kbd>c</kbd> Toggle Notes Canvas</a></li>\n<li class=\"slide-tool-item\" data-item=\"7\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.downloadDrawings(event)\"><kbd>d</kbd> Download Drawings</a></li>\n<li class=\"slide-tool-item\" data-item=\"8\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.keyboardHelp(event)\"><kbd>?</kbd> Keyboard Help</a></li>\n</ul>"}],"openButton":true},
'chalkboard': {"buttons":true,"boardmarkerWidth":5},
'multiplex': {"secret":"17330460484484288677","id":"3ae2ceb8171b2c3e","url":"https://reveal-multiplex.glitch.me/"},
'smaller': false,
 
        // Display controls in the bottom right corner
        controls: false,

        // Help the user learn the controls by providing hints, for example by
        // bouncing the down arrow when they first encounter a vertical slide
        controlsTutorial: false,

        // Determines where controls appear, "edges" or "bottom-right"
        controlsLayout: 'edges',

        // Visibility rule for backwards navigation arrows; "faded", "hidden"
        // or "visible"
        controlsBackArrows: 'faded',

        // Display a presentation progress bar
        progress: true,

        // Display the page number of the current slide
        slideNumber: 'c/t',

        // 'all', 'print', or 'speaker'
        showSlideNumber: 'all',

        // Add the current slide number to the URL hash so that reloading the
        // page/copying the URL will return you to the same slide
        hash: true,

        // Start with 1 for the hash rather than 0
        hashOneBasedIndex: false,

        // Flags if we should monitor the hash and change slides accordingly
        respondToHashChanges: true,

        // Push each slide change to the browser history
        history: true,

        // Enable keyboard shortcuts for navigation
        keyboard: true,

        // Enable the slide overview mode
        overview: true,

        // Disables the default reveal.js slide layout (scaling and centering)
        // so that you can use custom CSS layout
        disableLayout: false,

        // Vertical centering of slides
        center: false,

        // Enables touch navigation on devices with touch input
        touch: true,

        // Loop the presentation
        loop: false,

        // Change the presentation direction to be RTL
        rtl: false,

        // see https://revealjs.com/vertical-slides/#navigation-mode
        navigationMode: 'linear',

        // Randomizes the order of slides each time the presentation loads
        shuffle: false,

        // Turns fragments on and off globally
        fragments: true,

        // Flags whether to include the current fragment in the URL,
        // so that reloading brings you to the same fragment position
        fragmentInURL: false,

        // Flags if the presentation is running in an embedded mode,
        // i.e. contained within a limited portion of the screen
        embedded: false,

        // Flags if we should show a help overlay when the questionmark
        // key is pressed
        help: true,

        // Flags if it should be possible to pause the presentation (blackout)
        pause: true,

        // Flags if speaker notes should be visible to all viewers
        showNotes: false,

        // Global override for autoplaying embedded media (null/true/false)
        autoPlayMedia: null,

        // Global override for preloading lazy-loaded iframes (null/true/false)
        preloadIframes: null,

        // Number of milliseconds between automatically proceeding to the
        // next slide, disabled when set to 0, this value can be overwritten
        // by using a data-autoslide attribute on your slides
        autoSlide: 0,

        // Stop auto-sliding after user input
        autoSlideStoppable: true,

        // Use this method for navigation when auto-sliding
        autoSlideMethod: null,

        // Specify the average time in seconds that you think you will spend
        // presenting each slide. This is used to show a pacing timer in the
        // speaker view
        defaultTiming: null,

        // Enable slide navigation via mouse wheel
        mouseWheel: false,

        // The display mode that will be used to show slides
        display: 'block',

        // Hide cursor if inactive
        hideInactiveCursor: true,

        // Time before the cursor is hidden (in ms)
        hideCursorTime: 5000,

        // Opens links in an iframe preview overlay
        previewLinks: true,

        // Transition style (none/fade/slide/convex/concave/zoom)
        transition: 'fade',

        // Transition speed (default/fast/slow)
        transitionSpeed: 'default',

        // Transition style for full page slide backgrounds
        // (none/fade/slide/convex/concave/zoom)
        backgroundTransition: 'none',

        // Number of slides away from the current that are visible
        viewDistance: 3,

        // Number of slides away from the current that are visible on mobile
        // devices. It is advisable to set this to a lower number than
        // viewDistance in order to save resources.
        mobileViewDistance: 2,

        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1600,

        height: 900,

        // Factor of the display size that should remain empty around the content
        margin: 0.1,

        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [QuartoLineHighlight, PdfExport, RevealMenu, RevealChalkboard, QuartoSupport,

          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    
    <script>
      // htmlwidgets need to know to resize themselves when slides are shown/hidden.
      // Fire the "slideenter" event (handled by htmlwidgets.js) when the current
      // slide changes (different for each slide format).
      (function () {
        // dispatch for htmlwidgets
        function fireSlideEnter() {
          const event = window.document.createEvent("Event");
          event.initEvent("slideenter", true, true);
          window.document.dispatchEvent(event);
        }

        function fireSlideChanged(previousSlide, currentSlide) {
          fireSlideEnter();

          // dispatch for shiny
          if (window.jQuery) {
            if (previousSlide) {
              window.jQuery(previousSlide).trigger("hidden");
            }
            if (currentSlide) {
              window.jQuery(currentSlide).trigger("shown");
            }
          }
        }

        // hookup for slidy
        if (window.w3c_slidy) {
          window.w3c_slidy.add_observer(function (slide_num) {
            // slide_num starts at position 1
            fireSlideChanged(null, w3c_slidy.slides[slide_num - 1]);
          });
        }

      })();
    </script>

    <script id="quarto-html-after-body" type="application/javascript">
    window.document.addEventListener("DOMContentLoaded", function (event) {
      const toggleBodyColorMode = (bsSheetEl) => {
        const mode = bsSheetEl.getAttribute("data-mode");
        const bodyEl = window.document.querySelector("body");
        if (mode === "dark") {
          bodyEl.classList.add("quarto-dark");
          bodyEl.classList.remove("quarto-light");
        } else {
          bodyEl.classList.add("quarto-light");
          bodyEl.classList.remove("quarto-dark");
        }
      }
      const toggleBodyColorPrimary = () => {
        const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
        if (bsSheetEl) {
          toggleBodyColorMode(bsSheetEl);
        }
      }
      toggleBodyColorPrimary();  
      const tabsets =  window.document.querySelectorAll(".panel-tabset-tabby")
      tabsets.forEach(function(tabset) {
        const tabby = new Tabby('#' + tabset.id);
      });
      const isCodeAnnotation = (el) => {
        for (const clz of el.classList) {
          if (clz.startsWith('code-annotation-')) {                     
            return true;
          }
        }
        return false;
      }
      const onCopySuccess = function(e) {
        // button target
        const button = e.trigger;
        // don't keep focus
        button.blur();
        // flash "checked"
        button.classList.add('code-copy-button-checked');
        var currentTitle = button.getAttribute("title");
        button.setAttribute("title", "Copied!");
        let tooltip;
        if (window.bootstrap) {
          button.setAttribute("data-bs-toggle", "tooltip");
          button.setAttribute("data-bs-placement", "left");
          button.setAttribute("data-bs-title", "Copied!");
          tooltip = new bootstrap.Tooltip(button, 
            { trigger: "manual", 
              customClass: "code-copy-button-tooltip",
              offset: [0, -8]});
          tooltip.show();    
        }
        setTimeout(function() {
          if (tooltip) {
            tooltip.hide();
            button.removeAttribute("data-bs-title");
            button.removeAttribute("data-bs-toggle");
            button.removeAttribute("data-bs-placement");
          }
          button.setAttribute("title", currentTitle);
          button.classList.remove('code-copy-button-checked');
        }, 1000);
        // clear code selection
        e.clearSelection();
      }
      const getTextToCopy = function(trigger) {
          const codeEl = trigger.previousElementSibling.cloneNode(true);
          for (const childEl of codeEl.children) {
            if (isCodeAnnotation(childEl)) {
              childEl.remove();
            }
          }
          return codeEl.innerText;
      }
      const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
        text: getTextToCopy
      });
      clipboard.on('success', onCopySuccess);
      if (window.document.getElementById('quarto-embedded-source-code-modal')) {
        const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
          text: getTextToCopy,
          container: window.document.getElementById('quarto-embedded-source-code-modal')
        });
        clipboardModal.on('success', onCopySuccess);
      }
        var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
        var mailtoRegex = new RegExp(/^mailto:/);
          var filterRegex = new RegExp("https:\/\/giulianocolosimo\.github\.com");
        var isInternal = (href) => {
            return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
        }
        // Inspect non-navigation links and adorn them if external
     	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
        for (var i=0; i<links.length; i++) {
          const link = links[i];
          if (!isInternal(link.href)) {
            // undo the damage that might have been done by quarto-nav.js in the case of
            // links that we want to consider external
            if (link.dataset.originalHref !== undefined) {
              link.href = link.dataset.originalHref;
            }
          }
        }
      function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
        const config = {
          allowHTML: true,
          maxWidth: 500,
          delay: 100,
          arrow: false,
          appendTo: function(el) {
              return el.closest('section.slide') || el.parentElement;
          },
          interactive: true,
          interactiveBorder: 10,
          theme: 'light-border',
          placement: 'bottom-start',
        };
        if (contentFn) {
          config.content = contentFn;
        }
        if (onTriggerFn) {
          config.onTrigger = onTriggerFn;
        }
        if (onUntriggerFn) {
          config.onUntrigger = onUntriggerFn;
        }
          config['offset'] = [0,0];
          config['maxWidth'] = 700;
        window.tippy(el, config); 
      }
      const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
      for (var i=0; i<noterefs.length; i++) {
        const ref = noterefs[i];
        tippyHover(ref, function() {
          // use id or data attribute instead here
          let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
          try { href = new URL(href).hash; } catch {}
          const id = href.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note) {
            return note.innerHTML;
          } else {
            return "";
          }
        });
      }
      const findCites = (el) => {
        const parentEl = el.parentElement;
        if (parentEl) {
          const cites = parentEl.dataset.cites;
          if (cites) {
            return {
              el,
              cites: cites.split(' ')
            };
          } else {
            return findCites(el.parentElement)
          }
        } else {
          return undefined;
        }
      };
      var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
      for (var i=0; i<bibliorefs.length; i++) {
        const ref = bibliorefs[i];
        const citeInfo = findCites(ref);
        if (citeInfo) {
          tippyHover(citeInfo.el, function() {
            var popup = window.document.createElement('div');
            citeInfo.cites.forEach(function(cite) {
              var citeDiv = window.document.createElement('div');
              citeDiv.classList.add('hanging-indent');
              citeDiv.classList.add('csl-entry');
              var biblioDiv = window.document.getElementById('ref-' + cite);
              if (biblioDiv) {
                citeDiv.innerHTML = biblioDiv.innerHTML;
              }
              popup.appendChild(citeDiv);
            });
            return popup.innerHTML;
          });
        }
      }
    });
    </script>
    

</body></html>