<!DOCTYPE html>
<html lang="en"><head>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-html/tabby.min.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/light-border.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting-e26003cea8cd680ca0c55a263523d882.css" rel="stylesheet" id="quarto-text-highlighting-styles"><meta charset="utf-8">
  <meta name="generator" content="quarto-1.6.39">

  <meta name="dcterms.date" content="2025-01-20">
  <title>Biologia della Conservazione 24/25 – Biologia della conservazione</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="../../site_libs/revealjs/dist/reset.css">
  <link rel="stylesheet" href="../../site_libs/revealjs/dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
      vertical-align: middle;
    }
    /* CSS for syntax highlighting */
    pre > code.sourceCode { white-space: pre; position: relative; }
    pre > code.sourceCode > span { line-height: 1.25; }
    pre > code.sourceCode > span:empty { height: 1.2em; }
    .sourceCode { overflow: visible; }
    code.sourceCode > span { color: inherit; text-decoration: inherit; }
    div.sourceCode { margin: 1em 0; }
    pre.sourceCode { margin: 0; }
    @media screen {
    div.sourceCode { overflow: auto; }
    }
    @media print {
    pre > code.sourceCode { white-space: pre-wrap; }
    pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
    }
    pre.numberSource code
      { counter-reset: source-line 0; }
    pre.numberSource code > span
      { position: relative; left: -4em; counter-increment: source-line; }
    pre.numberSource code > span > a:first-child::before
      { content: counter(source-line);
        position: relative; left: -1em; text-align: right; vertical-align: baseline;
        border: none; display: inline-block;
        -webkit-touch-callout: none; -webkit-user-select: none;
        -khtml-user-select: none; -moz-user-select: none;
        -ms-user-select: none; user-select: none;
        padding: 0 4px; width: 4em;
        color: #aaaaaa;
      }
    pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
    div.sourceCode
      { color: #003b4f; background-color: #f1f3f5; }
    @media screen {
    pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
    }
    code span { color: #003b4f; } /* Normal */
    code span.al { color: #ad0000; } /* Alert */
    code span.an { color: #5e5e5e; } /* Annotation */
    code span.at { color: #657422; } /* Attribute */
    code span.bn { color: #ad0000; } /* BaseN */
    code span.bu { } /* BuiltIn */
    code span.cf { color: #003b4f; font-weight: bold; } /* ControlFlow */
    code span.ch { color: #20794d; } /* Char */
    code span.cn { color: #8f5902; } /* Constant */
    code span.co { color: #5e5e5e; } /* Comment */
    code span.cv { color: #5e5e5e; font-style: italic; } /* CommentVar */
    code span.do { color: #5e5e5e; font-style: italic; } /* Documentation */
    code span.dt { color: #ad0000; } /* DataType */
    code span.dv { color: #ad0000; } /* DecVal */
    code span.er { color: #ad0000; } /* Error */
    code span.ex { } /* Extension */
    code span.fl { color: #ad0000; } /* Float */
    code span.fu { color: #4758ab; } /* Function */
    code span.im { color: #00769e; } /* Import */
    code span.in { color: #5e5e5e; } /* Information */
    code span.kw { color: #003b4f; font-weight: bold; } /* Keyword */
    code span.op { color: #5e5e5e; } /* Operator */
    code span.ot { color: #003b4f; } /* Other */
    code span.pp { color: #ad0000; } /* Preprocessor */
    code span.sc { color: #5e5e5e; } /* SpecialChar */
    code span.ss { color: #20794d; } /* SpecialString */
    code span.st { color: #20794d; } /* String */
    code span.va { color: #111111; } /* Variable */
    code span.vs { color: #20794d; } /* VerbatimString */
    code span.wa { color: #5e5e5e; font-style: italic; } /* Warning */
    /* CSS for citations */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
      margin-bottom: 0em;
    }
    .hanging-indent div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }  </style>
  <link rel="stylesheet" href="../../site_libs/revealjs/dist/theme/quarto-fbff88913fe25870c86f77c0ed158baa.css">
  <link href="../../site_libs/revealjs/plugin/quarto-line-highlight/line-highlight.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-menu/menu.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-menu/quarto-menu.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-chalkboard/font-awesome/css/all.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/reveal-chalkboard/style.css" rel="stylesheet">
  <link href="../../site_libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">
  <style type="text/css">
    .reveal div.sourceCode {
      margin: 0;
      overflow: auto;
    }
    .reveal div.hanging-indent {
      margin-left: 1em;
      text-indent: -1em;
    }
    .reveal .slide:not(.center) {
      height: 100%;
    }
    .reveal .slide.scrollable {
      overflow-y: auto;
    }
    .reveal .footnotes {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide .absolute {
      position: absolute;
      display: block;
    }
    .reveal .footnotes ol {
      counter-reset: ol;
      list-style-type: none; 
      margin-left: 0;
    }
    .reveal .footnotes ol li:before {
      counter-increment: ol;
      content: counter(ol) ". "; 
    }
    .reveal .footnotes ol li > p:first-child {
      display: inline-block;
    }
    .reveal .slide ul,
    .reveal .slide ol {
      margin-bottom: 0.5em;
    }
    .reveal .slide ul li,
    .reveal .slide ol li {
      margin-top: 0.4em;
      margin-bottom: 0.2em;
    }
    .reveal .slide ul[role="tablist"] li {
      margin-bottom: 0;
    }
    .reveal .slide ul li > *:first-child,
    .reveal .slide ol li > *:first-child {
      margin-block-start: 0;
    }
    .reveal .slide ul li > *:last-child,
    .reveal .slide ol li > *:last-child {
      margin-block-end: 0;
    }
    .reveal .slide .columns:nth-child(3) {
      margin-block-start: 0.8em;
    }
    .reveal blockquote {
      box-shadow: none;
    }
    .reveal .tippy-content>* {
      margin-top: 0.2em;
      margin-bottom: 0.7em;
    }
    .reveal .tippy-content>*:last-child {
      margin-bottom: 0.2em;
    }
    .reveal .slide > img.stretch.quarto-figure-center,
    .reveal .slide > img.r-stretch.quarto-figure-center {
      display: block;
      margin-left: auto;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-left,
    .reveal .slide > img.r-stretch.quarto-figure-left  {
      display: block;
      margin-left: 0;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-right,
    .reveal .slide > img.r-stretch.quarto-figure-right  {
      display: block;
      margin-left: auto;
      margin-right: 0; 
    }
  </style>
<meta property="og:title" content="Biologia della conservazione – Biologia della Conservazione 24/25">
<meta property="og:description" content="Class 15 - Exercise in R">
<meta property="og:site_name" content="Biologia della Conservazione 24/25">
<meta property="og:locale" content="en_US">
</head>
<body class="quarto-light">
  <div class="reveal">
    <div class="slides">

<section id="title-slide" class="quarto-title-block center">
  <h1 class="title">Biologia della conservazione</h1>
  <p class="subtitle">Class 15 - Exercise in R</p>

<div class="quarto-title-authors">
</div>

  <p class="date">2025-01-20</p>
</section>
<section id="introduction" class="slide level2">
<h2>Introduction</h2>
<ul>
<li><p>In this exercise we will first briefly look at the very basic of R and RStudio. We will then try to use some functions from various R packages to compute some of the genetic indexes that we have discussed in the first part of the course</p></li>
<li><p>We will be using a dataset published by <span class="citation" data-cites="Groombridge2000">Groombridge, Jones, Bruford, &amp; Nichols (<a href="#/references" role="doc-biblioref" onclick="">2000</a>)</span></p></li>
<li><p>The dataset contains 153 individuals of Kestrel, a small bird of prey, typed at 12 microsatellite loci</p></li>
<li><p>The dataset is available <a href="https://drive.proton.me/urls/SHD6SKCV30#RfJOAMMTGpkQ">here</a> as an excel file. Download it and place in a directory of your convenience</p></li>
<li><p>Before moving forward with the excercise I will review some basic <strong>R</strong> stuff, including the installation process and how to retrieve some of the most important packages</p></li>
</ul>
</section>
<section id="setting-up-your-computers" class="slide level2">
<h2>Setting up your computers</h2>
<ul>
<li><p>Connect to Wi-Fi</p></li>
<li><p>Visit the <strong>R</strong> <a href="https://cran.r-project.org" class="uri">https://cran.r-project.org</a> page and download the <strong>R</strong> version compatible with your OS. Install the software following the instructions</p></li>
<li><p>Visit the <strong>RStudio</strong> <a href="https://www.rstudio.com/products/rstudio/download/" class="uri">https://www.rstudio.com/products/rstudio/download/</a> page and download the <strong>RStudio</strong> Desktop version compatible with your OS. Install the software following the instructions. Most recently <strong>RStudio</strong> has changed its name in to <strong>Posit</strong> <a href="https://posit.co" class="uri">https://posit.co</a>, but the core idea is the same!</p></li>
</ul>
<div class="callout callout-important callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>Important</strong></p>
</div>
<div class="callout-content">
<p><strong>RStudio</strong> is an Integrated Development Environment (IDE) within which it should be easier to organize your <strong>R</strong> workflow. <u>Learning all the specifics of <strong>R</strong> and <strong>RStudio</strong> is beyond the scope of this course and it is a student’s responsibility to learn the basics of these tools. That being said, you can always write me with your questions if something does not work on your computer and I will help troubleshooting potential issues.</u></p>
</div>
</div>
</div>
</section>
<section id="brief-intro-to-r-and-rstudio" class="slide level2">
<h2>Brief intro to R and RStudio</h2>
<ul>
<li><p>R is a very powerful programing language. It was conceived for statistical computing, but now it has become a well rounded tool that can be used to do a number of different things, including writing manuscripts and designing websites</p></li>
<li><p>At its core R can still be considered <em>number cruncher</em></p></li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a></a><span class="co"># basic arithmetic</span></span>
<span id="cb1-2"><a></a><span class="dv">3</span> <span class="sc">+</span> <span class="dv">5</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 8</code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a></a><span class="co"># assigning values to object</span></span>
<span id="cb3-2"><a></a>a <span class="ot">&lt;-</span> <span class="dv">5</span></span>
<span id="cb3-3"><a></a>b <span class="ot">&lt;-</span> <span class="dv">3</span></span>
<span id="cb3-4"><a></a>(c <span class="ot">&lt;-</span> a <span class="sc">+</span> b)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 8</code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb5-1"><a></a><span class="co"># multiple type of data</span></span>
<span id="cb5-2"><a></a><span class="fu">str</span>(x <span class="ot">&lt;-</span> a)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code> num 5</code></pre>
</div>
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a></a><span class="fu">str</span>(y <span class="ot">&lt;-</span> <span class="st">"a"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code> chr "a"</code></pre>
</div>
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb9-1"><a></a><span class="fu">str</span>(z <span class="ot">&lt;-</span> <span class="fu">c</span>(T, F))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code> logi [1:2] TRUE FALSE</code></pre>
</div>
</div>
</section>
<section class="slide level2">

<ul>
<li>Data can be organized in a number of different ways each one with its specific structure</li>
<li>The three most simple “objects” that contain data are vecotrs, matrices, and data frames</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb11"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb11-1"><a></a><span class="co"># a matrix</span></span>
<span id="cb11-2"><a></a>(mtrx_ex <span class="ot">&lt;-</span> <span class="fu">matrix</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">9</span>, <span class="at">nrow =</span> <span class="dv">3</span>, <span class="at">ncol =</span> <span class="dv">3</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>     [,1] [,2] [,3]
[1,]    1    4    7
[2,]    2    5    8
[3,]    3    6    9</code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb13"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb13-1"><a></a><span class="co"># a data frame</span></span>
<span id="cb13-2"><a></a>(df_ex <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(mtrx_ex))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>  V1 V2 V3
1  1  4  7
2  2  5  8
3  3  6  9</code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb15"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb15-1"><a></a><span class="co"># another data frame</span></span>
<span id="cb15-2"><a></a>(df_ex <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="at">id =</span> <span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">2</span>, <span class="dv">3</span>),</span>
<span id="cb15-3"><a></a>                     <span class="at">species =</span> <span class="fu">c</span>(<span class="st">"cat"</span>, <span class="st">"dog"</span>, <span class="st">"lion"</span>),</span>
<span id="cb15-4"><a></a>                     <span class="at">is.domestic =</span> <span class="fu">c</span>(T, T, F)))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>  id species is.domestic
1  1     cat        TRUE
2  2     dog        TRUE
3  3    lion       FALSE</code></pre>
</div>
</div>
</section>
<section class="slide level2">

<ul>
<li><p>When you first install <strong>R</strong> on your computer you will find only a few basic packages that ship with basic installation</p></li>
<li><p>If you type the following command at your prompt you will get a series of information regarding the version of R and the hardware on which is running</p></li>
<li><p>The command will also output information relative to the name of packages installed and loaded in your working session</p></li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb17-1"><a></a><span class="fu">sessionInfo</span>() <span class="co"># the output is from my computer and mybe different for you</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>R version 4.3.2 (2023-10-31)
Platform: aarch64-apple-darwin20 (64-bit)
Running under: macOS 15.2

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.11.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: Europe/Rome
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] readxl_1.4.3       plotly_4.10.4      cranlogs_2.1.1     learnPopGen_1.0.4 
 [5] RColorBrewer_1.1-3 lubridate_1.9.3    forcats_1.0.0      stringr_1.5.1     
 [9] dplyr_1.1.4        purrr_1.0.2        readr_2.1.5        tidyr_1.3.1       
[13] tibble_3.2.1       ggplot2_3.5.1      tidyverse_2.0.0    DT_0.33           

loaded via a namespace (and not attached):
 [1] fastmatch_1.1-4         gtable_0.3.5            xfun_0.43              
 [4] htmlwidgets_1.6.4       lattice_0.22-6          tzdb_0.4.0             
 [7] numDeriv_2016.8-1.1     quadprog_1.5-8          vctrs_0.6.5            
[10] tools_4.3.2             generics_0.1.3          parallel_4.3.2         
[13] fansi_1.0.6             pkgconfig_2.0.3         Matrix_1.6-5           
[16] data.table_1.15.4       scatterplot3d_0.3-44    lifecycle_1.0.4        
[19] compiler_4.3.2          munsell_0.5.1           mnormt_2.1.1           
[22] combinat_0.0-8          codetools_0.2-20        htmltools_0.5.8.1      
[25] maps_3.4.2              lazyeval_0.2.2          yaml_2.3.8             
[28] pillar_1.9.0            MASS_7.3-60.0.1         clusterGeneration_1.3.8
[31] iterators_1.0.14        foreach_1.5.2           nlme_3.1-164           
[34] phangorn_2.11.1         gtools_3.9.5            tidyselect_1.2.1       
[37] digest_0.6.35           stringi_1.8.4           fastmap_1.1.1          
[40] grid_4.3.2              colorspace_2.1-0        expm_0.999-9           
[43] cli_3.6.2               magrittr_2.0.3          optimParallel_1.0-2    
[46] utf8_1.2.4              ape_5.8                 withr_3.0.0            
[49] scales_1.3.0            timechange_0.3.0        httr_1.4.7             
[52] rmarkdown_2.26          igraph_2.0.3            cellranger_1.1.0       
[55] hms_1.1.3               coda_0.19-4.1           evaluate_0.23          
[58] knitr_1.45              phytools_2.1-1          doParallel_1.0.17      
[61] viridisLite_0.4.2       rlang_1.1.3             Rcpp_1.0.12            
[64] glue_1.7.0              rstudioapi_0.16.0       jsonlite_1.8.8         
[67] R6_2.5.1               </code></pre>
</div>
</div>
</section>
<section id="packages" class="slide level2">
<h2>Packages</h2>
<ul>
<li>There are thousands of available libraries (aka, <em>packages</em>) that can be used to perform very specific analyses and produce publication-grade graphical output.</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb19"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb19-1"><a></a>sI <span class="ot">&lt;-</span> <span class="fu">sessionInfo</span>()</span>
<span id="cb19-2"><a></a>sI<span class="sc">$</span>basePkgs <span class="co"># look at the basic packages installed and loaded</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] "stats"     "graphics"  "grDevices" "utils"     "datasets"  "methods"  
[7] "base"     </code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb21"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb21-1"><a></a><span class="co"># The following command provides a list of all packages that are installed</span></span>
<span id="cb21-2"><a></a><span class="fu">rownames</span>(<span class="fu">installed.packages</span>()) <span class="co"># the output is from my computer and mybe different for you</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>  [1] "abc"                   "abc.data"              "abind"                
  [4] "ade4"                  "adegenet"              "adehabitatHR"         
  [7] "adehabitatHS"          "adehabitatLT"          "adehabitatMA"         
 [10] "ADGofTest"             "afex"                  "AHMbook"              
 [13] "AICcmodavg"            "alphahull"             "amap"                 
 [16] "AmesHousing"           "amt"                   "animation"            
 [19] "AnnotationDbi"         "AnnotationFilter"      "anytime"              
 [22] "ape"                   "apex"                  "aphid"                
 [25] "aplot"                 "aqp"                   "ash"                  
 [28] "askpass"               "assertthat"            "automap"              
 [31] "av"                    "aws.s3"                "aws.signature"        
 [34] "backports"             "base"                  "base64enc"            
 [37] "BayesFactor"           "bayestestR"            "BBmisc"               
 [40] "bbmle"                 "bcp"                   "bcpa"                 
 [43] "bdsmatrix"             "beeswarm"              "benchmarkme"          
 [46] "benchmarkmeData"       "Bessel"                "BH"                   
 [49] "bib2df"                "bibtex"                "bigD"                 
 [52] "binom"                 "Biobase"               "BiocFileCache"        
 [55] "BiocGenerics"          "BiocIO"                "BiocManager"          
 [58] "BiocParallel"          "BiocStyle"             "BiocVersion"          
 [61] "biomaRt"               "biomformat"            "bios2mds"             
 [64] "Biostrings"            "biovizBase"            "bit"                  
 [67] "bit64"                 "bitops"                "blob"                 
 [70] "blogdown"              "bmp"                   "bold"                 
 [73] "bookdown"              "boot"                  "bootstrap"            
 [76] "brew"                  "brio"                  "broom"                
 [79] "broom.helpers"         "brotli"                "BSgenome"             
 [82] "bslib"                 "ca"                    "cachem"               
 [85] "calendR"               "calibrate"             "callr"                
 [88] "car"                   "carData"               "caret"                
 [91] "caTools"               "cellranger"            "CENFA"                
 [94] "changepoint"           "checkmate"             "chron"                
 [97] "circlize"              "CircStats"             "circular"             
[100] "class"                 "classInt"              "cli"                  
[103] "clipr"                 "clock"                 "clue"                 
[106] "cluster"               "clusterGeneration"     "ClusterR"             
[109] "clv"                   "coalescentMCMC"        "coda"                 
[112] "codetools"             "colordistance"         "colorRamps"           
[115] "colorspace"            "combinat"              "commonmark"           
[118] "compiler"              "conditionz"            "config"               
[121] "conflicted"            "contfrac"              "copula"               
[124] "corpcor"               "correlation"           "corrplot"             
[127] "countcolors"           "countdown"             "cowplot"              
[130] "cpp11"                 "cranlogs"              "crayon"               
[133] "credentials"           "crosstalk"             "crul"                 
[136] "ctmm"                  "curl"                  "dashCoreComponents"   
[139] "data.table"            "datasets"              "datawizard"           
[142] "DBI"                   "dbplyr"                "dbscan"               
[145] "DDRTree"               "decido"                "DECIPHER"             
[148] "DelayedArray"          "deldir"                "dendextend"           
[151] "denstrip"              "DEoptimR"              "desc"                 
[154] "deSolve"               "devtools"              "diagram"              
[157] "DiagrammeR"            "dichromat"             "diffobj"              
[160] "diffusionMap"          "digest"                "diptest"              
[163] "dismo"                 "distributional"        "diyabcGUI"            
[166] "docopt"                "doMC"                  "doParallel"           
[169] "doSNOW"                "dotCall64"             "downlit"              
[172] "downloader"            "dplyr"                 "DT"                   
[175] "dtplyr"                "dtw"                   "e1071"                
[178] "ecodist"               "effects"               "effectsize"           
[181] "elevatr"               "ellipse"               "ellipsis"             
[184] "elliptic"              "emdbook"               "emdist"               
[187] "emmeans"               "ensembldb"             "EnvStats"             
[190] "estimability"          "evaluate"              "evd"                  
[193] "exactextractr"         "exactRankTests"        "excelR"               
[196] "expm"                  "extrafont"             "extrafontdb"          
[199] "factoextra"            "FactoMineR"            "fansi"                
[202] "farver"                "fastICA"               "fastmap"              
[205] "fastmatch"             "fastmatrix"            "fastqcr"              
[208] "fasttime"              "fBasics"               "fdrtool"              
[211] "fftwtools"             "fields"                "fiery"                
[214] "filehash"              "filelock"              "fitdistrplus"         
[217] "flashClust"            "flexmix"               "flextable"            
[220] "fmesher"               "FNN"                   "fontawesome"          
[223] "fontBitstreamVera"     "fontLiberation"        "fontquiver"           
[226] "forcats"               "foreach"               "forecast"             
[229] "foreign"               "forestError"           "formatR"              
[232] "formattable"           "Formula"               "fpc"                  
[235] "fpeek"                 "fracdiff"              "fs"                   
[238] "furrr"                 "futile.logger"         "futile.options"       
[241] "future"                "future.apply"          "gap"                  
[244] "gap.datasets"          "gargle"                "gaston"               
[247] "gclus"                 "gdata"                 "gdistance"            
[250] "gdsfmt"                "gdtools"               "genepop"              
[253] "generics"              "genetics"              "GenomeInfoDb"         
[256] "GenomeInfoDbData"      "GenomicAlignments"     "GenomicFeatures"      
[259] "GenomicFiles"          "GenomicRanges"         "geojsonsf"            
[262] "geometries"            "geometry"              "geomorph"             
[265] "geoR"                  "geosphere"             "gert"                 
[268] "gfonts"                "GGally"                "gganimate"            
[271] "ggbeeswarm"            "ggdendro"              "ggeffects"            
[274] "ggforce"               "ggfun"                 "gggibbous"            
[277] "ggimage"               "ggmap"                 "ggnetwork"            
[280] "ggnewscale"            "ggplot2"               "ggplotify"            
[283] "ggpubr"                "ggraph"                "ggrepel"              
[286] "ggridges"              "ggsci"                 "ggsignif"             
[289] "ggsn"                  "ggspatial"             "ggstats"              
[292] "ggtext"                "ggthemes"              "ggtree"               
[295] "gh"                    "gifski"                "gitcreds"             
[298] "glasso"                "glmnet"                "GlobalOptions"        
[301] "globals"               "glue"                  "Gmedian"              
[304] "gmp"                   "goftest"               "googledrive"          
[307] "googlePolylines"       "googlesheets4"         "googleway"            
[310] "GOplot"                "gower"                 "gplots"               
[313] "graph4lg"              "graphics"              "graphlayouts"         
[316] "grDevices"             "grid"                  "gridBase"             
[319] "gridExtra"             "gridGraphics"          "gridSVG"              
[322] "gridtext"              "grImport2"             "gsl"                  
[325] "gss"                   "gstat"                 "gt"                   
[328] "gtable"                "gtExtras"              "gtools"               
[331] "gtsummary"             "haplotypes"            "hardhat"              
[334] "HardyWeinberg"         "haven"                 "HDInterval"           
[337] "hdrcde"                "here"                  "hexbin"               
[340] "hierfstat"             "highcharter"           "highr"                
[343] "HistData"              "Hmisc"                 "hms"                  
[346] "howmany"               "htmlTable"             "htmltools"            
[349] "htmlwidgets"           "httpcode"              "httpuv"               
[352] "httr"                  "httr2"                 "humaniformat"         
[355] "hwriter"               "hydroTSM"              "hypergeo"             
[358] "IDPmisc"               "ids"                   "igraph"               
[361] "igraphdata"            "imager"                "imputeTS"             
[364] "inbreedR"              "influenceR"            "ini"                  
[367] "INLA"                  "inline"                "insect"               
[370] "insight"               "intergraph"            "interp"               
[373] "intervals"             "ipred"                 "ips"                  
[376] "IRanges"               "irlba"                 "irr"                  
[379] "isoband"               "iterators"             "jackalope"            
[382] "jagsUI"                "jomo"                  "jpeg"                 
[385] "jqr"                   "jquerylib"             "jsonify"              
[388] "jsonlite"              "juicyjuice"            "kableExtra"           
[391] "KEGGREST"              "keras"                 "kernlab"              
[394] "KernSmooth"            "keyring"               "kknn"                 
[397] "km.ci"                 "kmer"                  "KMsurv"               
[400] "knitr"                 "ks"                    "labeling"             
[403] "labelled"              "lambda.r"              "later"                
[406] "lattice"               "latticeExtra"          "lava"                 
[409] "lavaan"                "lazyeval"              "leafem"               
[412] "leaflet"               "leaflet.providers"     "leafpop"              
[415] "leafsync"              "leaps"                 "learnPopGen"          
[418] "learnr"                "leastcostpath"         "leidenbase"           
[421] "lifecycle"             "linprog"               "listenv"              
[424] "lme4"                  "lmerTest"              "lmtest"               
[427] "locfdr"                "locfit"                "locpol"               
[430] "logger"                "longitudinalData"      "loo"                  
[433] "LPCM"                  "lpSolve"               "lubridate"            
[436] "lwgeom"                "magic"                 "magick"               
[439] "magrittr"              "MALDIquant"            "manipulate"           
[442] "mapdata"               "mapplots"              "mapproj"              
[445] "maps"                  "maptools"              "mapview"              
[448] "markdown"              "markovchain"           "MASS"                 
[451] "mathjaxr"              "Matrix"                "matrixcalc"           
[454] "MatrixGenerics"        "MatrixModels"          "matrixStats"          
[457] "maxstat"               "mclust"                "mcmcplots"            
[460] "MCMCvis"               "memoise"               "MEMSS"                
[463] "memuse"                "metadat"               "metafor"              
[466] "metagear"              "methods"               "Metrics"              
[469] "mgcv"                  "mice"                  "microsimr"            
[472] "mime"                  "miniUI"                "minqa"                
[475] "misc3d"                "mitml"                 "mitools"              
[478] "mixtools"              "mlmRev"                "mlr"                  
[481] "mmod"                  "mnormt"                "modeest"              
[484] "modelbased"            "ModelMetrics"          "modelr"               
[487] "modeltools"            "mongolite"             "move"                 
[490] "move2"                 "movecost"              "moveHMM"              
[493] "moveVis"               "msa"                   "msm"                  
[496] "multcomp"              "multcompView"          "multicool"            
[499] "multtest"              "munsell"               "mvtnorm"              
[502] "natserv"               "ncdf4"                 "network"              
[505] "nimble"                "nlme"                  "nloptr"               
[508] "NLP"                   "NMF"                   "nnet"                 
[511] "nortest"               "nothing"               "numDeriv"             
[514] "officer"               "openssl"               "openxlsx"             
[517] "optimParallel"         "optimx"                "Orcs"                 
[520] "ordinal"               "overlapping"           "padr"                 
[523] "paletteer"             "pan"                   "pander"               
[526] "parallel"              "parallelly"            "parallelMap"          
[529] "ParallelStructure"     "parameters"            "ParamHelpers"         
[532] "parsedate"             "patchwork"             "pbapply"              
[535] "pbivnorm"              "pbkrtest"              "pbs"                  
[538] "PBSmapping"            "pcaPP"                 "pdc"                  
[541] "pegas"                 "performance"           "permute"              
[544] "phangorn"              "pheatmap"              "photobiology"         
[547] "phyclust"              "phylobase"             "phylogram"            
[550] "phyloseq"              "phyloseqGraphTest"     "phytools"             
[553] "pillar"                "pinfsc50"              "pixmap"               
[556] "pkgbuild"              "pkgconfig"             "pkgdown"              
[559] "pkgload"               "pkgmaker"              "plogr"                
[562] "plot3D"                "plotKML"               "plotly"               
[565] "plotrix"               "plotROC"               "plyr"                 
[568] "PMA"                   "png"                   "polspline"            
[571] "polyclip"              "polynom"               "polysat"              
[574] "popbio"                "popgen"                "PopGenReport"         
[577] "poppr"                 "posterior"             "prabclus"             
[580] "pracma"                "praise"                "prettymapr"           
[583] "prettyunits"           "prismatic"             "pROC"                 
[586] "processx"              "prodlim"               "profvis"              
[589] "progress"              "progressr"             "promises"             
[592] "ProtGenerics"          "proxy"                 "ps"                   
[595] "pspline"               "psych"                 "purrr"                
[598] "pwr"                   "qap"                   "qgraph"               
[601] "qlcMatrix"             "qpdf"                  "quadprog"             
[604] "quantmod"              "quantreg"              "QuasR"                
[607] "QuickJSR"              "R.methodsS3"           "R.oo"                 
[610] "R.utils"               "R6"                    "ragg"                 
[613] "randomForest"          "ranger"                "RANN"                 
[616] "rapidjsonr"            "rappdirs"              "raster"               
[619] "rasterVis"             "ratelimitr"            "rayimage"             
[622] "rayrender"             "rayshader"             "rayvertex"            
[625] "rbenchmark"            "rbibutils"             "Rbowtie"              
[628] "Rcapture"              "rcmdcheck"             "RColorBrewer"         
[631] "Rcpp"                  "RcppArmadillo"         "RcppEigen"            
[634] "RcppNumerical"         "RcppParallel"          "RcppProgress"         
[637] "RcppThread"            "RcppTOML"              "RCurl"                
[640] "Rdpack"                "reactable"             "reactR"               
[643] "readbitmap"            "readr"                 "readxl"               
[646] "recapr"                "recipes"               "RefManageR"           
[649] "registry"              "rematch"               "rematch2"             
[652] "remotes"               "Renext"                "rentrez"              
[655] "renv"                  "reprex"                "reqres"               
[658] "reshape"               "reshape2"              "restfulr"             
[661] "reticulate"            "rex"                   "rgdal"                
[664] "rgeos"                 "rgl"                   "RgoogleMaps"          
[667] "rhdf5"                 "rhdf5filters"          "Rhdf5lib"             
[670] "Rhh"                   "Rhtslib"               "ritis"                
[673] "rjags"                 "rjson"                 "rlang"                
[676] "RLDNe"                 "rlist"                 "RMark"                
[679] "rmarkdown"             "rmetasim"              "Rmisc"                
[682] "Rmpfr"                 "rms"                   "rmutil"               
[685] "RMySQL"                "rnaturalearth"         "rnaturalearthdata"    
[688] "rncl"                  "RNeXML"                "rngtools"             
[691] "robustbase"            "rosm"                  "rotl"                 
[694] "routr"                 "roxygen2"              "rpart"                
[697] "rpart.plot"            "rphylopic"             "RPostgres"            
[700] "RPostgreSQL"           "rPref"                 "rprojroot"            
[703] "Rqc"                   "rredlist"              "RRPP"                 
[706] "rsample"               "Rsamtools"             "Rsolnp"               
[709] "RSpectra"              "RSQLite"               "rstan"                
[712] "rstatix"               "rstudioapi"            "Rsubread"             
[715] "rsvd"                  "rsvg"                  "rtracklayer"          
[718] "Rtsne"                 "Rttf2pt1"              "RUnit"                
[721] "runjags"               "rversions"             "rvest"                
[724] "rworldmap"             "rworldxtra"            "s2"                   
[727] "S4Arrays"              "S4Vectors"             "sandwich"             
[730] "sangeranalyseR"        "sangerseqR"            "sass"                 
[733] "satellite"             "scales"                "scatterpie"           
[736] "scatterplot3d"         "sdamr"                 "secr"                 
[739] "see"                   "segmented"             "segmenTier"           
[742] "selectr"               "seqinr"                "seriation"            
[745] "servr"                 "sessioninfo"           "sf"                   
[748] "sfheaders"             "sfsmisc"               "sftime"               
[751] "sgeostat"              "shape"                 "shiny"                
[754] "shinyBS"               "shinybusy"             "shinycssloaders"      
[757] "shinydashboard"        "shinyFeedback"         "shinyFiles"           
[760] "shinyhelper"           "shinyjs"               "shinythemes"          
[763] "shinyWidgets"          "ShortRead"             "showtext"             
[766] "showtextdb"            "SIBER"                 "signal"               
[769] "sjlabelled"            "sjmisc"                "sjPlot"               
[772] "sjstats"               "slam"                  "slider"               
[775] "slippymath"            "smatr"                 "sn"                   
[778] "sna"                   "snow"                  "SNPRelate"            
[781] "snpStats"              "sodium"                "softImpute"           
[784] "solrium"               "sourcetools"           "sp"                   
[787] "spacefillr"            "spacetime"             "spam"                 
[790] "SparseM"               "sparsesvd"             "spatial"              
[793] "spatialEco"            "SpatialPack"           "spatstat"             
[796] "spatstat.data"         "spatstat.explore"      "spatstat.geom"        
[799] "spatstat.linnet"       "spatstat.model"        "spatstat.random"      
[802] "spatstat.sparse"       "spatstat.utils"        "spData"               
[805] "spDataLarge"           "spdep"                 "splancs"              
[808] "splines"               "splus2R"               "sprex"                
[811] "SQUAREM"               "stable"                "stabledist"           
[814] "StanHeaders"           "stars"                 "statip"               
[817] "statmod"               "statnet.common"        "stats"                
[820] "stats4"                "stinepack"             "stringdist"           
[823] "stringi"               "stringr"               "SummarizedExperiment" 
[826] "suncalc"               "survey"                "survival"             
[829] "survminer"             "survMisc"              "svglite"              
[832] "swfscMisc"             "sys"                   "sysfonts"             
[835] "systemfonts"           "taxize"                "taxonomizr"           
[838] "tcltk"                 "tensor"                "tensorA"              
[841] "tensorflow"            "terra"                 "terrainmeshr"         
[844] "testthat"              "textshaping"           "tfautograph"          
[847] "tfruns"                "TH.data"               "tibble"               
[850] "tidygraph"             "tidyr"                 "tidyselect"           
[853] "tidytree"              "tidyverse"             "tiff"                 
[856] "timechange"            "timeDate"              "timeSeries"           
[859] "tinytex"               "tkrplot"               "tm"                   
[862] "tmap"                  "tmaptools"             "TMB"                  
[865] "tmvnsim"               "tools"                 "trajectories"         
[868] "trajr"                 "transformr"            "treeio"               
[871] "triebeard"             "tripack"               "truncnorm"            
[874] "TSclust"               "tseries"               "TSP"                  
[877] "TTR"                   "tweenr"                "tzdb"                 
[880] "ucminf"                "units"                 "unmarked"             
[883] "urca"                  "urlchecker"            "urltools"             
[886] "usethis"               "utf8"                  "utils"                
[889] "uuid"                  "V8"                    "VariantAnnotation"    
[892] "vcd"                   "vcfR"                  "vctrs"                
[895] "vegan"                 "VGAM"                  "vipor"                
[898] "viridis"               "viridisLite"           "visNetwork"           
[901] "vroom"                 "wakefield"             "waldo"                
[904] "warp"                  "wavelets"              "webshot"              
[907] "webutils"              "wesanderson"           "whisker"              
[910] "widgetframe"           "WikidataQueryServiceR" "WikidataR"            
[913] "WikipediR"             "wikitaxa"              "WILD3810"             
[916] "withr"                 "wk"                    "wordcloud"            
[919] "wordcloud2"            "worrms"                "wosr"                 
[922] "xaringan"              "xfun"                  "xgboost"              
[925] "xkcd"                  "XML"                   "xml2"                 
[928] "xopen"                 "xtable"                "xts"                  
[931] "XVector"               "yaImpute"              "yaml"                 
[934] "yarrr"                 "yulab.utils"           "zeallot"              
[937] "zip"                   "zlibbioc"              "zoo"                  </code></pre>
</div>
</div>
</section>
<section class="slide level2">

<h4 id="how-to-install-new-packages">How to install new packages</h4>
<ul>
<li>Most packages can be installed using the following procedure</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb23"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb23-1"><a></a><span class="fu">install.packages</span>(<span class="st">"ggplot2"</span>, <span class="at">repos =</span> <span class="st">"https://cloud.r-project.org"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<ul>
<li>Once the package has been installed without any error as output, it should be available and can be loaded as a library in your working session</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb24"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb24-1"><a></a><span class="fu">library</span>(ggplot2)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<ul>
<li>Once the library is loaded it is possible to star using the functions that are contained within the library</li>
<li><strong>R</strong> comes with a great help system that everybody is encouraged to read extensively and consult every time they are experiencing issues when running some code or using a function.</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb25-1"><a></a>?geom_segment</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="loading-the-data-in-to-r" class="slide level2">
<h2>Loading the data in to <strong>R</strong></h2>
<ul>
<li>We can read excel files in <strong>R</strong>, but we first need to install and load a specific library that can help us doing that</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb26"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb26-1"><a></a><span class="fu">install.packages</span>(<span class="st">"readxl"</span>, <span class="at">repos =</span> <span class="st">"https://cloud.r-project.org"</span>)</span>
<span id="cb26-2"><a></a><span class="fu">library</span>(readxl)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<ul>
<li>Now that the library is loaded we can load the excel file</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb27"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb27-1"><a></a>kestrel <span class="ot">&lt;-</span> <span class="fu">read_excel</span>(<span class="st">"/PATH/TO/EXCEL/FILE/Groombridge_et_al_genotypes_data.xlsx"</span>,</span>
<span id="cb27-2"><a></a>           <span class="at">sheet =</span> <span class="dv">1</span>, <span class="at">col_types =</span> <span class="st">"guess"</span>, <span class="at">na =</span> <span class="st">"</span><span class="sc">\\</span><span class="st">N"</span>)</span>
<span id="cb27-3"><a></a>kestrel</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 153 × 27
   ID      Species   Info  MS1_A MS1_B MS2_A MS2_B MS3_A MS3_B MS4_A MS4_B MS5_A
   &lt;chr&gt;   &lt;chr&gt;     &lt;chr&gt; &lt;dbl&gt; &lt;dbl&gt; &lt;dbl&gt; &lt;dbl&gt; &lt;dbl&gt; &lt;dbl&gt; &lt;dbl&gt; &lt;dbl&gt; &lt;dbl&gt;
 1 Kest_1  F.puncta… Muse…   110   110   134   134   210   210    97    97   136
 2 Kest_2  F.puncta… Muse…   108   108    NA    NA   176   176    97    97    NA
 3 Kest_3  F.puncta… Muse…   110   108   108   108   228   222    97    97   136
 4 Kest_4  F.puncta… Muse…   110   108   108   108   180   180    97    97   136
 5 Kest_5  F.puncta… Muse…   110   108   108   108   222   210    97    97   136
 6 Kest_6  F.puncta… Muse…   108   108    NA    NA    NA    NA    97    97    NA
 7 Kest_7  F.puncta… Muse…   108   108   134   134   216   216    97    97    NA
 8 Kest_8  F.puncta… Muse…   108   108   134   134   210   192    97    97    NA
 9 Kest_9  F.puncta… Muse…   110   108   136   134   228   216    97    97   136
10 Kest_10 F.puncta… Muse…   108   108    NA    NA    NA    NA    97    97   136
# ℹ 143 more rows
# ℹ 15 more variables: MS5_B &lt;dbl&gt;, MS6_A &lt;dbl&gt;, MS6_B &lt;dbl&gt;, MS7_A &lt;dbl&gt;,
#   MS7_B &lt;dbl&gt;, MS8_A &lt;dbl&gt;, MS8_B &lt;dbl&gt;, MS9_A &lt;dbl&gt;, MS9_B &lt;dbl&gt;,
#   MS10_A &lt;dbl&gt;, MS10_B &lt;dbl&gt;, MS11_A &lt;dbl&gt;, MS11_B &lt;dbl&gt;, MS12_A &lt;dbl&gt;,
#   MS12_B &lt;dbl&gt;</code></pre>
</div>
</div>
</section>
<section class="slide level2">

<ul>
<li><p>Data is organized in a tibble, a data frame with some extra features when compared to regular data frames</p></li>
<li><p>We can summarize the second and third column to overview the type and number of sample</p></li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb29"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb29-1"><a></a><span class="fu">table</span>(kestrel<span class="sc">$</span>Species, kestrel<span class="sc">$</span>Info)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>                 
                  Canary Island Kestrel European Kestrel Greater Kestrel
  F. newtoni                          0                0               0
  F.araea                             0                0               0
  F.naumanni                          0                0               0
  F.punctatus                         0                0               0
  F.rupicoloides                      0                0              10
  F.t.canariensis                     4                0               0
  F.t.dacotiae                        4                0               0
  F.t.rufescens                       0                0               0
  F.t.rupicolus                       0                0               0
  F.t.tinnunculus                     0               10               0
                 
                  Kestrel Lesser Kestrel Madagascan Kestrel Mauritius
  F. newtoni            0              0                  1         0
  F.araea               0              0                  0         0
  F.naumanni            0              8                  0         0
  F.punctatus           0              0                  0        75
  F.rupicoloides        0              0                  0         0
  F.t.canariensis       0              0                  0         0
  F.t.dacotiae          0              0                  0         0
  F.t.rufescens         1              0                  0         0
  F.t.rupicolus         0              0                  0         0
  F.t.tinnunculus       0              0                  0         0
                 
                  Museum Skin SA Rock Kestrel Seychelles Kestrel
  F. newtoni                0               0                  0
  F.araea                   0               0                  4
  F.naumanni                0               0                  0
  F.punctatus              26               0                  0
  F.rupicoloides            0               0                  0
  F.t.canariensis           0               0                  0
  F.t.dacotiae              0               0                  0
  F.t.rufescens             0               0                  0
  F.t.rupicolus             0              10                  0
  F.t.tinnunculus           0               0                  0</code></pre>
</div>
</div>
<ul>
<li>The results tell us that there are 6 different species of kestrel, coming from various locations</li>
</ul>
</section>
<section id="references" class="slide level2 smaller scrollable">
<h2>References</h2>


<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" data-line-spacing="2" role="list">
<div id="ref-Groombridge2000" class="csl-entry" role="listitem">
Groombridge, J. J., Jones, C. G., Bruford, M. W., &amp; Nichols, R. A. (2000). <a href="https://doi.org/10.1038/35001148"><span class="nocase">’Ghost’ alleles of the Mauritius kestrel.</span></a> <em>Nature</em>, <em>403</em>(6770), 616.
</div>
</div>
</section>
    </div>
  <div class="quarto-auto-generated-content" style="display: none;">
<p><img src="../../figures/Tor-Vergata_logo_cmyk.png" class="slide-logo"></p>
<div class="footer footer-default">
<p><a href="https://giulianocolosimo.github.io/biologia_della_conservazione_24-25/">Biologia della conservazione 24/25 - Giuliano Colosimo, PhD</a></p>
</div>
</div></div>

  <script>window.backupDefine = window.define; window.define = undefined;</script>
  <script src="../../site_libs/revealjs/dist/reveal.js"></script>
  <!-- reveal.js plugins -->
  <script src="../../site_libs/revealjs/plugin/quarto-line-highlight/line-highlight.js"></script>
  <script src="../../site_libs/revealjs/plugin/pdf-export/pdfexport.js"></script>
  <script src="../../site_libs/revealjs/plugin/reveal-menu/menu.js"></script>
  <script src="../../site_libs/revealjs/plugin/reveal-menu/quarto-menu.js"></script>
  <script src="../../site_libs/revealjs/plugin/reveal-chalkboard/plugin.js"></script>
  <script src="../../site_libs/revealjs/plugin/quarto-support/support.js"></script>
  

  <script src="../../site_libs/revealjs/plugin/notes/notes.js"></script>
  <script src="../../site_libs/revealjs/plugin/search/search.js"></script>
  <script src="../../site_libs/revealjs/plugin/zoom/zoom.js"></script>
  <script src="../../site_libs/revealjs/plugin/math/math.js"></script>
  <script>window.define = window.backupDefine; window.backupDefine = undefined;</script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
'controlsAuto': true,
'previewLinksAuto': false,
'pdfSeparateFragments': false,
'autoAnimateEasing': "ease",
'autoAnimateDuration': 1,
'autoAnimateUnmatched': true,
'jumpToSlide': true,
'menu': {"side":"left","useTextContentForMissingTitles":true,"markers":false,"loadIcons":false,"custom":[{"title":"Tools","icon":"<i class=\"fas fa-gear\"></i>","content":"<ul class=\"slide-menu-items\">\n<li class=\"slide-tool-item active\" data-item=\"0\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.fullscreen(event)\"><kbd>f</kbd> Fullscreen</a></li>\n<li class=\"slide-tool-item\" data-item=\"1\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.speakerMode(event)\"><kbd>s</kbd> Speaker View</a></li>\n<li class=\"slide-tool-item\" data-item=\"2\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>o</kbd> Slide Overview</a></li>\n<li class=\"slide-tool-item\" data-item=\"3\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.togglePdfExport(event)\"><kbd>e</kbd> PDF Export Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"4\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleScrollView(event)\"><kbd>r</kbd> Scroll View Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"5\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleChalkboard(event)\"><kbd>b</kbd> Toggle Chalkboard</a></li>\n<li class=\"slide-tool-item\" data-item=\"6\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleNotesCanvas(event)\"><kbd>c</kbd> Toggle Notes Canvas</a></li>\n<li class=\"slide-tool-item\" data-item=\"7\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.downloadDrawings(event)\"><kbd>d</kbd> Download Drawings</a></li>\n<li class=\"slide-tool-item\" data-item=\"8\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.keyboardHelp(event)\"><kbd>?</kbd> Keyboard Help</a></li>\n</ul>"}],"openButton":true},
'chalkboard': {"buttons":true,"boardmarkerWidth":5},
'smaller': false,
 
        // Display controls in the bottom right corner
        controls: false,

        // Help the user learn the controls by providing hints, for example by
        // bouncing the down arrow when they first encounter a vertical slide
        controlsTutorial: false,

        // Determines where controls appear, "edges" or "bottom-right"
        controlsLayout: 'edges',

        // Visibility rule for backwards navigation arrows; "faded", "hidden"
        // or "visible"
        controlsBackArrows: 'faded',

        // Display a presentation progress bar
        progress: true,

        // Display the page number of the current slide
        slideNumber: 'c/t',

        // 'all', 'print', or 'speaker'
        showSlideNumber: 'all',

        // Add the current slide number to the URL hash so that reloading the
        // page/copying the URL will return you to the same slide
        hash: true,

        // Start with 1 for the hash rather than 0
        hashOneBasedIndex: false,

        // Flags if we should monitor the hash and change slides accordingly
        respondToHashChanges: true,

        // Push each slide change to the browser history
        history: true,

        // Enable keyboard shortcuts for navigation
        keyboard: true,

        // Enable the slide overview mode
        overview: true,

        // Disables the default reveal.js slide layout (scaling and centering)
        // so that you can use custom CSS layout
        disableLayout: false,

        // Vertical centering of slides
        center: false,

        // Enables touch navigation on devices with touch input
        touch: true,

        // Loop the presentation
        loop: false,

        // Change the presentation direction to be RTL
        rtl: false,

        // see https://revealjs.com/vertical-slides/#navigation-mode
        navigationMode: 'linear',

        // Randomizes the order of slides each time the presentation loads
        shuffle: false,

        // Turns fragments on and off globally
        fragments: true,

        // Flags whether to include the current fragment in the URL,
        // so that reloading brings you to the same fragment position
        fragmentInURL: false,

        // Flags if the presentation is running in an embedded mode,
        // i.e. contained within a limited portion of the screen
        embedded: false,

        // Flags if we should show a help overlay when the questionmark
        // key is pressed
        help: true,

        // Flags if it should be possible to pause the presentation (blackout)
        pause: true,

        // Flags if speaker notes should be visible to all viewers
        showNotes: false,

        // Global override for autoplaying embedded media (null/true/false)
        autoPlayMedia: null,

        // Global override for preloading lazy-loaded iframes (null/true/false)
        preloadIframes: null,

        // Number of milliseconds between automatically proceeding to the
        // next slide, disabled when set to 0, this value can be overwritten
        // by using a data-autoslide attribute on your slides
        autoSlide: 0,

        // Stop auto-sliding after user input
        autoSlideStoppable: true,

        // Use this method for navigation when auto-sliding
        autoSlideMethod: null,

        // Specify the average time in seconds that you think you will spend
        // presenting each slide. This is used to show a pacing timer in the
        // speaker view
        defaultTiming: null,

        // Enable slide navigation via mouse wheel
        mouseWheel: false,

        // The display mode that will be used to show slides
        display: 'block',

        // Hide cursor if inactive
        hideInactiveCursor: true,

        // Time before the cursor is hidden (in ms)
        hideCursorTime: 5000,

        // Opens links in an iframe preview overlay
        previewLinks: true,

        // Transition style (none/fade/slide/convex/concave/zoom)
        transition: 'fade',

        // Transition speed (default/fast/slow)
        transitionSpeed: 'default',

        // Transition style for full page slide backgrounds
        // (none/fade/slide/convex/concave/zoom)
        backgroundTransition: 'none',

        // Number of slides away from the current that are visible
        viewDistance: 3,

        // Number of slides away from the current that are visible on mobile
        // devices. It is advisable to set this to a lower number than
        // viewDistance in order to save resources.
        mobileViewDistance: 2,

        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1600,

        height: 900,

        // Factor of the display size that should remain empty around the content
        margin: 0.1,

        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [QuartoLineHighlight, PdfExport, RevealMenu, RevealChalkboard, QuartoSupport,

          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    
    <script>
      // htmlwidgets need to know to resize themselves when slides are shown/hidden.
      // Fire the "slideenter" event (handled by htmlwidgets.js) when the current
      // slide changes (different for each slide format).
      (function () {
        // dispatch for htmlwidgets
        function fireSlideEnter() {
          const event = window.document.createEvent("Event");
          event.initEvent("slideenter", true, true);
          window.document.dispatchEvent(event);
        }

        function fireSlideChanged(previousSlide, currentSlide) {
          fireSlideEnter();

          // dispatch for shiny
          if (window.jQuery) {
            if (previousSlide) {
              window.jQuery(previousSlide).trigger("hidden");
            }
            if (currentSlide) {
              window.jQuery(currentSlide).trigger("shown");
            }
          }
        }

        // hookup for slidy
        if (window.w3c_slidy) {
          window.w3c_slidy.add_observer(function (slide_num) {
            // slide_num starts at position 1
            fireSlideChanged(null, w3c_slidy.slides[slide_num - 1]);
          });
        }

      })();
    </script>

    <script id="quarto-html-after-body" type="application/javascript">
    window.document.addEventListener("DOMContentLoaded", function (event) {
      const toggleBodyColorMode = (bsSheetEl) => {
        const mode = bsSheetEl.getAttribute("data-mode");
        const bodyEl = window.document.querySelector("body");
        if (mode === "dark") {
          bodyEl.classList.add("quarto-dark");
          bodyEl.classList.remove("quarto-light");
        } else {
          bodyEl.classList.add("quarto-light");
          bodyEl.classList.remove("quarto-dark");
        }
      }
      const toggleBodyColorPrimary = () => {
        const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
        if (bsSheetEl) {
          toggleBodyColorMode(bsSheetEl);
        }
      }
      toggleBodyColorPrimary();  
      const tabsets =  window.document.querySelectorAll(".panel-tabset-tabby")
      tabsets.forEach(function(tabset) {
        const tabby = new Tabby('#' + tabset.id);
      });
      const isCodeAnnotation = (el) => {
        for (const clz of el.classList) {
          if (clz.startsWith('code-annotation-')) {                     
            return true;
          }
        }
        return false;
      }
      const onCopySuccess = function(e) {
        // button target
        const button = e.trigger;
        // don't keep focus
        button.blur();
        // flash "checked"
        button.classList.add('code-copy-button-checked');
        var currentTitle = button.getAttribute("title");
        button.setAttribute("title", "Copied!");
        let tooltip;
        if (window.bootstrap) {
          button.setAttribute("data-bs-toggle", "tooltip");
          button.setAttribute("data-bs-placement", "left");
          button.setAttribute("data-bs-title", "Copied!");
          tooltip = new bootstrap.Tooltip(button, 
            { trigger: "manual", 
              customClass: "code-copy-button-tooltip",
              offset: [0, -8]});
          tooltip.show();    
        }
        setTimeout(function() {
          if (tooltip) {
            tooltip.hide();
            button.removeAttribute("data-bs-title");
            button.removeAttribute("data-bs-toggle");
            button.removeAttribute("data-bs-placement");
          }
          button.setAttribute("title", currentTitle);
          button.classList.remove('code-copy-button-checked');
        }, 1000);
        // clear code selection
        e.clearSelection();
      }
      const getTextToCopy = function(trigger) {
          const codeEl = trigger.previousElementSibling.cloneNode(true);
          for (const childEl of codeEl.children) {
            if (isCodeAnnotation(childEl)) {
              childEl.remove();
            }
          }
          return codeEl.innerText;
      }
      const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
        text: getTextToCopy
      });
      clipboard.on('success', onCopySuccess);
      if (window.document.getElementById('quarto-embedded-source-code-modal')) {
        const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
          text: getTextToCopy,
          container: window.document.getElementById('quarto-embedded-source-code-modal')
        });
        clipboardModal.on('success', onCopySuccess);
      }
        var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
        var mailtoRegex = new RegExp(/^mailto:/);
          var filterRegex = new RegExp("https:\/\/giulianocolosimo\.github\.com");
        var isInternal = (href) => {
            return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
        }
        // Inspect non-navigation links and adorn them if external
     	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
        for (var i=0; i<links.length; i++) {
          const link = links[i];
          if (!isInternal(link.href)) {
            // undo the damage that might have been done by quarto-nav.js in the case of
            // links that we want to consider external
            if (link.dataset.originalHref !== undefined) {
              link.href = link.dataset.originalHref;
            }
          }
        }
      function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
        const config = {
          allowHTML: true,
          maxWidth: 500,
          delay: 100,
          arrow: false,
          appendTo: function(el) {
              return el.closest('section.slide') || el.parentElement;
          },
          interactive: true,
          interactiveBorder: 10,
          theme: 'light-border',
          placement: 'bottom-start',
        };
        if (contentFn) {
          config.content = contentFn;
        }
        if (onTriggerFn) {
          config.onTrigger = onTriggerFn;
        }
        if (onUntriggerFn) {
          config.onUntrigger = onUntriggerFn;
        }
          config['offset'] = [0,0];
          config['maxWidth'] = 700;
        window.tippy(el, config); 
      }
      const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
      for (var i=0; i<noterefs.length; i++) {
        const ref = noterefs[i];
        tippyHover(ref, function() {
          // use id or data attribute instead here
          let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
          try { href = new URL(href).hash; } catch {}
          const id = href.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note) {
            return note.innerHTML;
          } else {
            return "";
          }
        });
      }
      const findCites = (el) => {
        const parentEl = el.parentElement;
        if (parentEl) {
          const cites = parentEl.dataset.cites;
          if (cites) {
            return {
              el,
              cites: cites.split(' ')
            };
          } else {
            return findCites(el.parentElement)
          }
        } else {
          return undefined;
        }
      };
      var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
      for (var i=0; i<bibliorefs.length; i++) {
        const ref = bibliorefs[i];
        const citeInfo = findCites(ref);
        if (citeInfo) {
          tippyHover(citeInfo.el, function() {
            var popup = window.document.createElement('div');
            citeInfo.cites.forEach(function(cite) {
              var citeDiv = window.document.createElement('div');
              citeDiv.classList.add('hanging-indent');
              citeDiv.classList.add('csl-entry');
              var biblioDiv = window.document.getElementById('ref-' + cite);
              if (biblioDiv) {
                citeDiv.innerHTML = biblioDiv.innerHTML;
              }
              popup.appendChild(citeDiv);
            });
            return popup.innerHTML;
          });
        }
      }
    });
    </script>
    

</body></html>